{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import the qiime2 python module to allow for inline visualizations\n",
    "\n",
    "import qiime2 as q2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#which version of qiime2 and python are we using?\n",
    "\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First, import the paired forward and reverse sequences with a manifest file. Manifest prepared by listing all files in bash and formatting in Excel. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported zhan_manifest_runbb_v2_grid.csv as PairedEndFastqManifestPhred33 to 18S-paired-end-demux_r1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path zhan_manifest_runbb_v2_grid.csv \\\n",
    "  --output-path 18S-paired-end-demux_r1.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Trim reverse primers\n",
    "See: https://docs.qiime2.org/2020.6/plugins/available/cutadapt/trim-paired/\n",
    "default is to match IUPAC wildcards ( --p-match-adapter-wildcards T )\n",
    "- these should be the F/R marker primers\n",
    "- cannot handle inosine bases; using N instead\n",
    "- 6bp of barcode\n",
    "- f primer/ r primer\n",
    "- or could just trim off length of primer+barcode? 32bp F; 32bp R but this provides a test of barcode retetion between pcr1/pcr2 \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1000_294_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1000_295_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1000_294_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1000_295_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1000_294_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1000_295_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1000_294_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1000_295_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,434 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.34 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,434\n",
      "  Read 1 with adapter:                  55,287 (99.7%)\n",
      "  Read 2 with adapter:                  54,939 (99.1%)\n",
      "Pairs written (passing filters):        55,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,675,552 bp\n",
      "  Read 1:    16,334,861 bp\n",
      "  Read 2:    16,340,691 bp\n",
      "Total written (filtered):     30,526,069 bp (93.4%)\n",
      "  Read 1:    15,229,081 bp\n",
      "  Read 2:    15,296,988 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t384\t0.0\t1\t0 376 8\n",
      "20\t54749\t0.0\t2\t53804 932 13\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t54575\t0.0\t1\t51773 2802\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1001_296_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1001_297_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1001_296_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1001_297_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1001_296_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1001_297_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1001_296_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1001_297_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,332 reads  @     76.5 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.94 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,332\n",
      "  Read 1 with adapter:                  51,299 (99.9%)\n",
      "  Read 2 with adapter:                  51,027 (99.4%)\n",
      "Pairs written (passing filters):        51,332 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,285,302 bp\n",
      "  Read 1:    15,142,696 bp\n",
      "  Read 2:    15,142,606 bp\n",
      "Total written (filtered):     28,290,129 bp (93.4%)\n",
      "  Read 1:    14,116,925 bp\n",
      "  Read 2:    14,173,204 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51299 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t342\t0.0\t1\t0 338 4\n",
      "20\t50814\t0.0\t2\t49945 855 14\n",
      "21\t101\t0.0\t2\t0 100 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51027 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t50720\t0.0\t1\t48040 2680\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1002_298_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1002_299_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1002_298_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1002_299_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1002_298_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1002_299_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1002_298_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1002_299_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,859 reads  @     76.4 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.96 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,859\n",
      "  Read 1 with adapter:                  64,821 (99.9%)\n",
      "  Read 2 with adapter:                  64,294 (99.1%)\n",
      "Pairs written (passing filters):        64,859 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,261,814 bp\n",
      "  Read 1:    19,132,538 bp\n",
      "  Read 2:    19,129,276 bp\n",
      "Total written (filtered):     35,744,220 bp (93.4%)\n",
      "  Read 1:    17,836,356 bp\n",
      "  Read 2:    17,907,864 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t412\t0.0\t1\t0 405 7\n",
      "20\t64219\t0.0\t2\t63154 1051 14\n",
      "21\t138\t0.0\t2\t0 134 4\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64294 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t332\t0.0\t1\t0 332\n",
      "19\t63804\t0.0\t1\t60370 3434\n",
      "20\t158\t0.0\t1\t0 158\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1003_300_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1003_301_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1003_300_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1003_301_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1003_300_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S1003_301_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1003_300_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S1003_301_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,653 reads  @     80.2 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.84 s (81 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,653\n",
      "  Read 1 with adapter:                  47,628 (99.9%)\n",
      "  Read 2 with adapter:                  47,057 (98.7%)\n",
      "Pairs written (passing filters):        47,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,115,199 bp\n",
      "  Read 1:    14,057,621 bp\n",
      "  Read 2:    14,057,578 bp\n",
      "Total written (filtered):     26,268,925 bp (93.4%)\n",
      "  Read 1:    13,105,259 bp\n",
      "  Read 2:    13,163,666 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t293\t0.0\t1\t0 286 7\n",
      "20\t47204\t0.0\t2\t46308 883 13\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t46772\t0.0\t1\t43848 2924\n",
      "20\t57\t0.0\t1\t0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S853_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S853_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S853_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S853_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S853_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S853_1_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S853_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S853_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,425 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.37 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,425\n",
      "  Read 1 with adapter:                  50,405 (100.0%)\n",
      "  Read 2 with adapter:                  49,973 (99.1%)\n",
      "Pairs written (passing filters):        50,425 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,750,028 bp\n",
      "  Read 1:    14,874,875 bp\n",
      "  Read 2:    14,875,153 bp\n",
      "Total written (filtered):     27,792,726 bp (93.4%)\n",
      "  Read 1:    13,866,920 bp\n",
      "  Read 2:    13,925,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50405 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t305\t0.0\t1\t0 301 4\n",
      "20\t49946\t0.0\t2\t49205 735 6\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49973 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t49631\t0.0\t1\t47106 2525\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S854_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S854_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S854_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S854_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S854_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S854_3_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S854_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S854_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        47,362 reads  @     96.8 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.60 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,362\n",
      "  Read 1 with adapter:                  47,334 (99.9%)\n",
      "  Read 2 with adapter:                  46,926 (99.1%)\n",
      "Pairs written (passing filters):        47,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,941,778 bp\n",
      "  Read 1:    13,971,294 bp\n",
      "  Read 2:    13,970,484 bp\n",
      "Total written (filtered):     26,103,843 bp (93.4%)\n",
      "  Read 1:    13,024,815 bp\n",
      "  Read 2:    13,079,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t324\t0.0\t1\t0 315 9\n",
      "20\t46876\t0.0\t2\t46085 783 8\n",
      "21\t93\t0.0\t2\t0 92 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t217\t0.0\t1\t0 217\n",
      "19\t46630\t0.0\t1\t43776 2854\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S855_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S855_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S855_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S855_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S855_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S855_5_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S855_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S855_5_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,691 reads  @     78.2 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.05 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,691\n",
      "  Read 1 with adapter:                  51,666 (100.0%)\n",
      "  Read 2 with adapter:                  51,140 (98.9%)\n",
      "Pairs written (passing filters):        51,691 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,494,970 bp\n",
      "  Read 1:    15,248,830 bp\n",
      "  Read 2:    15,246,140 bp\n",
      "Total written (filtered):     28,490,337 bp (93.4%)\n",
      "  Read 1:    14,215,732 bp\n",
      "  Read 2:    14,274,605 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t329\t0.0\t1\t0 318 11\n",
      "20\t51211\t0.0\t2\t50239 963 9\n",
      "21\t89\t0.0\t2\t0 85 4\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t225\t0.0\t1\t0 225\n",
      "19\t50815\t0.0\t1\t47733 3082\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S856_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S856_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S856_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S856_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S856_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S856_7_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S856_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S856_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,278 reads  @     78.9 µs/read;   0.76 M reads/minute\n",
      "Finished in 5.08 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,278\n",
      "  Read 1 with adapter:                  64,248 (100.0%)\n",
      "  Read 2 with adapter:                  63,736 (99.2%)\n",
      "Pairs written (passing filters):        64,278 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,923,968 bp\n",
      "  Read 1:    18,961,990 bp\n",
      "  Read 2:    18,961,978 bp\n",
      "Total written (filtered):     35,428,515 bp (93.4%)\n",
      "  Read 1:    17,677,313 bp\n",
      "  Read 2:    17,751,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64248 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t412\t0.0\t1\t0 407 5\n",
      "20\t63653\t0.0\t2\t62640 996 17\n",
      "21\t129\t0.0\t2\t0 127 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 63736 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t356\t0.0\t1\t0 356\n",
      "19\t63232\t0.0\t1\t59395 3837\n",
      "20\t148\t0.0\t1\t0 148\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S857_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S857_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S857_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S857_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S857_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S857_9_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S857_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S857_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,507 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.75 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,507\n",
      "  Read 1 with adapter:                  48,480 (99.9%)\n",
      "  Read 2 with adapter:                  48,078 (99.1%)\n",
      "Pairs written (passing filters):        48,507 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,617,445 bp\n",
      "  Read 1:    14,309,553 bp\n",
      "  Read 2:    14,307,892 bp\n",
      "Total written (filtered):     26,734,712 bp (93.4%)\n",
      "  Read 1:    13,340,188 bp\n",
      "  Read 2:    13,394,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t324\t0.0\t1\t0 317 7\n",
      "20\t48018\t0.0\t2\t47248 755 15\n",
      "21\t99\t0.0\t2\t0 97 2\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t47770\t0.0\t1\t44693 3077\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S858_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S858_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S858_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S858_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S858_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S858_11_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S858_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S858_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,725 reads  @     75.2 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.91 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,725\n",
      "  Read 1 with adapter:                  51,692 (99.9%)\n",
      "  Read 2 with adapter:                  51,215 (99.0%)\n",
      "Pairs written (passing filters):        51,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,506,550 bp\n",
      "  Read 1:    15,258,773 bp\n",
      "  Read 2:    15,247,777 bp\n",
      "Total written (filtered):     28,499,926 bp (93.4%)\n",
      "  Read 1:    14,225,136 bp\n",
      "  Read 2:    14,274,790 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t312\t0.0\t1\t0 309 3\n",
      "20\t51231\t0.0\t2\t50405 814 12\n",
      "21\t97\t0.0\t2\t0 97\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t50891\t0.0\t1\t47861 3030\n",
      "20\t113\t0.0\t1\t0 113\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S859_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S859_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S859_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S859_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S859_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S859_13_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S859_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S859_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,674 reads  @     70.2 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.58 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,674\n",
      "  Read 1 with adapter:                  50,654 (100.0%)\n",
      "  Read 2 with adapter:                  50,308 (99.3%)\n",
      "Pairs written (passing filters):        50,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,882,477 bp\n",
      "  Read 1:    14,948,342 bp\n",
      "  Read 2:    14,934,135 bp\n",
      "Total written (filtered):     27,913,359 bp (93.4%)\n",
      "  Read 1:    13,934,943 bp\n",
      "  Read 2:    13,978,416 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t314\t0.0\t1\t0 308 6\n",
      "20\t50208\t0.0\t2\t49403 792 13\n",
      "21\t85\t0.0\t2\t0 85\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50308 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t49939\t0.0\t1\t47412 2527\n",
      "20\t118\t0.0\t1\t0 118\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S860_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S860_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S860_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S860_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S860_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S860_15_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S860_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S860_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,203 reads  @     73.1 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.92 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,203\n",
      "  Read 1 with adapter:                  53,176 (99.9%)\n",
      "  Read 2 with adapter:                  52,317 (98.3%)\n",
      "Pairs written (passing filters):        53,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,348,177 bp\n",
      "  Read 1:    15,694,868 bp\n",
      "  Read 2:    15,653,309 bp\n",
      "Total written (filtered):     29,290,986 bp (93.4%)\n",
      "  Read 1:    14,631,561 bp\n",
      "  Read 2:    14,659,425 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53176 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t345\t0.0\t1\t0 342 3\n",
      "20\t52677\t0.0\t2\t51802 862 13\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52317 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t255\t0.0\t1\t0 255\n",
      "19\t51946\t0.0\t1\t49519 2427\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S861_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S861_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S861_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S861_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S861_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S861_17_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S861_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S861_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        43,558 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.26 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,558\n",
      "  Read 1 with adapter:                  43,540 (100.0%)\n",
      "  Read 2 with adapter:                  43,292 (99.4%)\n",
      "Pairs written (passing filters):        43,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,698,658 bp\n",
      "  Read 1:    12,849,594 bp\n",
      "  Read 2:    12,849,064 bp\n",
      "Total written (filtered):     24,005,615 bp (93.4%)\n",
      "  Read 1:    11,978,975 bp\n",
      "  Read 2:    12,026,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43540 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t295\t0.0\t1\t0 287 8\n",
      "20\t43130\t0.0\t2\t42424 700 6\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43292 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t206\t0.0\t1\t0 206\n",
      "19\t43004\t0.0\t1\t40855 2149\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S862_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S862_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S862_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S862_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S862_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S862_19_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S862_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S862_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,065 reads  @     75.0 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.78 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,065\n",
      "  Read 1 with adapter:                  50,033 (99.9%)\n",
      "  Read 2 with adapter:                  49,750 (99.4%)\n",
      "Pairs written (passing filters):        50,065 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,537,746 bp\n",
      "  Read 1:    14,769,036 bp\n",
      "  Read 2:    14,768,710 bp\n",
      "Total written (filtered):     27,592,198 bp (93.4%)\n",
      "  Read 1:    13,768,567 bp\n",
      "  Read 2:    13,823,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t306\t0.0\t1\t0 301 5\n",
      "20\t49596\t0.0\t2\t48818 772 6\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49750 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t262\t0.0\t1\t0 262\n",
      "19\t49397\t0.0\t1\t46832 2565\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S863_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S863_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S863_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S863_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S863_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S863_21_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S863_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S863_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,927 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.19 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,927\n",
      "  Read 1 with adapter:                  43,897 (99.9%)\n",
      "  Read 2 with adapter:                  43,596 (99.2%)\n",
      "Pairs written (passing filters):        43,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,913,031 bp\n",
      "  Read 1:    12,958,457 bp\n",
      "  Read 2:    12,954,574 bp\n",
      "Total written (filtered):     24,207,100 bp (93.4%)\n",
      "  Read 1:    12,080,709 bp\n",
      "  Read 2:    12,126,391 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43897 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t285\t0.0\t1\t0 279 6\n",
      "20\t43512\t0.0\t2\t42837 668 7\n",
      "21\t63\t0.0\t2\t0 63\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43596 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t222\t0.0\t1\t0 222\n",
      "19\t43293\t0.0\t1\t41015 2278\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S864_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S864_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S864_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S864_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S864_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S864_23_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S864_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S864_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,385 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.73 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,385\n",
      "  Read 1 with adapter:                  38,367 (100.0%)\n",
      "  Read 2 with adapter:                  38,125 (99.3%)\n",
      "Pairs written (passing filters):        38,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,647,121 bp\n",
      "  Read 1:    11,323,566 bp\n",
      "  Read 2:    11,323,555 bp\n",
      "Total written (filtered):     21,155,703 bp (93.4%)\n",
      "  Read 1:    10,556,360 bp\n",
      "  Read 2:    10,599,343 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38367 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t228\t0.0\t1\t0 226 2\n",
      "20\t38042\t0.0\t2\t37380 656 6\n",
      "21\t60\t0.0\t2\t0 57 3\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38125 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t245\t0.0\t1\t0 245\n",
      "19\t37798\t0.0\t1\t35757 2041\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S865_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S865_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S865_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S865_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S865_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S865_25_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S865_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S865_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,080 reads  @     72.7 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.37 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,080\n",
      "  Read 1 with adapter:                  46,051 (99.9%)\n",
      "  Read 2 with adapter:                  45,778 (99.3%)\n",
      "Pairs written (passing filters):        46,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,186,022 bp\n",
      "  Read 1:    13,593,584 bp\n",
      "  Read 2:    13,592,438 bp\n",
      "Total written (filtered):     25,395,537 bp (93.4%)\n",
      "  Read 1:    12,672,722 bp\n",
      "  Read 2:    12,722,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46051 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t270\t0.0\t1\t0 263 7\n",
      "20\t45648\t0.0\t2\t44937 698 13\n",
      "21\t90\t0.0\t2\t0 89 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45778 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t237\t0.0\t1\t0 237\n",
      "19\t45463\t0.0\t1\t43031 2432\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S866_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S866_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S866_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S866_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S866_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S866_27_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S866_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S866_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,399 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.11 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,399\n",
      "  Read 1 with adapter:                  44,368 (99.9%)\n",
      "  Read 2 with adapter:                  44,034 (99.2%)\n",
      "Pairs written (passing filters):        44,399 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,183,694 bp\n",
      "  Read 1:    13,097,624 bp\n",
      "  Read 2:    13,086,070 bp\n",
      "Total written (filtered):     24,459,938 bp (93.4%)\n",
      "  Read 1:    12,210,426 bp\n",
      "  Read 2:    12,249,512 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t43992\t0.0\t2\t43296 686 10\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t179\t0.0\t1\t0 179\n",
      "19\t43764\t0.0\t1\t41519 2245\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S867_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S867_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S867_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S867_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S867_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S867_29_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S867_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S867_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:02        42,058 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.91 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,058\n",
      "  Read 1 with adapter:                  42,037 (100.0%)\n",
      "  Read 2 with adapter:                  41,821 (99.4%)\n",
      "Pairs written (passing filters):        42,058 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,802,216 bp\n",
      "  Read 1:    12,406,853 bp\n",
      "  Read 2:    12,395,363 bp\n",
      "Total written (filtered):     23,167,228 bp (93.4%)\n",
      "  Read 1:    11,566,336 bp\n",
      "  Read 2:    11,600,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t281\t0.0\t1\t0 275 6\n",
      "20\t41671\t0.0\t2\t40964 703 4\n",
      "21\t56\t0.0\t2\t0 55 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t208\t0.0\t1\t0 208\n",
      "19\t41533\t0.0\t1\t39583 1950\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S868_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S868_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S868_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S868_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S868_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S868_31_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S868_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S868_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,684 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.57 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,684\n",
      "  Read 1 with adapter:                  43,665 (100.0%)\n",
      "  Read 2 with adapter:                  42,888 (98.2%)\n",
      "Pairs written (passing filters):        43,684 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,735,953 bp\n",
      "  Read 1:    12,886,470 bp\n",
      "  Read 2:    12,849,483 bp\n",
      "Total written (filtered):     24,048,021 bp (93.4%)\n",
      "  Read 1:    12,013,325 bp\n",
      "  Read 2:    12,034,696 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t253\t0.0\t1\t0 246 7\n",
      "20\t43288\t0.0\t2\t42541 733 14\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42888 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t174\t0.0\t1\t0 174\n",
      "19\t42625\t0.0\t1\t40660 1965\n",
      "20\t89\t0.0\t1\t0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S869_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S869_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S869_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S869_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S869_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S869_33_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S869_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S869_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,861 reads  @     80.3 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.80 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,861\n",
      "  Read 1 with adapter:                  46,840 (100.0%)\n",
      "  Read 2 with adapter:                  46,397 (99.0%)\n",
      "Pairs written (passing filters):        46,861 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,645,214 bp\n",
      "  Read 1:    13,823,985 bp\n",
      "  Read 2:    13,821,229 bp\n",
      "Total written (filtered):     25,827,185 bp (93.4%)\n",
      "  Read 1:    12,887,337 bp\n",
      "  Read 2:    12,939,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46840 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t269\t0.0\t1\t0 265 4\n",
      "20\t46464\t0.0\t2\t45652 799 13\n",
      "21\t69\t0.0\t2\t0 68 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t241\t0.0\t1\t0 241\n",
      "19\t46077\t0.0\t1\t43546 2531\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S870_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S870_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S870_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S870_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S870_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S870_35_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S870_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S870_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,851 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 2.37 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,851\n",
      "  Read 1 with adapter:                  28,832 (99.9%)\n",
      "  Read 2 with adapter:                  28,494 (98.8%)\n",
      "Pairs written (passing filters):        28,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,018,359 bp\n",
      "  Read 1:     8,509,742 bp\n",
      "  Read 2:     8,508,617 bp\n",
      "Total written (filtered):     15,900,474 bp (93.4%)\n",
      "  Read 1:     7,933,193 bp\n",
      "  Read 2:     7,967,281 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 28832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t156\t0.0\t1\t0 152 4\n",
      "20\t28608\t0.0\t2\t28072 530 6\n",
      "21\t47\t0.0\t2\t0 45 2\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28494 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t111\t0.0\t1\t0 111\n",
      "19\t28322\t0.0\t1\t26613 1709\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S871_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S871_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S871_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S871_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S871_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S871_37_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S871_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S871_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,900 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.90 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,900\n",
      "  Read 1 with adapter:                  54,866 (99.9%)\n",
      "  Read 2 with adapter:                  54,315 (98.9%)\n",
      "Pairs written (passing filters):        54,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,389,798 bp\n",
      "  Read 1:    16,195,298 bp\n",
      "  Read 2:    16,194,500 bp\n",
      "Total written (filtered):     30,260,788 bp (93.4%)\n",
      "  Read 1:    15,098,147 bp\n",
      "  Read 2:    15,162,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54866 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t305\t0.0\t1\t0 300 5\n",
      "20\t54418\t0.0\t2\t53473 925 20\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t228\t0.0\t1\t0 228\n",
      "19\t53985\t0.0\t1\t50877 3108\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S872_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S872_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S872_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S872_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S872_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S872_39_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S872_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S872_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,529 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.14 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,529\n",
      "  Read 1 with adapter:                  43,508 (100.0%)\n",
      "  Read 2 with adapter:                  43,045 (98.9%)\n",
      "Pairs written (passing filters):        43,529 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,681,765 bp\n",
      "  Read 1:    12,840,903 bp\n",
      "  Read 2:    12,840,862 bp\n",
      "Total written (filtered):     23,994,063 bp (93.4%)\n",
      "  Read 1:    11,970,896 bp\n",
      "  Read 2:    12,023,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43508 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t247\t0.0\t1\t0 242 5\n",
      "20\t43141\t0.0\t2\t42390 747 4\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t42689\t0.0\t1\t40168 2521\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S873_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S873_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S873_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S873_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S873_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S873_41_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S873_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S873_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[  8<--------] 00:00:07        98,667 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 7.22 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,667\n",
      "  Read 1 with adapter:                  98,613 (99.9%)\n",
      "  Read 2 with adapter:                  97,484 (98.8%)\n",
      "Pairs written (passing filters):        98,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:    58,206,055 bp\n",
      "  Read 1:    29,106,734 bp\n",
      "  Read 2:    29,099,321 bp\n",
      "Total written (filtered):     54,382,169 bp (93.4%)\n",
      "  Read 1:    27,134,768 bp\n",
      "  Read 2:    27,247,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 98613 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t521\t0.0\t1\t0 508 13\n",
      "20\t97834\t0.0\t2\t96108 1708 18\n",
      "21\t181\t0.0\t2\t0 178 3\n",
      "22\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 97484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t461\t0.0\t1\t0 461\n",
      "19\t96838\t0.0\t1\t90609 6229\n",
      "20\t185\t0.0\t1\t0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S874_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S874_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S874_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S874_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S874_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S874_43_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S874_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S874_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,478 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.97 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,478\n",
      "  Read 1 with adapter:                  63,436 (99.9%)\n",
      "  Read 2 with adapter:                  62,700 (98.8%)\n",
      "Pairs written (passing filters):        63,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,426,813 bp\n",
      "  Read 1:    18,725,756 bp\n",
      "  Read 2:    18,701,057 bp\n",
      "Total written (filtered):     34,967,135 bp (93.4%)\n",
      "  Read 1:    17,457,197 bp\n",
      "  Read 2:    17,509,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 63436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t321\t0.0\t1\t0 314 7\n",
      "20\t62943\t0.0\t2\t61838 1091 14\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 62700 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t281\t0.0\t1\t0 281\n",
      "19\t62319\t0.0\t1\t58827 3492\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S875_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S875_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S875_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S875_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S875_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S875_45_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S875_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S875_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,386 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.35 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,386\n",
      "  Read 1 with adapter:                  53,356 (99.9%)\n",
      "  Read 2 with adapter:                  52,780 (98.9%)\n",
      "Pairs written (passing filters):        53,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,480,250 bp\n",
      "  Read 1:    15,748,732 bp\n",
      "  Read 2:    15,731,518 bp\n",
      "Total written (filtered):     29,410,651 bp (93.4%)\n",
      "  Read 1:    14,681,841 bp\n",
      "  Read 2:    14,728,810 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t333\t0.0\t1\t0 324 9\n",
      "20\t52878\t0.0\t2\t51938 933 7\n",
      "21\t106\t0.0\t2\t0 106\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t217\t0.0\t1\t0 217\n",
      "19\t52458\t0.0\t1\t49686 2772\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S876_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S876_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S876_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S876_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S876_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S876_47_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S876_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S876_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,154 reads  @     87.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.37 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,154\n",
      "  Read 1 with adapter:                  61,127 (100.0%)\n",
      "  Read 2 with adapter:                  59,296 (97.0%)\n",
      "Pairs written (passing filters):        61,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,999,231 bp\n",
      "  Read 1:    18,040,230 bp\n",
      "  Read 2:    17,959,001 bp\n",
      "Total written (filtered):     33,650,408 bp (93.5%)\n",
      "  Read 1:    16,817,885 bp\n",
      "  Read 2:    16,832,523 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61127 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t21\t0.0\t1\t0 0 21\n",
      "19\t345\t0.0\t1\t0 341 4\n",
      "20\t60603\t0.0\t2\t59514 1076 13\n",
      "21\t124\t0.0\t2\t0 124\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t251\t0.0\t1\t0 251\n",
      "19\t58940\t0.0\t1\t55648 3292\n",
      "20\t105\t0.0\t1\t0 105\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S877_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S877_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S877_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S877_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S877_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S877_49_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S877_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S877_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,672 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.01 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,672\n",
      "  Read 1 with adapter:                  48,647 (99.9%)\n",
      "  Read 2 with adapter:                  48,451 (99.5%)\n",
      "Pairs written (passing filters):        48,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,715,267 bp\n",
      "  Read 1:    14,358,228 bp\n",
      "  Read 2:    14,357,039 bp\n",
      "Total written (filtered):     26,822,090 bp (93.4%)\n",
      "  Read 1:    13,385,448 bp\n",
      "  Read 2:    13,436,642 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t297\t0.0\t1\t0 293 4\n",
      "20\t48218\t0.0\t2\t47290 917 11\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48451 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t48085\t0.0\t1\t45871 2214\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S878_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S878_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S878_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S878_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S878_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S878_51_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S878_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S878_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,892 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.52 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,892\n",
      "  Read 1 with adapter:                  48,867 (99.9%)\n",
      "  Read 2 with adapter:                  48,675 (99.6%)\n",
      "Pairs written (passing filters):        48,892 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,845,717 bp\n",
      "  Read 1:    14,423,134 bp\n",
      "  Read 2:    14,422,583 bp\n",
      "Total written (filtered):     26,943,906 bp (93.4%)\n",
      "  Read 1:    13,446,000 bp\n",
      "  Read 2:    13,497,906 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48867 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t295\t0.0\t1\t0 291 4\n",
      "20\t48455\t0.0\t2\t47663 781 11\n",
      "21\t81\t0.0\t2\t0 80 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t48353\t0.0\t1\t46144 2209\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S879_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S879_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S879_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S879_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S879_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S879_53_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S879_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S879_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,745 reads  @     65.8 µs/read;   0.91 M reads/minute\n",
      "Finished in 3.24 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,745\n",
      "  Read 1 with adapter:                  48,722 (100.0%)\n",
      "  Read 2 with adapter:                  48,551 (99.6%)\n",
      "Pairs written (passing filters):        48,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,759,443 bp\n",
      "  Read 1:    14,379,770 bp\n",
      "  Read 2:    14,379,673 bp\n",
      "Total written (filtered):     26,862,844 bp (93.4%)\n",
      "  Read 1:    13,405,525 bp\n",
      "  Read 2:    13,457,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t333\t0.0\t1\t0 327 6\n",
      "20\t48246\t0.0\t2\t47466 763 17\n",
      "21\t100\t0.0\t2\t0 96 4\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48551 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t48236\t0.0\t1\t46167 2069\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S880_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S880_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S880_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S880_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S880_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S880_55_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S880_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S880_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,139 reads  @     67.7 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.28 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,139\n",
      "  Read 1 with adapter:                  48,122 (100.0%)\n",
      "  Read 2 with adapter:                  47,938 (99.6%)\n",
      "Pairs written (passing filters):        48,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,401,700 bp\n",
      "  Read 1:    14,200,783 bp\n",
      "  Read 2:    14,200,917 bp\n",
      "Total written (filtered):     26,528,816 bp (93.4%)\n",
      "  Read 1:    13,238,547 bp\n",
      "  Read 2:    13,290,269 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t306\t0.0\t1\t0 297 9\n",
      "20\t47698\t0.0\t2\t46956 732 10\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47938 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t281\t0.0\t1\t0 281\n",
      "19\t47550\t0.0\t1\t45473 2077\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S881_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S881_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S881_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S881_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S881_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S881_57_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S881_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S881_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,061 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.78 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,061\n",
      "  Read 1 with adapter:                  53,036 (100.0%)\n",
      "  Read 2 with adapter:                  52,859 (99.6%)\n",
      "Pairs written (passing filters):        53,061 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,304,901 bp\n",
      "  Read 1:    15,652,986 bp\n",
      "  Read 2:    15,651,915 bp\n",
      "Total written (filtered):     29,240,243 bp (93.4%)\n",
      "  Read 1:    14,592,491 bp\n",
      "  Read 2:    14,647,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t6\t0.0\t1\t0 0 6\n",
      "19\t362\t0.0\t1\t0 358 4\n",
      "20\t52550\t0.0\t2\t51640 896 14\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t253\t0.0\t1\t0 253\n",
      "19\t52511\t0.0\t1\t50249 2262\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S882_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S882_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S882_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S882_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S882_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S882_59_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S882_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S882_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,625 reads  @     83.6 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.75 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,625\n",
      "  Read 1 with adapter:                  56,587 (99.9%)\n",
      "  Read 2 with adapter:                  56,381 (99.6%)\n",
      "Pairs written (passing filters):        56,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,398,424 bp\n",
      "  Read 1:    16,704,338 bp\n",
      "  Read 2:    16,694,086 bp\n",
      "Total written (filtered):     31,195,887 bp (93.4%)\n",
      "  Read 1:    15,572,878 bp\n",
      "  Read 2:    15,623,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 56587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t382\t0.0\t1\t0 377 5\n",
      "20\t56070\t0.0\t2\t55167 897 6\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 56381 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t56023\t0.0\t1\t53490 2533\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S883_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S883_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S883_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S883_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S883_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S883_61_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S883_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S883_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,690 reads  @     74.7 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.29 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,690\n",
      "  Read 1 with adapter:                  43,674 (100.0%)\n",
      "  Read 2 with adapter:                  43,519 (99.6%)\n",
      "Pairs written (passing filters):        43,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,771,279 bp\n",
      "  Read 1:    12,888,541 bp\n",
      "  Read 2:    12,882,738 bp\n",
      "Total written (filtered):     24,071,250 bp (93.4%)\n",
      "  Read 1:    12,015,263 bp\n",
      "  Read 2:    12,055,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43674 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t293\t0.0\t1\t0 291 2\n",
      "20\t43262\t0.0\t2\t42539 713 10\n",
      "21\t83\t0.0\t2\t0 83\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43519 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t43239\t0.0\t1\t41456 1783\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S884_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S884_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S884_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S884_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S884_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S884_63_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S884_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S884_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,463 reads  @     75.2 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.67 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,463\n",
      "  Read 1 with adapter:                  48,448 (100.0%)\n",
      "  Read 2 with adapter:                  47,662 (98.3%)\n",
      "Pairs written (passing filters):        48,463 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,549,900 bp\n",
      "  Read 1:    14,296,567 bp\n",
      "  Read 2:    14,253,333 bp\n",
      "Total written (filtered):     26,675,736 bp (93.4%)\n",
      "  Read 1:    13,327,814 bp\n",
      "  Read 2:    13,347,922 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t311\t0.0\t1\t0 309 2\n",
      "20\t48010\t0.0\t2\t47204 792 14\n",
      "21\t94\t0.0\t2\t0 93 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47662 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t237\t0.0\t1\t0 237\n",
      "19\t47355\t0.0\t1\t45478 1877\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S885_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S885_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S885_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S885_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S885_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S885_65_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S885_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S885_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,369 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.33 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,369\n",
      "  Read 1 with adapter:                  48,353 (100.0%)\n",
      "  Read 2 with adapter:                  48,114 (99.5%)\n",
      "Pairs written (passing filters):        48,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,535,647 bp\n",
      "  Read 1:    14,268,842 bp\n",
      "  Read 2:    14,266,805 bp\n",
      "Total written (filtered):     26,654,746 bp (93.4%)\n",
      "  Read 1:    13,301,967 bp\n",
      "  Read 2:    13,352,779 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48353 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t301\t0.0\t1\t0 295 6\n",
      "20\t47907\t0.0\t2\t47054 843 10\n",
      "21\t98\t0.0\t2\t0 96 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48114 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t47790\t0.0\t1\t45668 2122\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S886_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S886_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S886_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S886_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S886_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S886_67_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S886_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S886_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,752 reads  @     67.9 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.74 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,752\n",
      "  Read 1 with adapter:                  54,725 (100.0%)\n",
      "  Read 2 with adapter:                  54,492 (99.5%)\n",
      "Pairs written (passing filters):        54,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,302,283 bp\n",
      "  Read 1:    16,151,605 bp\n",
      "  Read 2:    16,150,678 bp\n",
      "Total written (filtered):     30,172,731 bp (93.4%)\n",
      "  Read 1:    15,057,269 bp\n",
      "  Read 2:    15,115,462 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t294\t0.0\t1\t0 285 9\n",
      "20\t54283\t0.0\t2\t53350 922 11\n",
      "21\t98\t0.0\t2\t0 93 5\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t54108\t0.0\t1\t51545 2563\n",
      "20\t126\t0.0\t1\t0 126\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S887_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S887_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S887_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S887_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S887_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S887_69_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S887_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S887_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,711 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.83 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,711\n",
      "  Read 1 with adapter:                  52,690 (100.0%)\n",
      "  Read 2 with adapter:                  52,418 (99.4%)\n",
      "Pairs written (passing filters):        52,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,097,653 bp\n",
      "  Read 1:    15,549,734 bp\n",
      "  Read 2:    15,547,919 bp\n",
      "Total written (filtered):     29,048,184 bp (93.4%)\n",
      "  Read 1:    14,496,086 bp\n",
      "  Read 2:    14,552,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t291\t0.0\t1\t0 283 8\n",
      "20\t52259\t0.0\t2\t51364 886 9\n",
      "21\t89\t0.0\t2\t0 87 2\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52418 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t52085\t0.0\t1\t49597 2488\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S888_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S888_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S888_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S888_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S888_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S888_71_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S888_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S888_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,764 reads  @     78.2 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.62 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,764\n",
      "  Read 1 with adapter:                  45,741 (99.9%)\n",
      "  Read 2 with adapter:                  45,484 (99.4%)\n",
      "Pairs written (passing filters):        45,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,999,790 bp\n",
      "  Read 1:    13,500,177 bp\n",
      "  Read 2:    13,499,613 bp\n",
      "Total written (filtered):     25,221,074 bp (93.4%)\n",
      "  Read 1:    12,585,518 bp\n",
      "  Read 2:    12,635,556 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45741 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t277\t0.0\t1\t0 270 7\n",
      "20\t45326\t0.0\t2\t44568 746 12\n",
      "21\t88\t0.0\t2\t0 88\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t247\t0.0\t1\t0 247\n",
      "19\t45129\t0.0\t1\t42964 2165\n",
      "20\t108\t0.0\t1\t0 108\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S889_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S889_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S889_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S889_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S889_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S889_73_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S889_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S889_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,718 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.79 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,718\n",
      "  Read 1 with adapter:                  54,701 (100.0%)\n",
      "  Read 2 with adapter:                  54,412 (99.4%)\n",
      "Pairs written (passing filters):        54,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,280,907 bp\n",
      "  Read 1:    16,141,793 bp\n",
      "  Read 2:    16,139,114 bp\n",
      "Total written (filtered):     30,153,416 bp (93.4%)\n",
      "  Read 1:    15,047,953 bp\n",
      "  Read 2:    15,105,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54701 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t326\t0.0\t1\t0 318 8\n",
      "20\t54222\t0.0\t2\t53252 952 18\n",
      "21\t108\t0.0\t2\t0 107 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t276\t0.0\t1\t0 276\n",
      "19\t54037\t0.0\t1\t51423 2614\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S890_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S890_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S890_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S890_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S890_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S890_75_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S890_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S890_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,483 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.04 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,483\n",
      "  Read 1 with adapter:                  49,452 (99.9%)\n",
      "  Read 2 with adapter:                  49,226 (99.5%)\n",
      "Pairs written (passing filters):        49,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,189,021 bp\n",
      "  Read 1:    14,597,249 bp\n",
      "  Read 2:    14,591,772 bp\n",
      "Total written (filtered):     27,264,983 bp (93.4%)\n",
      "  Read 1:    13,608,381 bp\n",
      "  Read 2:    13,656,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49452 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t282\t0.0\t1\t0 278 4\n",
      "20\t49045\t0.0\t2\t48238 798 9\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49226 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t48922\t0.0\t1\t46689 2233\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S891_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S891_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S891_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S891_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S891_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S891_77_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S891_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S891_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        50,636 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.31 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,636\n",
      "  Read 1 with adapter:                  50,616 (100.0%)\n",
      "  Read 2 with adapter:                  50,414 (99.6%)\n",
      "Pairs written (passing filters):        50,636 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,865,349 bp\n",
      "  Read 1:    14,937,413 bp\n",
      "  Read 2:    14,927,936 bp\n",
      "Total written (filtered):     27,895,534 bp (93.4%)\n",
      "  Read 1:    13,925,287 bp\n",
      "  Read 2:    13,970,247 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50616 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t302\t0.0\t1\t0 294 8\n",
      "20\t50190\t0.0\t2\t49226 957 7\n",
      "21\t84\t0.0\t2\t0 82 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50414 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t269\t0.0\t1\t0 269\n",
      "19\t50053\t0.0\t1\t47902 2151\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S892_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S892_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S892_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S892_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S892_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S892_79_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S892_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S892_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,129 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.34 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,129\n",
      "  Read 1 with adapter:                  52,097 (99.9%)\n",
      "  Read 2 with adapter:                  51,340 (98.5%)\n",
      "Pairs written (passing filters):        52,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,716,366 bp\n",
      "  Read 1:    15,377,279 bp\n",
      "  Read 2:    15,339,087 bp\n",
      "Total written (filtered):     28,699,312 bp (93.4%)\n",
      "  Read 1:    14,335,536 bp\n",
      "  Read 2:    14,363,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52097 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t316\t0.0\t1\t0 309 7\n",
      "20\t51644\t0.0\t2\t50788 851 5\n",
      "21\t95\t0.0\t2\t0 93 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51340 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t248\t0.0\t1\t0 248\n",
      "19\t50993\t0.0\t1\t48749 2244\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S893_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S893_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S893_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S893_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S893_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S893_81_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S893_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S893_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,977 reads  @     70.8 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.12 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,977\n",
      "  Read 1 with adapter:                  43,954 (99.9%)\n",
      "  Read 2 with adapter:                  43,661 (99.3%)\n",
      "Pairs written (passing filters):        43,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,945,556 bp\n",
      "  Read 1:    12,973,076 bp\n",
      "  Read 2:    12,972,480 bp\n",
      "Total written (filtered):     24,237,141 bp (93.4%)\n",
      "  Read 1:    12,094,081 bp\n",
      "  Read 2:    12,143,060 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t224\t0.0\t1\t0 218 6\n",
      "20\t43577\t0.0\t2\t42911 661 5\n",
      "21\t103\t0.0\t2\t0 99 4\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43661 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t43358\t0.0\t1\t41243 2115\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S894_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S894_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S894_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S894_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S894_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S894_83_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S894_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S894_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,656 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.06 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,656\n",
      "  Read 1 with adapter:                  42,632 (99.9%)\n",
      "  Read 2 with adapter:                  42,336 (99.2%)\n",
      "Pairs written (passing filters):        42,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,165,464 bp\n",
      "  Read 1:    12,582,642 bp\n",
      "  Read 2:    12,582,822 bp\n",
      "Total written (filtered):     23,508,675 bp (93.4%)\n",
      "  Read 1:    11,730,133 bp\n",
      "  Read 2:    11,778,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t42246\t0.0\t2\t41611 627 8\n",
      "21\t105\t0.0\t2\t0 104 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t42074\t0.0\t1\t39742 2332\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S895_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S895_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S895_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S895_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S895_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S895_85_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S895_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S895_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,959 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 5.04 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,959\n",
      "  Read 1 with adapter:                  61,920 (99.9%)\n",
      "  Read 2 with adapter:                  61,233 (98.8%)\n",
      "Pairs written (passing filters):        61,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,552,341 bp\n",
      "  Read 1:    18,276,018 bp\n",
      "  Read 2:    18,276,323 bp\n",
      "Total written (filtered):     34,150,865 bp (93.4%)\n",
      "  Read 1:    17,037,754 bp\n",
      "  Read 2:    17,113,111 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 61920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t319\t0.0\t1\t0 316 3\n",
      "20\t61409\t0.0\t2\t60441 953 15\n",
      "21\t141\t0.0\t2\t0 140 1\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 61233 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t324\t0.0\t1\t0 324\n",
      "19\t60800\t0.0\t1\t56916 3884\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S896_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S896_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S896_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S896_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S896_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S896_87_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S896_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S896_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,231 reads  @     79.0 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.45 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,231\n",
      "  Read 1 with adapter:                  43,200 (99.9%)\n",
      "  Read 2 with adapter:                  42,942 (99.3%)\n",
      "Pairs written (passing filters):        43,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,506,096 bp\n",
      "  Read 1:    12,753,060 bp\n",
      "  Read 2:    12,753,036 bp\n",
      "Total written (filtered):     23,826,513 bp (93.4%)\n",
      "  Read 1:    11,889,180 bp\n",
      "  Read 2:    11,937,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t244\t0.0\t1\t0 240 4\n",
      "20\t42814\t0.0\t2\t42101 701 12\n",
      "21\t100\t0.0\t2\t0 99 1\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t283\t0.0\t1\t0 283\n",
      "19\t42571\t0.0\t1\t40219 2352\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S897_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S897_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S897_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S897_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S897_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S897_89_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S897_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S897_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,464 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.86 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,464\n",
      "  Read 1 with adapter:                  46,428 (99.9%)\n",
      "  Read 2 with adapter:                  46,152 (99.3%)\n",
      "Pairs written (passing filters):        46,464 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,410,724 bp\n",
      "  Read 1:    13,706,003 bp\n",
      "  Read 2:    13,704,721 bp\n",
      "Total written (filtered):     25,605,533 bp (93.4%)\n",
      "  Read 1:    12,777,581 bp\n",
      "  Read 2:    12,827,952 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t261\t0.0\t1\t0 255 6\n",
      "20\t46014\t0.0\t2\t45248 759 7\n",
      "21\t103\t0.0\t2\t0 102 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t45851\t0.0\t1\t43280 2571\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S898_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S898_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S898_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S898_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S898_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S898_91_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S898_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S898_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,994 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.57 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,994\n",
      "  Read 1 with adapter:                  54,956 (99.9%)\n",
      "  Read 2 with adapter:                  54,569 (99.2%)\n",
      "Pairs written (passing filters):        54,994 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,435,677 bp\n",
      "  Read 1:    16,222,260 bp\n",
      "  Read 2:    16,213,417 bp\n",
      "Total written (filtered):     30,300,081 bp (93.4%)\n",
      "  Read 1:    15,123,317 bp\n",
      "  Read 2:    15,176,764 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t327\t0.0\t1\t0 324 3\n",
      "20\t54460\t0.0\t2\t53559 893 8\n",
      "21\t120\t0.0\t2\t0 115 5\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t258\t0.0\t1\t0 258\n",
      "19\t54211\t0.0\t1\t51350 2861\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S899_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S899_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S899_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S899_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S899_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S899_93_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S899_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S899_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,325 reads  @     87.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.73 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,325\n",
      "  Read 1 with adapter:                  42,295 (99.9%)\n",
      "  Read 2 with adapter:                  42,096 (99.5%)\n",
      "Pairs written (passing filters):        42,324 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,960,966 bp\n",
      "  Read 1:    12,485,548 bp\n",
      "  Read 2:    12,475,418 bp\n",
      "Total written (filtered):     23,315,186 bp (93.4%)\n",
      "  Read 1:    11,639,482 bp\n",
      "  Read 2:    11,675,704 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42295 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t223\t0.0\t1\t0 219 4\n",
      "20\t41928\t0.0\t2\t41254 665 9\n",
      "21\t114\t0.0\t2\t0 113 1\n",
      "22\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42096 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t41824\t0.0\t1\t39773 2051\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S900_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S900_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S900_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S900_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S900_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S900_95_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S900_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S900_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,679 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.32 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,679\n",
      "  Read 1 with adapter:                  51,645 (99.9%)\n",
      "  Read 2 with adapter:                  50,637 (98.0%)\n",
      "Pairs written (passing filters):        51,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,435,406 bp\n",
      "  Read 1:    15,244,441 bp\n",
      "  Read 2:    15,190,965 bp\n",
      "Total written (filtered):     28,440,678 bp (93.4%)\n",
      "  Read 1:    14,211,670 bp\n",
      "  Read 2:    14,229,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t260\t0.0\t1\t0 254 6\n",
      "20\t51224\t0.0\t2\t50326 880 18\n",
      "21\t123\t0.0\t2\t0 120 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t218\t0.0\t1\t0 218\n",
      "19\t50347\t0.0\t1\t48063 2284\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S901_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S901_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S901_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S901_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S901_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S901_97_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S901_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S901_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           178 reads  @    283.4 µs/read;   0.21 M reads/minute\n",
      "Finished in 0.05 s (306 us/read; 0.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                178\n",
      "  Read 1 with adapter:                     178 (100.0%)\n",
      "  Read 2 with adapter:                      13 (7.3%)\n",
      "Pairs written (passing filters):           178 (100.0%)\n",
      "\n",
      "Total basepairs processed:        60,423 bp\n",
      "  Read 1:        52,510 bp\n",
      "  Read 2:         7,913 bp\n",
      "Total written (filtered):         56,603 bp (93.7%)\n",
      "  Read 1:        48,950 bp\n",
      "  Read 2:         7,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 178 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t176\t0.0\t2\t175 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S902_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S902_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S902_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S902_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S902_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S902_99_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S902_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S902_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,957 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.19 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,957\n",
      "  Read 1 with adapter:                  53,928 (99.9%)\n",
      "  Read 2 with adapter:                  53,666 (99.5%)\n",
      "Pairs written (passing filters):        53,957 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,832,354 bp\n",
      "  Read 1:    15,917,037 bp\n",
      "  Read 2:    15,915,317 bp\n",
      "Total written (filtered):     29,734,528 bp (93.4%)\n",
      "  Read 1:    14,838,673 bp\n",
      "  Read 2:    14,895,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 53928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t334\t0.0\t1\t0 332 2\n",
      "20\t53449\t0.0\t2\t52267 1163 19\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53666 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t323\t0.0\t1\t0 323\n",
      "19\t53212\t0.0\t1\t50658 2554\n",
      "20\t131\t0.0\t1\t0 131\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S903_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S903_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S903_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S903_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S903_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S903_101_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S903_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S903_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        11,017 reads  @     78.3 µs/read;   0.77 M reads/minute\n",
      "Finished in 0.87 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,017\n",
      "  Read 1 with adapter:                  11,013 (100.0%)\n",
      "  Read 2 with adapter:                  10,855 (98.5%)\n",
      "Pairs written (passing filters):        11,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,499,798 bp\n",
      "  Read 1:     3,249,788 bp\n",
      "  Read 2:     3,250,010 bp\n",
      "Total written (filtered):      6,073,336 bp (93.4%)\n",
      "  Read 1:     3,029,532 bp\n",
      "  Read 2:     3,043,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 11013 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t63\t0.0\t1\t0 62 1\n",
      "20\t10907\t0.0\t2\t10601 303 3\n",
      "21\t15\t0.0\t2\t0 15\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 10855 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t69\t0.0\t1\t0 69\n",
      "19\t10756\t0.0\t1\t9999 757\n",
      "20\t30\t0.0\t1\t0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S904_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S904_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S904_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S904_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S904_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S904_103_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S904_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S904_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,184 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.06 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,184\n",
      "  Read 1 with adapter:                  41,150 (99.9%)\n",
      "  Read 2 with adapter:                  40,988 (99.5%)\n",
      "Pairs written (passing filters):        41,184 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,297,843 bp\n",
      "  Read 1:    12,149,231 bp\n",
      "  Read 2:    12,148,612 bp\n",
      "Total written (filtered):     22,696,352 bp (93.4%)\n",
      "  Read 1:    11,326,381 bp\n",
      "  Read 2:    11,369,971 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41150 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t276\t0.0\t1\t0 272 4\n",
      "20\t40744\t0.0\t2\t40025 708 11\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t199\t0.0\t1\t0 199\n",
      "19\t40721\t0.0\t1\t38881 1840\n",
      "20\t68\t0.0\t1\t0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S905_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S905_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S905_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S905_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S905_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S905_105_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S905_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S905_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,654 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.61 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,654\n",
      "  Read 1 with adapter:                  49,628 (99.9%)\n",
      "  Read 2 with adapter:                  49,423 (99.5%)\n",
      "Pairs written (passing filters):        49,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,295,376 bp\n",
      "  Read 1:    14,647,907 bp\n",
      "  Read 2:    14,647,469 bp\n",
      "Total written (filtered):     27,364,093 bp (93.4%)\n",
      "  Read 1:    13,655,534 bp\n",
      "  Read 2:    13,708,559 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t333\t0.0\t1\t0 326 7\n",
      "20\t49130\t0.0\t2\t48228 890 12\n",
      "21\t112\t0.0\t2\t0 112\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 49423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t215\t0.0\t1\t0 215\n",
      "19\t49120\t0.0\t1\t46853 2267\n",
      "20\t88\t0.0\t1\t0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S906_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S906_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S906_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S906_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S906_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S906_107_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S906_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S906_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,862 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.33 s (69 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,862\n",
      "  Read 1 with adapter:                  47,835 (99.9%)\n",
      "  Read 2 with adapter:                  47,645 (99.5%)\n",
      "Pairs written (passing filters):        47,862 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,237,752 bp\n",
      "  Read 1:    14,119,243 bp\n",
      "  Read 2:    14,118,509 bp\n",
      "Total written (filtered):     26,376,133 bp (93.4%)\n",
      "  Read 1:    13,162,744 bp\n",
      "  Read 2:    13,213,389 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47835 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t322\t0.0\t1\t0 319 3\n",
      "20\t47365\t0.0\t2\t46531 821 13\n",
      "21\t107\t0.0\t2\t0 103 4\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t47314\t0.0\t1\t45074 2240\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S907_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S907_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S907_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S907_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S907_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S907_109_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S907_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S907_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        68,049 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.94 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,049\n",
      "  Read 1 with adapter:                  68,011 (99.9%)\n",
      "  Read 2 with adapter:                  67,738 (99.5%)\n",
      "Pairs written (passing filters):        68,049 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,148,612 bp\n",
      "  Read 1:    20,074,206 bp\n",
      "  Read 2:    20,074,406 bp\n",
      "Total written (filtered):     37,501,867 bp (93.4%)\n",
      "  Read 1:    18,714,240 bp\n",
      "  Read 2:    18,787,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 68011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t459\t0.0\t1\t0 456 3\n",
      "20\t67346\t0.0\t2\t66188 1135 23\n",
      "21\t147\t0.0\t2\t0 146 1\n",
      "22\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 67738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t386\t0.0\t1\t0 386\n",
      "19\t67209\t0.0\t1\t64052 3157\n",
      "20\t143\t0.0\t1\t0 143\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S908_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S908_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S908_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S908_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S908_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S908_111_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S908_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S908_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,397 reads  @     75.6 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.84 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,397\n",
      "  Read 1 with adapter:                  50,363 (99.9%)\n",
      "  Read 2 with adapter:                  50,142 (99.5%)\n",
      "Pairs written (passing filters):        50,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,733,428 bp\n",
      "  Read 1:    14,867,102 bp\n",
      "  Read 2:    14,866,326 bp\n",
      "Total written (filtered):     27,773,875 bp (93.4%)\n",
      "  Read 1:    13,860,094 bp\n",
      "  Read 2:    13,913,781 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 50363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t26\t0.0\t1\t0 0 26\n",
      "19\t367\t0.0\t1\t0 358 9\n",
      "20\t49839\t0.0\t2\t48968 851 20\n",
      "21\t95\t0.0\t2\t0 90 5\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t254\t0.0\t1\t0 254\n",
      "19\t49787\t0.0\t1\t47501 2286\n",
      "20\t101\t0.0\t1\t0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S909_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S909_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S909_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S909_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S909_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S909_113_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S909_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S909_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        93,108 reads  @     73.9 µs/read;   0.81 M reads/minute\n",
      "Finished in 6.89 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,108\n",
      "  Read 1 with adapter:                  93,040 (99.9%)\n",
      "  Read 2 with adapter:                  92,680 (99.5%)\n",
      "Pairs written (passing filters):        93,107 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,916,871 bp\n",
      "  Read 1:    27,466,829 bp\n",
      "  Read 2:    27,450,042 bp\n",
      "Total written (filtered):     51,295,441 bp (93.4%)\n",
      "  Read 1:    25,606,089 bp\n",
      "  Read 2:    25,689,352 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 93040 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t35\t0.0\t1\t0 0 35\n",
      "19\t617\t0.0\t1\t0 610 7\n",
      "20\t92105\t0.0\t2\t90440 1634 31\n",
      "21\t214\t0.0\t2\t0 212 2\n",
      "22\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 92680 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t405\t0.0\t1\t0 405\n",
      "19\t92100\t0.0\t1\t88042 4058\n",
      "20\t175\t0.0\t1\t0 175\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S910_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S910_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S910_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S910_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S910_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S910_115_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S910_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S910_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,178 reads  @     81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.45 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,178\n",
      "  Read 1 with adapter:                  42,144 (99.9%)\n",
      "  Read 2 with adapter:                  42,017 (99.6%)\n",
      "Pairs written (passing filters):        42,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,877,252 bp\n",
      "  Read 1:    12,442,489 bp\n",
      "  Read 2:    12,434,763 bp\n",
      "Total written (filtered):     23,236,333 bp (93.4%)\n",
      "  Read 1:    11,599,775 bp\n",
      "  Read 2:    11,636,558 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42144 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t271\t0.0\t1\t0 263 8\n",
      "20\t41761\t0.0\t2\t40996 757 8\n",
      "21\t79\t0.0\t2\t0 78 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t41731\t0.0\t1\t39989 1742\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S911_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S911_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S911_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S911_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S911_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S911_117_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S911_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S911_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,298 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.77 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,298\n",
      "  Read 1 with adapter:                  51,272 (99.9%)\n",
      "  Read 2 with adapter:                  50,655 (98.7%)\n",
      "Pairs written (passing filters):        51,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,230,895 bp\n",
      "  Read 1:    15,132,897 bp\n",
      "  Read 2:    15,097,998 bp\n",
      "Total written (filtered):     28,243,303 bp (93.4%)\n",
      "  Read 1:    14,107,630 bp\n",
      "  Read 2:    14,135,673 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51272 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t324\t0.0\t1\t0 320 4\n",
      "20\t50785\t0.0\t2\t49877 895 13\n",
      "21\t119\t0.0\t2\t0 116 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t207\t0.0\t1\t0 207\n",
      "19\t50361\t0.0\t1\t48284 2077\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S912_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S912_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S912_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S912_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S912_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S912_119_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S912_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S912_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,105 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 5.94 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,105\n",
      "  Read 1 with adapter:                  80,064 (99.9%)\n",
      "  Read 2 with adapter:                  79,491 (99.2%)\n",
      "Pairs written (passing filters):        80,105 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,260,029 bp\n",
      "  Read 1:    23,630,249 bp\n",
      "  Read 2:    23,629,780 bp\n",
      "Total written (filtered):     44,149,186 bp (93.4%)\n",
      "  Read 1:    22,029,515 bp\n",
      "  Read 2:    22,119,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t22\t0.0\t1\t0 0 22\n",
      "19\t721\t0.0\t1\t0 713 8\n",
      "20\t79144\t0.0\t2\t77842 1283 19\n",
      "21\t135\t0.0\t2\t0 133 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 79491 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t375\t0.0\t1\t0 375\n",
      "19\t78961\t0.0\t1\t74799 4162\n",
      "20\t155\t0.0\t1\t0 155\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S913_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S913_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S913_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S913_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S913_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S913_121_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S913_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S913_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        46,954 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.67 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,954\n",
      "  Read 1 with adapter:                  46,929 (99.9%)\n",
      "  Read 2 with adapter:                  46,497 (99.0%)\n",
      "Pairs written (passing filters):        46,954 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,701,141 bp\n",
      "  Read 1:    13,850,918 bp\n",
      "  Read 2:    13,850,223 bp\n",
      "Total written (filtered):     25,879,597 bp (93.4%)\n",
      "  Read 1:    12,912,653 bp\n",
      "  Read 2:    12,966,944 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t430\t0.0\t1\t0 424 6\n",
      "20\t46386\t0.0\t2\t45630 741 15\n",
      "21\t83\t0.0\t2\t0 82 1\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46497 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t235\t0.0\t1\t0 235\n",
      "19\t46191\t0.0\t1\t43341 2850\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S914_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S914_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S914_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S914_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S914_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S914_123_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S914_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S914_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,452 reads  @     83.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.10 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,452\n",
      "  Read 1 with adapter:                  48,430 (100.0%)\n",
      "  Read 2 with adapter:                  48,018 (99.1%)\n",
      "Pairs written (passing filters):        48,452 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,582,760 bp\n",
      "  Read 1:    14,292,892 bp\n",
      "  Read 2:    14,289,868 bp\n",
      "Total written (filtered):     26,702,250 bp (93.4%)\n",
      "  Read 1:    13,324,599 bp\n",
      "  Read 2:    13,377,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t406\t0.0\t1\t0 402 4\n",
      "20\t47920\t0.0\t2\t47104 808 8\n",
      "21\t73\t0.0\t2\t0 73\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48018 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t212\t0.0\t1\t0 212\n",
      "19\t47719\t0.0\t1\t44921 2798\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S915_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S915_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S915_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S915_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S915_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S915_125_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S915_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S915_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,816 reads  @     69.1 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.12 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,816\n",
      "  Read 1 with adapter:                  44,806 (100.0%)\n",
      "  Read 2 with adapter:                  44,445 (99.2%)\n",
      "Pairs written (passing filters):        44,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,441,406 bp\n",
      "  Read 1:    13,220,713 bp\n",
      "  Read 2:    13,220,693 bp\n",
      "Total written (filtered):     24,701,330 bp (93.4%)\n",
      "  Read 1:    12,324,919 bp\n",
      "  Read 2:    12,376,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t409\t0.0\t1\t0 403 6\n",
      "20\t44283\t0.0\t2\t43530 748 5\n",
      "21\t77\t0.0\t2\t0 76 1\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44445 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t260\t0.0\t1\t0 260\n",
      "19\t44098\t0.0\t1\t41447 2651\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S916_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S916_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S916_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S916_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S916_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S916_127_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S916_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S916_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,887 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.97 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,887\n",
      "  Read 1 with adapter:                  52,857 (99.9%)\n",
      "  Read 2 with adapter:                  52,403 (99.1%)\n",
      "Pairs written (passing filters):        52,886 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,200,445 bp\n",
      "  Read 1:    15,600,763 bp\n",
      "  Read 2:    15,599,682 bp\n",
      "Total written (filtered):     29,147,942 bp (93.4%)\n",
      "  Read 1:    14,543,749 bp\n",
      "  Read 2:    14,604,193 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52857 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t23\t0.0\t1\t0 0 23\n",
      "19\t511\t0.0\t1\t0 503 8\n",
      "20\t52186\t0.0\t2\t51367 809 10\n",
      "21\t118\t0.0\t2\t0 115 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t259\t0.0\t1\t0 259\n",
      "19\t52053\t0.0\t1\t48660 3393\n",
      "20\t91\t0.0\t1\t0 91\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S917_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S917_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S917_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S917_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S917_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S917_129_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S917_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S917_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        84,986 reads  @     74.9 µs/read;   0.80 M reads/minute\n",
      "Finished in 6.39 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,986\n",
      "  Read 1 with adapter:                  84,939 (99.9%)\n",
      "  Read 2 with adapter:                  83,829 (98.6%)\n",
      "Pairs written (passing filters):        84,986 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,113,912 bp\n",
      "  Read 1:    25,069,709 bp\n",
      "  Read 2:    25,044,203 bp\n",
      "Total written (filtered):     46,823,196 bp (93.4%)\n",
      "  Read 1:    23,371,540 bp\n",
      "  Read 2:    23,451,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 84939 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t806\t0.0\t1\t0 790 16\n",
      "20\t83906\t0.0\t2\t82406 1480 20\n",
      "21\t147\t0.0\t2\t0 146 1\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 83829 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t361\t0.0\t1\t0 361\n",
      "19\t83311\t0.0\t1\t77729 5582\n",
      "20\t157\t0.0\t1\t0 157\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S918_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S918_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S918_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S918_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S918_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S918_131_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S918_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S918_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,497 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.83 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,497\n",
      "  Read 1 with adapter:                  52,470 (99.9%)\n",
      "  Read 2 with adapter:                  52,122 (99.3%)\n",
      "Pairs written (passing filters):        52,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,961,606 bp\n",
      "  Read 1:    15,486,424 bp\n",
      "  Read 2:    15,475,182 bp\n",
      "Total written (filtered):     28,922,418 bp (93.4%)\n",
      "  Read 1:    14,437,429 bp\n",
      "  Read 2:    14,484,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52470 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t492\t0.0\t1\t0 489 3\n",
      "20\t51834\t0.0\t2\t50900 918 16\n",
      "21\t101\t0.0\t2\t0 101\n",
      "22\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t242\t0.0\t1\t0 242\n",
      "19\t51763\t0.0\t1\t49135 2628\n",
      "20\t117\t0.0\t1\t0 117\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S919_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S919_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S919_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S919_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S919_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S919_133_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S919_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S919_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        43,229 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.61 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,229\n",
      "  Read 1 with adapter:                  43,214 (100.0%)\n",
      "  Read 2 with adapter:                  42,188 (97.6%)\n",
      "Pairs written (passing filters):        43,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,453,323 bp\n",
      "  Read 1:    12,752,191 bp\n",
      "  Read 2:    12,701,132 bp\n",
      "Total written (filtered):     23,787,881 bp (93.5%)\n",
      "  Read 1:    11,888,207 bp\n",
      "  Read 2:    11,899,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43214 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t386\t0.0\t1\t0 383 3\n",
      "20\t42710\t0.0\t2\t41974 730 6\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42188 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t183\t0.0\t1\t0 183\n",
      "19\t41936\t0.0\t1\t39815 2121\n",
      "20\t69\t0.0\t1\t0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S920_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S920_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S920_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S920_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S920_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S920_135_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S920_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S920_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,657 reads  @     77.3 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.33 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,657\n",
      "  Read 1 with adapter:                  42,628 (99.9%)\n",
      "  Read 2 with adapter:                  42,473 (99.6%)\n",
      "Pairs written (passing filters):        42,657 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,166,367 bp\n",
      "  Read 1:    12,583,306 bp\n",
      "  Read 2:    12,583,061 bp\n",
      "Total written (filtered):     23,507,107 bp (93.4%)\n",
      "  Read 1:    11,730,938 bp\n",
      "  Read 2:    11,776,169 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t289\t0.0\t1\t0 285 4\n",
      "20\t42217\t0.0\t2\t41553 656 8\n",
      "21\t87\t0.0\t2\t0 85 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42473 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t173\t0.0\t1\t0 173\n",
      "19\t42222\t0.0\t1\t40286 1936\n",
      "20\t78\t0.0\t1\t0 78\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S921_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S921_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S921_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S921_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S921_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S921_137_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S921_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S921_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,570 reads  @     74.4 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.49 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,570\n",
      "  Read 1 with adapter:                  46,531 (99.9%)\n",
      "  Read 2 with adapter:                  46,354 (99.5%)\n",
      "Pairs written (passing filters):        46,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,474,945 bp\n",
      "  Read 1:    13,737,253 bp\n",
      "  Read 2:    13,737,692 bp\n",
      "Total written (filtered):     25,663,931 bp (93.4%)\n",
      "  Read 1:    12,806,821 bp\n",
      "  Read 2:    12,857,110 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t280\t0.0\t1\t0 274 6\n",
      "20\t46128\t0.0\t2\t45369 748 11\n",
      "21\t74\t0.0\t2\t0 73 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46354 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t46056\t0.0\t1\t43934 2122\n",
      "20\t77\t0.0\t1\t0 77\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S922_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S922_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S922_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S922_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S922_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S922_139_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S922_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S922_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        44,250 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.00 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,250\n",
      "  Read 1 with adapter:                  44,213 (99.9%)\n",
      "  Read 2 with adapter:                  44,059 (99.6%)\n",
      "Pairs written (passing filters):        44,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,107,096 bp\n",
      "  Read 1:    13,053,394 bp\n",
      "  Read 2:    13,053,702 bp\n",
      "Total written (filtered):     24,386,049 bp (93.4%)\n",
      "  Read 1:    12,169,365 bp\n",
      "  Read 2:    12,216,684 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44213 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t320\t0.0\t1\t0 311 9\n",
      "20\t43781\t0.0\t2\t43018 752 11\n",
      "21\t71\t0.0\t2\t0 69 2\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t193\t0.0\t1\t0 193\n",
      "19\t43776\t0.0\t1\t41741 2035\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S923_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S923_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S923_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S923_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S923_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S923_141_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S923_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S923_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,788 reads  @     68.6 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.68 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,788\n",
      "  Read 1 with adapter:                  38,768 (99.9%)\n",
      "  Read 2 with adapter:                  38,622 (99.6%)\n",
      "Pairs written (passing filters):        38,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,884,739 bp\n",
      "  Read 1:    11,442,394 bp\n",
      "  Read 2:    11,442,345 bp\n",
      "Total written (filtered):     21,375,903 bp (93.4%)\n",
      "  Read 1:    10,667,184 bp\n",
      "  Read 2:    10,708,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t231\t0.0\t1\t0 226 5\n",
      "20\t38429\t0.0\t2\t37779 645 5\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38622 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t265\t0.0\t1\t0 265\n",
      "19\t38284\t0.0\t1\t36542 1742\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S924_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S924_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S924_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S924_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S924_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S924_143_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S924_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S924_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,659 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.87 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,659\n",
      "  Read 1 with adapter:                  40,630 (99.9%)\n",
      "  Read 2 with adapter:                  40,493 (99.6%)\n",
      "Pairs written (passing filters):        40,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,987,753 bp\n",
      "  Read 1:    11,994,244 bp\n",
      "  Read 2:    11,993,509 bp\n",
      "Total written (filtered):     22,406,079 bp (93.4%)\n",
      "  Read 1:    11,181,806 bp\n",
      "  Read 2:    11,224,273 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t239\t0.0\t1\t0 235 4\n",
      "20\t40275\t0.0\t2\t39579 686 10\n",
      "21\t75\t0.0\t2\t0 74 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t40218\t0.0\t1\t38419 1799\n",
      "20\t72\t0.0\t1\t0 72\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S925_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S925_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S925_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S925_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S925_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S925_145_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S925_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S925_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        44,441 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.15 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,441\n",
      "  Read 1 with adapter:                  44,406 (99.9%)\n",
      "  Read 2 with adapter:                  44,215 (99.5%)\n",
      "Pairs written (passing filters):        44,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,214,200 bp\n",
      "  Read 1:    13,109,766 bp\n",
      "  Read 2:    13,104,434 bp\n",
      "Total written (filtered):     24,486,304 bp (93.4%)\n",
      "  Read 1:    12,221,830 bp\n",
      "  Read 2:    12,264,474 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t296\t0.0\t1\t0 290 6\n",
      "20\t43976\t0.0\t2\t43182 783 11\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t201\t0.0\t1\t0 201\n",
      "19\t43938\t0.0\t1\t42059 1879\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S926_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S926_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S926_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S926_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S926_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S926_147_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S926_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S926_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,807 reads  @     72.2 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.48 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,807\n",
      "  Read 1 with adapter:                  47,763 (99.9%)\n",
      "  Read 2 with adapter:                  47,590 (99.5%)\n",
      "Pairs written (passing filters):        47,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,197,833 bp\n",
      "  Read 1:    14,102,439 bp\n",
      "  Read 2:    14,095,394 bp\n",
      "Total written (filtered):     26,338,743 bp (93.4%)\n",
      "  Read 1:    13,147,424 bp\n",
      "  Read 2:    13,191,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t27\t0.0\t1\t0 0 27\n",
      "19\t319\t0.0\t1\t0 312 7\n",
      "20\t47312\t0.0\t2\t46432 860 20\n",
      "21\t82\t0.0\t2\t0 82\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t47271\t0.0\t1\t45217 2054\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S927_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S927_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S927_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S927_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S927_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S927_149_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S927_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S927_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,117 reads  @     71.0 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.38 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,117\n",
      "  Read 1 with adapter:                  47,090 (99.9%)\n",
      "  Read 2 with adapter:                  46,571 (98.8%)\n",
      "Pairs written (passing filters):        47,117 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,775,252 bp\n",
      "  Read 1:    13,899,242 bp\n",
      "  Read 2:    13,876,010 bp\n",
      "Total written (filtered):     25,948,927 bp (93.4%)\n",
      "  Read 1:    12,957,625 bp\n",
      "  Read 2:    12,991,302 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47090 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t298\t0.0\t1\t0 292 6\n",
      "20\t46650\t0.0\t2\t45836 796 18\n",
      "21\t93\t0.0\t2\t0 91 2\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46571 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t214\t0.0\t1\t0 214\n",
      "19\t46284\t0.0\t1\t44228 2056\n",
      "20\t73\t0.0\t1\t0 73\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S928_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S928_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S928_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S928_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S928_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S928_151_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S928_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S928_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,229 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.38 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,229\n",
      "  Read 1 with adapter:                  44,205 (99.9%)\n",
      "  Read 2 with adapter:                  43,951 (99.4%)\n",
      "Pairs written (passing filters):        44,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,094,190 bp\n",
      "  Read 1:    13,047,347 bp\n",
      "  Read 2:    13,046,843 bp\n",
      "Total written (filtered):     24,375,257 bp (93.4%)\n",
      "  Read 1:    12,163,379 bp\n",
      "  Read 2:    12,211,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t241\t0.0\t1\t0 239 2\n",
      "20\t43842\t0.0\t2\t43041 786 15\n",
      "21\t75\t0.0\t2\t0 72 3\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43951 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t211\t0.0\t1\t0 211\n",
      "19\t43633\t0.0\t1\t41521 2112\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S929_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S929_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S929_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S929_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S929_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S929_153_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S929_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S929_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,019 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.00 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,019\n",
      "  Read 1 with adapter:                  42,994 (99.9%)\n",
      "  Read 2 with adapter:                  42,756 (99.4%)\n",
      "Pairs written (passing filters):        43,019 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,380,099 bp\n",
      "  Read 1:    12,690,329 bp\n",
      "  Read 2:    12,689,770 bp\n",
      "Total written (filtered):     23,708,107 bp (93.4%)\n",
      "  Read 1:    11,830,592 bp\n",
      "  Read 2:    11,877,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t274\t0.0\t1\t0 271 3\n",
      "20\t42576\t0.0\t2\t41903 667 6\n",
      "21\t101\t0.0\t2\t0 101\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42756 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t42487\t0.0\t1\t40423 2064\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S930_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S930_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S930_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S930_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S930_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S930_155_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S930_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S930_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,014 reads  @     67.2 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.72 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,014\n",
      "  Read 1 with adapter:                  39,994 (100.0%)\n",
      "  Read 2 with adapter:                  39,776 (99.4%)\n",
      "Pairs written (passing filters):        40,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,607,471 bp\n",
      "  Read 1:    11,804,121 bp\n",
      "  Read 2:    11,803,350 bp\n",
      "Total written (filtered):     22,052,071 bp (93.4%)\n",
      "  Read 1:    11,004,346 bp\n",
      "  Read 2:    11,047,725 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t214\t0.0\t1\t0 208 6\n",
      "20\t39674\t0.0\t2\t39016 652 6\n",
      "21\t67\t0.0\t2\t0 66 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t39517\t0.0\t1\t37600 1917\n",
      "20\t70\t0.0\t1\t0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S931_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S931_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S931_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S931_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S931_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S931_157_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S931_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S931_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        39,197 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.76 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,197\n",
      "  Read 1 with adapter:                  39,181 (100.0%)\n",
      "  Read 2 with adapter:                  38,974 (99.4%)\n",
      "Pairs written (passing filters):        39,197 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,125,737 bp\n",
      "  Read 1:    11,562,765 bp\n",
      "  Read 2:    11,562,972 bp\n",
      "Total written (filtered):     21,601,901 bp (93.4%)\n",
      "  Read 1:    10,779,261 bp\n",
      "  Read 2:    10,822,640 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 39181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t218\t0.0\t1\t0 214 4\n",
      "20\t38857\t0.0\t2\t38233 609 15\n",
      "21\t74\t0.0\t2\t0 72 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t254\t0.0\t1\t0 254\n",
      "19\t38640\t0.0\t1\t36615 2025\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S932_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S932_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S932_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S932_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S932_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S932_159_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S932_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S932_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        66,684 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.81 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,684\n",
      "  Read 1 with adapter:                  66,651 (100.0%)\n",
      "  Read 2 with adapter:                  66,279 (99.4%)\n",
      "Pairs written (passing filters):        66,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,339,641 bp\n",
      "  Read 1:    19,671,456 bp\n",
      "  Read 2:    19,668,185 bp\n",
      "Total written (filtered):     36,747,494 bp (93.4%)\n",
      "  Read 1:    18,338,435 bp\n",
      "  Read 2:    18,409,059 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 66651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t28\t0.0\t1\t0 0 28\n",
      "19\t406\t0.0\t1\t0 400 6\n",
      "20\t66065\t0.0\t2\t65002 1046 17\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 66279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t305\t0.0\t1\t0 305\n",
      "19\t65844\t0.0\t1\t62465 3379\n",
      "20\t130\t0.0\t1\t0 130\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S933_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S933_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S933_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S933_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S933_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S933_161_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S933_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S933_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,455 reads  @     71.0 µs/read;   0.84 M reads/minute\n",
      "Finished in 5.80 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,455\n",
      "  Read 1 with adapter:                  81,416 (100.0%)\n",
      "  Read 2 with adapter:                  80,823 (99.2%)\n",
      "Pairs written (passing filters):        81,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,036,073 bp\n",
      "  Read 1:    24,029,207 bp\n",
      "  Read 2:    24,006,866 bp\n",
      "Total written (filtered):     44,872,552 bp (93.4%)\n",
      "  Read 1:    22,401,151 bp\n",
      "  Read 2:    22,471,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t479\t0.0\t1\t0 473 6\n",
      "20\t80724\t0.0\t2\t79336 1364 24\n",
      "21\t131\t0.0\t2\t0 130 1\n",
      "22\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t340\t0.0\t1\t0 340\n",
      "19\t80315\t0.0\t1\t76443 3872\n",
      "20\t168\t0.0\t1\t0 168\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S934_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S934_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S934_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S934_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S934_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S934_163_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S934_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S934_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,407 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 2.89 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,407\n",
      "  Read 1 with adapter:                  42,376 (99.9%)\n",
      "  Read 2 with adapter:                  42,175 (99.5%)\n",
      "Pairs written (passing filters):        42,406 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,009,270 bp\n",
      "  Read 1:    12,509,622 bp\n",
      "  Read 2:    12,499,648 bp\n",
      "Total written (filtered):     23,360,358 bp (93.4%)\n",
      "  Read 1:    11,661,938 bp\n",
      "  Read 2:    11,698,420 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t211\t0.0\t1\t0 207 4\n",
      "20\t42051\t0.0\t2\t41344 698 9\n",
      "21\t80\t0.0\t2\t0 79 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42175 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t182\t0.0\t1\t0 182\n",
      "19\t41908\t0.0\t1\t40004 1904\n",
      "20\t85\t0.0\t1\t0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S935_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S935_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S935_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S935_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S935_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S935_165_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S935_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S935_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,823 reads  @     74.1 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.13 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,823\n",
      "  Read 1 with adapter:                  41,810 (100.0%)\n",
      "  Read 2 with adapter:                  41,083 (98.2%)\n",
      "Pairs written (passing filters):        41,823 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,638,835 bp\n",
      "  Read 1:    12,337,676 bp\n",
      "  Read 2:    12,301,159 bp\n",
      "Total written (filtered):     23,022,298 bp (93.4%)\n",
      "  Read 1:    11,501,591 bp\n",
      "  Read 2:    11,520,707 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41810 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t227\t0.0\t1\t0 227\n",
      "20\t41454\t0.0\t2\t40755 690 9\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41083 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t189\t0.0\t1\t0 189\n",
      "19\t40830\t0.0\t1\t39029 1801\n",
      "20\t64\t0.0\t1\t0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S936_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S936_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S936_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S936_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S936_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S936_167_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S936_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S936_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        42,009 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 2.97 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,009\n",
      "  Read 1 with adapter:                  41,985 (99.9%)\n",
      "  Read 2 with adapter:                  41,670 (99.2%)\n",
      "Pairs written (passing filters):        42,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,780,180 bp\n",
      "  Read 1:    12,390,676 bp\n",
      "  Read 2:    12,389,504 bp\n",
      "Total written (filtered):     23,149,047 bp (93.4%)\n",
      "  Read 1:    11,551,142 bp\n",
      "  Read 2:    11,597,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41985 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t262\t0.0\t1\t0 259 3\n",
      "20\t41617\t0.0\t2\t40925 686 6\n",
      "21\t72\t0.0\t2\t0 72\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 41670 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t202\t0.0\t1\t0 202\n",
      "19\t41397\t0.0\t1\t39337 2060\n",
      "20\t71\t0.0\t1\t0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S937_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S937_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S937_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S937_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S937_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S937_169_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S937_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S937_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        48,022 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.39 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,022\n",
      "  Read 1 with adapter:                  47,982 (99.9%)\n",
      "  Read 2 with adapter:                  47,695 (99.3%)\n",
      "Pairs written (passing filters):        48,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,329,962 bp\n",
      "  Read 1:    14,164,124 bp\n",
      "  Read 2:    14,165,838 bp\n",
      "Total written (filtered):     26,464,447 bp (93.4%)\n",
      "  Read 1:    13,204,679 bp\n",
      "  Read 2:    13,259,768 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t301\t0.0\t1\t0 297 4\n",
      "20\t47545\t0.0\t2\t46734 797 14\n",
      "21\t90\t0.0\t2\t0 88 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47695 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t47392\t0.0\t1\t44709 2683\n",
      "20\t84\t0.0\t1\t0 84\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S938_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S938_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S938_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S938_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S938_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S938_171_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S938_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S938_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,281 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.86 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,281\n",
      "  Read 1 with adapter:                  51,247 (99.9%)\n",
      "  Read 2 with adapter:                  50,820 (99.1%)\n",
      "Pairs written (passing filters):        51,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,252,214 bp\n",
      "  Read 1:    15,126,470 bp\n",
      "  Read 2:    15,125,744 bp\n",
      "Total written (filtered):     28,262,011 bp (93.4%)\n",
      "  Read 1:    14,101,706 bp\n",
      "  Read 2:    14,160,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51247 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t341\t0.0\t1\t0 338 3\n",
      "20\t50744\t0.0\t2\t49880 855 9\n",
      "21\t107\t0.0\t2\t0 105 2\n",
      "22\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t239\t0.0\t1\t0 239\n",
      "19\t50483\t0.0\t1\t47689 2794\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S939_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S939_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S939_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S939_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S939_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S939_173_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S939_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S939_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,458 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.64 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,458\n",
      "  Read 1 with adapter:                  45,411 (99.9%)\n",
      "  Read 2 with adapter:                  45,102 (99.2%)\n",
      "Pairs written (passing filters):        45,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,815,314 bp\n",
      "  Read 1:    13,405,607 bp\n",
      "  Read 2:    13,409,707 bp\n",
      "Total written (filtered):     25,050,481 bp (93.4%)\n",
      "  Read 1:    12,497,512 bp\n",
      "  Read 2:    12,552,969 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45411 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t263\t0.0\t1\t0 262 1\n",
      "20\t45011\t0.0\t2\t44270 730 11\n",
      "21\t88\t0.0\t2\t0 87 1\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t300\t0.0\t1\t0 300\n",
      "19\t44702\t0.0\t1\t42158 2544\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S940_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S940_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S940_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S940_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S940_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S940_175_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S940_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S940_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        96,159 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 7.84 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             96,159\n",
      "  Read 1 with adapter:                  95,954 (99.8%)\n",
      "  Read 2 with adapter:                  95,388 (99.2%)\n",
      "Pairs written (passing filters):        96,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    56,691,814 bp\n",
      "  Read 1:    28,332,248 bp\n",
      "  Read 2:    28,359,566 bp\n",
      "Total written (filtered):     52,961,049 bp (93.4%)\n",
      "  Read 1:    26,413,542 bp\n",
      "  Read 2:    26,547,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 95954 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t39\t0.0\t1\t0 0 39\n",
      "19\t602\t0.0\t1\t0 590 12\n",
      "20\t95070\t0.0\t2\t93410 1640 20\n",
      "21\t180\t0.0\t2\t0 177 3\n",
      "22\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 95388 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t482\t0.0\t1\t0 482\n",
      "19\t94737\t0.0\t1\t89177 5560\n",
      "20\t169\t0.0\t1\t0 169\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S941_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S941_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S941_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S941_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S941_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S941_177_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S941_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S941_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,796 reads  @     74.5 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.12 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,796\n",
      "  Read 1 with adapter:                  54,739 (99.9%)\n",
      "  Read 2 with adapter:                  54,410 (99.3%)\n",
      "Pairs written (passing filters):        54,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,314,001 bp\n",
      "  Read 1:    16,159,165 bp\n",
      "  Read 2:    16,154,836 bp\n",
      "Total written (filtered):     30,185,776 bp (93.4%)\n",
      "  Read 1:    15,064,593 bp\n",
      "  Read 2:    15,121,183 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t17\t0.0\t1\t0 0 17\n",
      "19\t342\t0.0\t1\t0 335 7\n",
      "20\t54243\t0.0\t2\t53369 862 12\n",
      "21\t106\t0.0\t2\t0 105 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 54410 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t229\t0.0\t1\t0 229\n",
      "19\t54089\t0.0\t1\t51252 2837\n",
      "20\t92\t0.0\t1\t0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S942_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S942_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S942_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S942_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S942_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S942_179_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S942_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S942_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,630 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.81 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,630\n",
      "  Read 1 with adapter:                  47,589 (99.9%)\n",
      "  Read 2 with adapter:                  47,330 (99.4%)\n",
      "Pairs written (passing filters):        47,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,091,541 bp\n",
      "  Read 1:    14,049,171 bp\n",
      "  Read 2:    14,042,370 bp\n",
      "Total written (filtered):     26,240,799 bp (93.4%)\n",
      "  Read 1:    13,097,585 bp\n",
      "  Read 2:    13,143,214 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t296\t0.0\t1\t0 289 7\n",
      "20\t47173\t0.0\t2\t46388 772 13\n",
      "21\t86\t0.0\t2\t0 85 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47330 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t47026\t0.0\t1\t44711 2315\n",
      "20\t95\t0.0\t1\t0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S943_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S943_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S943_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S943_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S943_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S943_181_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S943_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S943_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        45,781 reads  @     81.5 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.77 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,781\n",
      "  Read 1 with adapter:                  45,749 (99.9%)\n",
      "  Read 2 with adapter:                  44,965 (98.2%)\n",
      "Pairs written (passing filters):        45,781 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,969,324 bp\n",
      "  Read 1:    13,503,763 bp\n",
      "  Read 2:    13,465,561 bp\n",
      "Total written (filtered):     25,200,296 bp (93.4%)\n",
      "  Read 1:    12,588,965 bp\n",
      "  Read 2:    12,611,331 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45749 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t302\t0.0\t1\t0 299 3\n",
      "20\t45300\t0.0\t2\t44524 766 10\n",
      "21\t98\t0.0\t2\t0 97 1\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t191\t0.0\t1\t0 191\n",
      "19\t44688\t0.0\t1\t42677 2011\n",
      "20\t86\t0.0\t1\t0 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S944_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S944_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S944_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S944_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S944_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S944_183_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S944_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S944_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,070 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.30 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,070\n",
      "  Read 1 with adapter:                  46,963 (99.8%)\n",
      "  Read 2 with adapter:                  46,675 (99.2%)\n",
      "Pairs written (passing filters):        47,070 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,759,605 bp\n",
      "  Read 1:    13,874,517 bp\n",
      "  Read 2:    13,885,088 bp\n",
      "Total written (filtered):     25,933,810 bp (93.4%)\n",
      "  Read 1:    12,935,405 bp\n",
      "  Read 2:    12,998,405 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46963 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t291\t0.0\t1\t0 281 10\n",
      "20\t46536\t0.0\t2\t45755 776 5\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t46353\t0.0\t1\t44000 2353\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S945_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S945_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S945_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S945_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S945_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S945_185_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S945_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S945_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,631 reads  @     71.3 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.50 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,631\n",
      "  Read 1 with adapter:                  48,516 (99.8%)\n",
      "  Read 2 with adapter:                  48,184 (99.1%)\n",
      "Pairs written (passing filters):        48,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,680,117 bp\n",
      "  Read 1:    14,334,301 bp\n",
      "  Read 2:    14,345,816 bp\n",
      "Total written (filtered):     26,794,660 bp (93.4%)\n",
      "  Read 1:    13,364,185 bp\n",
      "  Read 2:    13,430,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48516 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t331\t0.0\t1\t0 326 5\n",
      "20\t48040\t0.0\t2\t47269 760 11\n",
      "21\t87\t0.0\t2\t0 86 1\n",
      "22\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48184 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t230\t0.0\t1\t0 230\n",
      "19\t47879\t0.0\t1\t45175 2704\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S946_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S946_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S946_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S946_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S946_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S946_187_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S946_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S946_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,153 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.91 s (76 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,153\n",
      "  Read 1 with adapter:                  51,093 (99.9%)\n",
      "  Read 2 with adapter:                  50,647 (99.0%)\n",
      "Pairs written (passing filters):        51,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,171,467 bp\n",
      "  Read 1:    15,085,135 bp\n",
      "  Read 2:    15,086,332 bp\n",
      "Total written (filtered):     28,187,625 bp (93.4%)\n",
      "  Read 1:    14,063,468 bp\n",
      "  Read 2:    14,124,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 51093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t325\t0.0\t1\t0 319 6\n",
      "20\t50619\t0.0\t2\t49751 854 14\n",
      "21\t102\t0.0\t2\t0 101 1\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 50647 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t50333\t0.0\t1\t47550 2783\n",
      "20\t98\t0.0\t1\t0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S947_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S947_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S947_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S947_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S947_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S947_189_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S947_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S947_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,568 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.83 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,568\n",
      "  Read 1 with adapter:                  48,547 (100.0%)\n",
      "  Read 2 with adapter:                  48,142 (99.1%)\n",
      "Pairs written (passing filters):        48,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,654,310 bp\n",
      "  Read 1:    14,326,790 bp\n",
      "  Read 2:    14,327,520 bp\n",
      "Total written (filtered):     26,769,039 bp (93.4%)\n",
      "  Read 1:    13,356,036 bp\n",
      "  Read 2:    13,413,003 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 48547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t299\t0.0\t1\t0 295 4\n",
      "20\t48119\t0.0\t2\t47314 792 13\n",
      "21\t97\t0.0\t2\t0 95 2\n",
      "22\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t297\t0.0\t1\t0 297\n",
      "19\t47729\t0.0\t1\t45043 2686\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S948_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S948_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S948_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S948_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S948_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S948_191_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S948_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S948_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,157 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.28 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,157\n",
      "  Read 1 with adapter:                  52,118 (99.9%)\n",
      "  Read 2 with adapter:                  51,657 (99.0%)\n",
      "Pairs written (passing filters):        52,157 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,767,094 bp\n",
      "  Read 1:    15,385,282 bp\n",
      "  Read 2:    15,381,812 bp\n",
      "Total written (filtered):     28,743,613 bp (93.4%)\n",
      "  Read 1:    14,343,113 bp\n",
      "  Read 2:    14,400,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t337\t0.0\t1\t0 334 3\n",
      "20\t51626\t0.0\t2\t50747 867 12\n",
      "21\t112\t0.0\t2\t0 109 3\n",
      "22\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 51657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t271\t0.0\t1\t0 271\n",
      "19\t51286\t0.0\t1\t48284 3002\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S949_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S949_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S949_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S949_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S949_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S949_193_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S949_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S949_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        59,125 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.71 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,125\n",
      "  Read 1 with adapter:                  59,078 (99.9%)\n",
      "  Read 2 with adapter:                  58,498 (98.9%)\n",
      "Pairs written (passing filters):        59,125 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,866,577 bp\n",
      "  Read 1:    17,440,599 bp\n",
      "  Read 2:    17,425,978 bp\n",
      "Total written (filtered):     32,573,972 bp (93.4%)\n",
      "  Read 1:    16,259,280 bp\n",
      "  Read 2:    16,314,692 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t414\t0.0\t1\t0 406 8\n",
      "20\t58483\t0.0\t2\t57397 1072 14\n",
      "21\t125\t0.0\t2\t0 119 6\n",
      "22\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 58498 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t278\t0.0\t1\t0 278\n",
      "19\t58118\t0.0\t1\t54863 3255\n",
      "20\t102\t0.0\t1\t0 102\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S950_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S950_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S950_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S950_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S950_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S950_195_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S950_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S950_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,786 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.86 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,786\n",
      "  Read 1 with adapter:                  59,748 (99.9%)\n",
      "  Read 2 with adapter:                  59,422 (99.4%)\n",
      "Pairs written (passing filters):        59,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,257,595 bp\n",
      "  Read 1:    17,635,656 bp\n",
      "  Read 2:    17,621,939 bp\n",
      "Total written (filtered):     32,933,990 bp (93.4%)\n",
      "  Read 1:    16,440,910 bp\n",
      "  Read 2:    16,493,080 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 59748 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t367\t0.0\t1\t0 363 4\n",
      "20\t59223\t0.0\t2\t58177 1028 18\n",
      "21\t119\t0.0\t2\t0 116 3\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 59422 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t270\t0.0\t1\t0 270\n",
      "19\t59041\t0.0\t1\t56202 2839\n",
      "20\t111\t0.0\t1\t0 111\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S951_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S951_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S951_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S951_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S951_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S951_197_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S951_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S951_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,308 reads  @     80.7 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.02 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,308\n",
      "  Read 1 with adapter:                  49,275 (99.9%)\n",
      "  Read 2 with adapter:                  48,368 (98.1%)\n",
      "Pairs written (passing filters):        49,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,043,225 bp\n",
      "  Read 1:    14,545,409 bp\n",
      "  Read 2:    14,497,816 bp\n",
      "Total written (filtered):     27,139,097 bp (93.4%)\n",
      "  Read 1:    13,560,115 bp\n",
      "  Read 2:    13,578,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 49275 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t322\t0.0\t1\t0 317 5\n",
      "20\t48816\t0.0\t2\t47913 892 11\n",
      "21\t94\t0.0\t2\t0 92 2\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 48368 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t233\t0.0\t1\t0 233\n",
      "19\t48060\t0.0\t1\t45951 2109\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S952_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S952_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S952_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S952_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S952_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S952_199_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S952_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S952_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,287 reads  @     76.6 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.12 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,287\n",
      "  Read 1 with adapter:                  40,264 (99.9%)\n",
      "  Read 2 with adapter:                  39,942 (99.1%)\n",
      "Pairs written (passing filters):        40,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,756,932 bp\n",
      "  Read 1:    11,884,360 bp\n",
      "  Read 2:    11,872,572 bp\n",
      "Total written (filtered):     22,192,762 bp (93.4%)\n",
      "  Read 1:    11,078,945 bp\n",
      "  Read 2:    11,113,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t240\t0.0\t1\t0 233 7\n",
      "20\t39910\t0.0\t2\t39126 778 6\n",
      "21\t76\t0.0\t2\t0 76\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t203\t0.0\t1\t0 203\n",
      "19\t39679\t0.0\t1\t37485 2194\n",
      "20\t60\t0.0\t1\t0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S953_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S953_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S953_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S953_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S953_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S953_201_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S953_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S953_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           269 reads  @    194.9 µs/read;   0.31 M reads/minute\n",
      "Finished in 0.06 s (206 us/read; 0.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                269\n",
      "  Read 1 with adapter:                     269 (100.0%)\n",
      "  Read 2 with adapter:                      22 (8.2%)\n",
      "Pairs written (passing filters):           269 (100.0%)\n",
      "\n",
      "Total basepairs processed:        90,565 bp\n",
      "  Read 1:        79,355 bp\n",
      "  Read 2:        11,210 bp\n",
      "Total written (filtered):         84,746 bp (93.6%)\n",
      "  Read 1:        73,976 bp\n",
      "  Read 2:        10,770 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 269 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t268\t0.0\t2\t263 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 22 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t22\t0.0\t1\t0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S954_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S954_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S954_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S954_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S954_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S954_203_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S954_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S954_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00        12,115 reads  @     79.8 µs/read;   0.75 M reads/minute\n",
      "Finished in 0.98 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             12,115\n",
      "  Read 1 with adapter:                  12,102 (99.9%)\n",
      "  Read 2 with adapter:                  11,675 (96.4%)\n",
      "Pairs written (passing filters):        12,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,145,992 bp\n",
      "  Read 1:     3,573,456 bp\n",
      "  Read 2:     3,572,536 bp\n",
      "Total written (filtered):      6,682,198 bp (93.5%)\n",
      "  Read 1:     3,331,433 bp\n",
      "  Read 2:     3,350,765 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 12102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t3\t0.0\t1\t0 0 3\n",
      "19\t94\t0.0\t1\t0 91 3\n",
      "20\t11950\t0.0\t2\t11675 271 4\n",
      "21\t27\t0.0\t2\t0 27\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 11675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t85\t0.0\t1\t0 85\n",
      "19\t11559\t0.0\t1\t10378 1181\n",
      "20\t31\t0.0\t1\t0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S955_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S955_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S955_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S955_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S955_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S955_205_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S955_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S955_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           108 reads  @    336.9 µs/read;   0.18 M reads/minute\n",
      "Finished in 0.04 s (366 us/read; 0.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                108\n",
      "  Read 1 with adapter:                     108 (100.0%)\n",
      "  Read 2 with adapter:                      12 (11.1%)\n",
      "Pairs written (passing filters):           108 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,530 bp\n",
      "  Read 1:        31,860 bp\n",
      "  Read 2:         7,670 bp\n",
      "Total written (filtered):         37,133 bp (93.9%)\n",
      "  Read 1:        29,703 bp\n",
      "  Read 2:         7,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 108 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 0 1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t106\t0.0\t2\t102 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t12\t0.0\t1\t0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S956_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S956_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S956_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S956_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S956_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S956_207_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S956_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S956_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,512 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.12 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,512\n",
      "  Read 1 with adapter:                  42,492 (100.0%)\n",
      "  Read 2 with adapter:                  42,266 (99.4%)\n",
      "Pairs written (passing filters):        42,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,076,476 bp\n",
      "  Read 1:    12,540,786 bp\n",
      "  Read 2:    12,535,690 bp\n",
      "Total written (filtered):     23,423,856 bp (93.4%)\n",
      "  Read 1:    11,691,116 bp\n",
      "  Read 2:    11,732,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 42492 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t25\t0.0\t1\t0 0 25\n",
      "19\t245\t0.0\t1\t0 239 6\n",
      "20\t42123\t0.0\t2\t41386 732 5\n",
      "21\t73\t0.0\t2\t0 71 2\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t185\t0.0\t1\t0 185\n",
      "19\t42000\t0.0\t1\t40114 1886\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S957_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S957_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S957_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S957_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S957_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S957_209_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S957_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S957_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,095 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.99 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,095\n",
      "  Read 1 with adapter:                  43,080 (100.0%)\n",
      "  Read 2 with adapter:                  42,955 (99.7%)\n",
      "Pairs written (passing filters):        43,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,425,391 bp\n",
      "  Read 1:    12,712,434 bp\n",
      "  Read 2:    12,712,957 bp\n",
      "Total written (filtered):     23,747,917 bp (93.4%)\n",
      "  Read 1:    11,850,984 bp\n",
      "  Read 2:    11,896,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43080 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t266\t0.0\t1\t0 260 6\n",
      "20\t42680\t0.0\t2\t41937 732 11\n",
      "21\t92\t0.0\t2\t0 89 3\n",
      "22\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 42955 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t188\t0.0\t1\t0 188\n",
      "19\t42700\t0.0\t1\t40833 1867\n",
      "20\t67\t0.0\t1\t0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S958_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S958_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S958_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S958_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S958_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S958_211_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S958_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S958_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,486 reads  @     67.8 µs/read;   0.89 M reads/minute\n",
      "Finished in 3.10 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,486\n",
      "  Read 1 with adapter:                  45,457 (99.9%)\n",
      "  Read 2 with adapter:                  45,318 (99.6%)\n",
      "Pairs written (passing filters):        45,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,836,057 bp\n",
      "  Read 1:    13,418,146 bp\n",
      "  Read 2:    13,417,911 bp\n",
      "Total written (filtered):     25,066,167 bp (93.4%)\n",
      "  Read 1:    12,509,180 bp\n",
      "  Read 2:    12,556,987 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45457 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t265\t0.0\t1\t0 259 6\n",
      "20\t45078\t0.0\t2\t44308 757 13\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t224\t0.0\t1\t0 224\n",
      "19\t44988\t0.0\t1\t42969 2019\n",
      "20\t106\t0.0\t1\t0 106\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S959_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S959_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S959_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S959_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S959_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S959_213_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S959_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S959_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           101 reads  @    395.7 µs/read;   0.15 M reads/minute\n",
      "Finished in 0.04 s (429 us/read; 0.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                101\n",
      "  Read 1 with adapter:                     100 (99.0%)\n",
      "  Read 2 with adapter:                      21 (20.8%)\n",
      "Pairs written (passing filters):           101 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,118 bp\n",
      "  Read 1:        29,795 bp\n",
      "  Read 2:         9,323 bp\n",
      "Total written (filtered):         36,698 bp (93.8%)\n",
      "  Read 1:        27,795 bp\n",
      "  Read 2:         8,903 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t100\t0.0\t2\t98 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 21 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t21\t0.0\t1\t0 21\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S960_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S960_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S960_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S960_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S960_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S960_215_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S960_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S960_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        43,617 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.85 s (65 us/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,617\n",
      "  Read 1 with adapter:                  43,589 (99.9%)\n",
      "  Read 2 with adapter:                  43,436 (99.6%)\n",
      "Pairs written (passing filters):        43,617 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,730,171 bp\n",
      "  Read 1:    12,867,000 bp\n",
      "  Read 2:    12,863,171 bp\n",
      "Total written (filtered):     24,033,387 bp (93.4%)\n",
      "  Read 1:    11,995,381 bp\n",
      "  Read 2:    12,038,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t256\t0.0\t1\t0 254 2\n",
      "20\t43217\t0.0\t2\t42469 738 10\n",
      "21\t81\t0.0\t2\t0 78 3\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t194\t0.0\t1\t0 194\n",
      "19\t43167\t0.0\t1\t41208 1959\n",
      "20\t75\t0.0\t1\t0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S961_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S961_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S961_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S961_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S961_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S961_217_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S961_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S961_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        36,752 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.46 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,752\n",
      "  Read 1 with adapter:                  36,739 (100.0%)\n",
      "  Read 2 with adapter:                  36,626 (99.7%)\n",
      "Pairs written (passing filters):        36,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,683,624 bp\n",
      "  Read 1:    10,841,808 bp\n",
      "  Read 2:    10,841,816 bp\n",
      "Total written (filtered):     20,253,258 bp (93.4%)\n",
      "  Read 1:    10,107,211 bp\n",
      "  Read 2:    10,146,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 36739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t9\t0.0\t1\t0 0 9\n",
      "19\t268\t0.0\t1\t0 262 6\n",
      "20\t36385\t0.0\t2\t35787 588 10\n",
      "21\t51\t0.0\t2\t0 50 1\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 36626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t181\t0.0\t1\t0 181\n",
      "19\t36389\t0.0\t1\t34751 1638\n",
      "20\t56\t0.0\t1\t0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S962_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S962_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S962_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S962_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S962_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S962_219_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S962_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S962_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,138 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.26 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,138\n",
      "  Read 1 with adapter:                  47,115 (100.0%)\n",
      "  Read 2 with adapter:                  46,987 (99.7%)\n",
      "Pairs written (passing filters):        47,138 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,809,655 bp\n",
      "  Read 1:    13,905,500 bp\n",
      "  Read 2:    13,904,155 bp\n",
      "Total written (filtered):     25,974,904 bp (93.4%)\n",
      "  Read 1:    12,963,374 bp\n",
      "  Read 2:    13,011,530 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47115 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t295\t0.0\t1\t0 284 11\n",
      "20\t46687\t0.0\t2\t45929 751 7\n",
      "21\t85\t0.0\t2\t0 83 2\n",
      "22\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 46987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t244\t0.0\t1\t0 244\n",
      "19\t46627\t0.0\t1\t44525 2102\n",
      "20\t116\t0.0\t1\t0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S963_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S963_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S963_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S963_221_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S963_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S963_221_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S963_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S963_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           113 reads  @    398.9 µs/read;   0.15 M reads/minute\n",
      "Finished in 0.05 s (423 us/read; 0.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                113\n",
      "  Read 1 with adapter:                     113 (100.0%)\n",
      "  Read 2 with adapter:                      10 (8.8%)\n",
      "Pairs written (passing filters):           113 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,019 bp\n",
      "  Read 1:        33,335 bp\n",
      "  Read 2:         7,684 bp\n",
      "Total written (filtered):         38,561 bp (94.0%)\n",
      "  Read 1:        31,077 bp\n",
      "  Read 2:         7,484 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 113 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t111\t0.0\t2\t111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 10 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t10\t0.0\t1\t0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S964_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S964_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S964_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S964_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S964_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S964_223_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S964_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S964_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,033 reads  @     77.9 µs/read;   0.77 M reads/minute\n",
      "Finished in 2.51 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,033\n",
      "  Read 1 with adapter:                  32,012 (99.9%)\n",
      "  Read 2 with adapter:                  31,709 (99.0%)\n",
      "Pairs written (passing filters):        32,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,890,097 bp\n",
      "  Read 1:     9,449,722 bp\n",
      "  Read 2:     9,440,375 bp\n",
      "Total written (filtered):     17,647,561 bp (93.4%)\n",
      "  Read 1:     8,809,549 bp\n",
      "  Read 2:     8,838,012 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32012 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t5\t0.0\t1\t0 0 5\n",
      "19\t154\t0.0\t1\t0 151 3\n",
      "20\t31777\t0.0\t2\t31247 523 7\n",
      "21\t55\t0.0\t2\t0 55\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t162\t0.0\t1\t0 162\n",
      "19\t31493\t0.0\t1\t29491 2002\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S965_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S965_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S965_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S965_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S965_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S965_225_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S965_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S965_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,888 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.47 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,888\n",
      "  Read 1 with adapter:                  47,856 (99.9%)\n",
      "  Read 2 with adapter:                  47,547 (99.3%)\n",
      "Pairs written (passing filters):        47,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,253,688 bp\n",
      "  Read 1:    14,126,827 bp\n",
      "  Read 2:    14,126,861 bp\n",
      "Total written (filtered):     26,393,496 bp (93.4%)\n",
      "  Read 1:    13,169,905 bp\n",
      "  Read 2:    13,223,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47856 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t288\t0.0\t1\t0 284 4\n",
      "20\t47458\t0.0\t2\t46571 874 13\n",
      "21\t78\t0.0\t2\t0 75 3\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t216\t0.0\t1\t0 216\n",
      "19\t47238\t0.0\t1\t44501 2737\n",
      "20\t93\t0.0\t1\t0 93\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S966_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S966_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S966_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S966_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S966_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S966_227_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S966_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S966_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        38,711 reads  @     69.2 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.69 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,711\n",
      "  Read 1 with adapter:                  38,684 (99.9%)\n",
      "  Read 2 with adapter:                  38,471 (99.4%)\n",
      "Pairs written (passing filters):        38,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,838,579 bp\n",
      "  Read 1:    11,419,693 bp\n",
      "  Read 2:    11,418,886 bp\n",
      "Total written (filtered):     21,334,227 bp (93.4%)\n",
      "  Read 1:    10,646,160 bp\n",
      "  Read 2:    10,688,067 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 38684 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t216\t0.0\t1\t0 215 1\n",
      "20\t38373\t0.0\t2\t37660 699 14\n",
      "21\t61\t0.0\t2\t0 60 1\n",
      "22\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 38471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t38161\t0.0\t1\t36009 2152\n",
      "20\t90\t0.0\t1\t0 90\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S967_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S967_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S967_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S967_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S967_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S967_229_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S967_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S967_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           424 reads  @    119.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 0.05 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                424\n",
      "  Read 1 with adapter:                     424 (100.0%)\n",
      "  Read 2 with adapter:                      28 (6.6%)\n",
      "Pairs written (passing filters):           424 (100.0%)\n",
      "\n",
      "Total basepairs processed:       141,664 bp\n",
      "  Read 1:       125,080 bp\n",
      "  Read 2:        16,584 bp\n",
      "Total written (filtered):        132,629 bp (93.6%)\n",
      "  Read 1:       116,605 bp\n",
      "  Read 2:        16,024 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t6\t0.0\t1\t0 6\n",
      "20\t417\t0.0\t2\t410 7\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S968_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S968_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S968_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S968_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S968_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S968_231_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S968_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S968_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,747 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.29 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,747\n",
      "  Read 1 with adapter:                  47,725 (100.0%)\n",
      "  Read 2 with adapter:                  47,384 (99.2%)\n",
      "Pairs written (passing filters):        47,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,162,566 bp\n",
      "  Read 1:    14,085,355 bp\n",
      "  Read 2:    14,077,211 bp\n",
      "Total written (filtered):     26,308,101 bp (93.4%)\n",
      "  Read 1:    13,131,054 bp\n",
      "  Read 2:    13,177,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47725 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t12\t0.0\t1\t0 0 12\n",
      "19\t313\t0.0\t1\t0 308 5\n",
      "20\t47287\t0.0\t2\t46534 742 11\n",
      "21\t88\t0.0\t2\t0 85 3\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47384 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t219\t0.0\t1\t0 219\n",
      "19\t47078\t0.0\t1\t44437 2641\n",
      "20\t87\t0.0\t1\t0 87\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S969_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S969_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S969_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S969_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S969_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S969_233_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S969_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S969_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        81,659 reads  @     66.4 µs/read;   0.90 M reads/minute\n",
      "Finished in 5.44 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,659\n",
      "  Read 1 with adapter:                  81,623 (100.0%)\n",
      "  Read 2 with adapter:                  81,104 (99.3%)\n",
      "Pairs written (passing filters):        81,659 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,178,405 bp\n",
      "  Read 1:    24,089,117 bp\n",
      "  Read 2:    24,089,288 bp\n",
      "Total written (filtered):     45,005,525 bp (93.4%)\n",
      "  Read 1:    22,456,990 bp\n",
      "  Read 2:    22,548,535 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 81623 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t525\t0.0\t1\t0 520 5\n",
      "20\t80886\t0.0\t2\t79508 1360 18\n",
      "21\t160\t0.0\t2\t0 155 5\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 81104 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t358\t0.0\t1\t0 358\n",
      "19\t80611\t0.0\t1\t76106 4505\n",
      "20\t135\t0.0\t1\t0 135\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S970_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S970_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S970_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S970_235_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S970_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S970_235_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S970_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S970_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,009 reads  @     71.8 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.26 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,009\n",
      "  Read 1 with adapter:                  44,990 (100.0%)\n",
      "  Read 2 with adapter:                  44,584 (99.1%)\n",
      "Pairs written (passing filters):        45,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,553,091 bp\n",
      "  Read 1:    13,277,643 bp\n",
      "  Read 2:    13,275,448 bp\n",
      "Total written (filtered):     24,806,510 bp (93.4%)\n",
      "  Read 1:    12,378,052 bp\n",
      "  Read 2:    12,428,458 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t18\t0.0\t1\t0 0 18\n",
      "19\t307\t0.0\t1\t0 301 6\n",
      "20\t44556\t0.0\t2\t43781 764 11\n",
      "21\t84\t0.0\t2\t0 83 1\n",
      "22\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44584 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t44250\t0.0\t1\t41670 2580\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S971_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S971_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S971_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S971_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S971_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S971_237_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S971_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S971_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           120 reads  @    294.6 µs/read;   0.20 M reads/minute\n",
      "Finished in 0.04 s (319 us/read; 0.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                120\n",
      "  Read 1 with adapter:                     120 (100.0%)\n",
      "  Read 2 with adapter:                       9 (7.5%)\n",
      "Pairs written (passing filters):           120 (100.0%)\n",
      "\n",
      "Total basepairs processed:        42,869 bp\n",
      "  Read 1:        35,400 bp\n",
      "  Read 2:         7,469 bp\n",
      "Total written (filtered):         40,289 bp (94.0%)\n",
      "  Read 1:        33,000 bp\n",
      "  Read 2:         7,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 120 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t120\t0.0\t2\t118 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 9 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t9\t0.0\t1\t0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S972_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S972_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S972_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S972_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S972_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S972_239_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S972_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S972_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,385 reads  @     65.5 µs/read;   0.92 M reads/minute\n",
      "Finished in 2.14 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,385\n",
      "  Read 1 with adapter:                  32,367 (99.9%)\n",
      "  Read 2 with adapter:                  32,256 (99.6%)\n",
      "Pairs written (passing filters):        32,385 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,104,890 bp\n",
      "  Read 1:     9,553,562 bp\n",
      "  Read 2:     9,551,328 bp\n",
      "Total written (filtered):     17,844,920 bp (93.4%)\n",
      "  Read 1:     8,906,349 bp\n",
      "  Read 2:     8,938,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32367 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t7\t0.0\t1\t0 0 7\n",
      "19\t187\t0.0\t1\t0 184 3\n",
      "20\t32111\t0.0\t2\t31594 513 4\n",
      "21\t50\t0.0\t2\t0 50\n",
      "22\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 32256 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t168\t0.0\t1\t0 168\n",
      "19\t32027\t0.0\t1\t30656 1371\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S973_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S973_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S973_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S973_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S973_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S973_241_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S973_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S973_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        64,783 reads  @     63.4 µs/read;   0.95 M reads/minute\n",
      "Finished in 4.11 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,783\n",
      "  Read 1 with adapter:                  64,743 (99.9%)\n",
      "  Read 2 with adapter:                  64,602 (99.7%)\n",
      "Pairs written (passing filters):        64,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,221,696 bp\n",
      "  Read 1:    19,110,960 bp\n",
      "  Read 2:    19,110,736 bp\n",
      "Total written (filtered):     35,699,827 bp (93.4%)\n",
      "  Read 1:    17,816,346 bp\n",
      "  Read 2:    17,883,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 64743 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t24\t0.0\t1\t0 0 24\n",
      "19\t377\t0.0\t1\t0 372 5\n",
      "20\t64194\t0.0\t2\t63142 1034 18\n",
      "21\t117\t0.0\t2\t0 114 3\n",
      "22\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 64602 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t297\t0.0\t1\t0 297\n",
      "19\t64191\t0.0\t1\t61450 2741\n",
      "20\t114\t0.0\t1\t0 114\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S974_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S974_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S974_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S974_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S974_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S974_243_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S974_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S974_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,499 reads  @     65.4 µs/read;   0.92 M reads/minute\n",
      "Finished in 3.46 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,499\n",
      "  Read 1 with adapter:                  52,466 (99.9%)\n",
      "  Read 2 with adapter:                  52,318 (99.7%)\n",
      "Pairs written (passing filters):        52,499 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,973,971 bp\n",
      "  Read 1:    15,487,185 bp\n",
      "  Read 2:    15,486,786 bp\n",
      "Total written (filtered):     28,930,892 bp (93.4%)\n",
      "  Read 1:    14,438,025 bp\n",
      "  Read 2:    14,492,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t14\t0.0\t1\t0 0 14\n",
      "19\t294\t0.0\t1\t0 290 4\n",
      "20\t52025\t0.0\t2\t51185 833 7\n",
      "21\t104\t0.0\t2\t0 103 1\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52318 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t243\t0.0\t1\t0 243\n",
      "19\t51955\t0.0\t1\t49704 2251\n",
      "20\t120\t0.0\t1\t0 120\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S975_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S975_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S975_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S975_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S975_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S975_245_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S975_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S975_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           167 reads  @    223.4 µs/read;   0.27 M reads/minute\n",
      "Finished in 0.04 s (246 us/read; 0.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                167\n",
      "  Read 1 with adapter:                     167 (100.0%)\n",
      "  Read 2 with adapter:                      35 (21.0%)\n",
      "Pairs written (passing filters):           167 (100.0%)\n",
      "\n",
      "Total basepairs processed:        64,457 bp\n",
      "  Read 1:        49,265 bp\n",
      "  Read 2:        15,192 bp\n",
      "Total written (filtered):         60,418 bp (93.7%)\n",
      "  Read 1:        45,926 bp\n",
      "  Read 2:        14,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t166\t0.0\t2\t164 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t35\t0.0\t1\t0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S976_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S976_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S976_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S976_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S976_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S976_247_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S976_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S976_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,647 reads  @     68.6 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.85 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,647\n",
      "  Read 1 with adapter:                  55,604 (99.9%)\n",
      "  Read 2 with adapter:                  55,416 (99.6%)\n",
      "Pairs written (passing filters):        55,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,824,368 bp\n",
      "  Read 1:    16,415,448 bp\n",
      "  Read 2:    16,408,920 bp\n",
      "Total written (filtered):     30,659,617 bp (93.4%)\n",
      "  Read 1:    15,303,457 bp\n",
      "  Read 2:    15,356,160 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 55604 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t20\t0.0\t1\t0 0 20\n",
      "19\t270\t0.0\t1\t0 267 3\n",
      "20\t55128\t0.0\t2\t54207 907 14\n",
      "21\t151\t0.0\t2\t0 151\n",
      "22\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 55416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t253\t0.0\t1\t0 253\n",
      "19\t55054\t0.0\t1\t52687 2367\n",
      "20\t109\t0.0\t1\t0 109\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S977_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S977_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S977_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S977_249_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S977_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S977_249_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S977_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S977_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,654 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.07 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,654\n",
      "  Read 1 with adapter:                  44,626 (99.9%)\n",
      "  Read 2 with adapter:                  44,515 (99.7%)\n",
      "Pairs written (passing filters):        44,654 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,345,486 bp\n",
      "  Read 1:    13,172,904 bp\n",
      "  Read 2:    13,172,582 bp\n",
      "Total written (filtered):     24,607,423 bp (93.4%)\n",
      "  Read 1:    12,280,470 bp\n",
      "  Read 2:    12,326,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 44626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t234\t0.0\t1\t0 230 4\n",
      "20\t44234\t0.0\t2\t43532 693 9\n",
      "21\t116\t0.0\t2\t0 113 3\n",
      "22\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 44515 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t232\t0.0\t1\t0 232\n",
      "19\t44207\t0.0\t1\t42312 1895\n",
      "20\t76\t0.0\t1\t0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S978_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S978_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S978_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S978_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S978_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S978_251_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S978_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S978_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        80,457 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 5.64 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,457\n",
      "  Read 1 with adapter:                  80,423 (100.0%)\n",
      "  Read 2 with adapter:                  80,133 (99.6%)\n",
      "Pairs written (passing filters):        80,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,466,792 bp\n",
      "  Read 1:    23,734,790 bp\n",
      "  Read 2:    23,732,002 bp\n",
      "Total written (filtered):     44,336,239 bp (93.4%)\n",
      "  Read 1:    22,126,523 bp\n",
      "  Read 2:    22,209,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 80423 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t29\t0.0\t1\t0 0 29\n",
      "19\t440\t0.0\t1\t0 436 4\n",
      "20\t79701\t0.0\t2\t78326 1353 22\n",
      "21\t201\t0.0\t2\t0 201\n",
      "22\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 80133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t423\t0.0\t1\t0 423\n",
      "19\t79528\t0.0\t1\t75871 3657\n",
      "20\t182\t0.0\t1\t0 182\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S979_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S979_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S979_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S979_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S979_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S979_253_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S979_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S979_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           138 reads  @    253.8 µs/read;   0.24 M reads/minute\n",
      "Finished in 0.04 s (274 us/read; 0.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                138\n",
      "  Read 1 with adapter:                     138 (100.0%)\n",
      "  Read 2 with adapter:                      13 (9.4%)\n",
      "Pairs written (passing filters):           138 (100.0%)\n",
      "\n",
      "Total basepairs processed:        50,498 bp\n",
      "  Read 1:        40,710 bp\n",
      "  Read 2:         9,788 bp\n",
      "Total written (filtered):         47,478 bp (94.0%)\n",
      "  Read 1:        37,950 bp\n",
      "  Read 2:         9,528 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t136\t0.0\t2\t132 4\n",
      "21\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 13 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t13\t0.0\t1\t0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S980_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S980_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S980_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S980_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S980_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S980_255_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S980_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S980_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[8=----------] 00:00:02        33,695 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 2.33 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,695\n",
      "  Read 1 with adapter:                  33,671 (99.9%)\n",
      "  Read 2 with adapter:                  33,490 (99.4%)\n",
      "Pairs written (passing filters):        33,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,874,930 bp\n",
      "  Read 1:     9,939,733 bp\n",
      "  Read 2:     9,935,197 bp\n",
      "Total written (filtered):     18,565,437 bp (93.4%)\n",
      "  Read 1:     9,266,454 bp\n",
      "  Read 2:     9,298,983 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 33671 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t10\t0.0\t1\t0 0 10\n",
      "19\t230\t0.0\t1\t0 223 7\n",
      "20\t33346\t0.0\t2\t32792 546 8\n",
      "21\t61\t0.0\t2\t0 59 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33490 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t155\t0.0\t1\t0 155\n",
      "19\t33276\t0.0\t1\t31651 1625\n",
      "20\t59\t0.0\t1\t0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S981_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S981_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S981_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S981_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S981_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S981_257_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S981_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S981_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,186 reads  @     66.7 µs/read;   0.90 M reads/minute\n",
      "Finished in 2.09 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,186\n",
      "  Read 1 with adapter:                  31,167 (99.9%)\n",
      "  Read 2 with adapter:                  31,036 (99.5%)\n",
      "Pairs written (passing filters):        31,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,399,591 bp\n",
      "  Read 1:     9,199,858 bp\n",
      "  Read 2:     9,199,733 bp\n",
      "Total written (filtered):     17,186,751 bp (93.4%)\n",
      "  Read 1:     8,576,615 bp\n",
      "  Read 2:     8,610,136 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 31167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t197\t0.0\t1\t0 196 1\n",
      "20\t30858\t0.0\t2\t30391 457 10\n",
      "21\t80\t0.0\t2\t0 78 2\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31036 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t141\t0.0\t1\t0 141\n",
      "19\t30841\t0.0\t1\t29453 1388\n",
      "20\t54\t0.0\t1\t0 54\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S982_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S982_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S982_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S982_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S982_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S982_259_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S982_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S982_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,850 reads  @     68.1 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.47 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,850\n",
      "  Read 1 with adapter:                  35,833 (100.0%)\n",
      "  Read 2 with adapter:                  35,645 (99.4%)\n",
      "Pairs written (passing filters):        35,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,151,269 bp\n",
      "  Read 1:    10,575,739 bp\n",
      "  Read 2:    10,575,530 bp\n",
      "Total written (filtered):     19,757,612 bp (93.4%)\n",
      "  Read 1:     9,859,224 bp\n",
      "  Read 2:     9,898,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 35833 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t257\t0.0\t1\t0 251 6\n",
      "20\t35455\t0.0\t2\t34849 599 7\n",
      "21\t86\t0.0\t2\t0 84 2\n",
      "22\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 35645 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t192\t0.0\t1\t0 192\n",
      "19\t35374\t0.0\t1\t33631 1743\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S983_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S983_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S983_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S983_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S983_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S983_261_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S983_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S983_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           387 reads  @    139.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 0.06 s (147 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                387\n",
      "  Read 1 with adapter:                     387 (100.0%)\n",
      "  Read 2 with adapter:                      14 (3.6%)\n",
      "Pairs written (passing filters):           387 (100.0%)\n",
      "\n",
      "Total basepairs processed:       124,299 bp\n",
      "  Read 1:       114,165 bp\n",
      "  Read 2:        10,134 bp\n",
      "Total written (filtered):        116,280 bp (93.5%)\n",
      "  Read 1:       106,426 bp\n",
      "  Read 2:         9,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 387 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t4\t0.0\t1\t0 4\n",
      "20\t381\t0.0\t2\t372 9\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 14 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t14\t0.0\t1\t0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S984_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S984_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S984_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S984_263_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S984_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S984_263_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S984_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S984_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,630 reads  @     69.1 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.19 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,630\n",
      "  Read 1 with adapter:                  45,607 (99.9%)\n",
      "  Read 2 with adapter:                  45,390 (99.5%)\n",
      "Pairs written (passing filters):        45,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,913,800 bp\n",
      "  Read 1:    13,460,836 bp\n",
      "  Read 2:    13,452,964 bp\n",
      "Total written (filtered):     25,139,695 bp (93.4%)\n",
      "  Read 1:    12,549,028 bp\n",
      "  Read 2:    12,590,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 45607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t19\t0.0\t1\t0 0 19\n",
      "19\t428\t0.0\t1\t0 423 5\n",
      "20\t45047\t0.0\t2\t44248 788 11\n",
      "21\t92\t0.0\t2\t0 91 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45390 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t195\t0.0\t1\t0 195\n",
      "19\t45113\t0.0\t1\t42938 2175\n",
      "20\t82\t0.0\t1\t0 82\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S985_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S985_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S985_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S985_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S985_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S985_265_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S985_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S985_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        41,146 reads  @     66.3 µs/read;   0.91 M reads/minute\n",
      "Finished in 2.75 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,146\n",
      "  Read 1 with adapter:                  41,129 (100.0%)\n",
      "  Read 2 with adapter:                  40,960 (99.5%)\n",
      "Pairs written (passing filters):        41,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,275,489 bp\n",
      "  Read 1:    12,138,056 bp\n",
      "  Read 2:    12,137,433 bp\n",
      "Total written (filtered):     22,675,054 bp (93.4%)\n",
      "  Read 1:    11,315,742 bp\n",
      "  Read 2:    11,359,312 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 41129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t8\t0.0\t1\t0 0 8\n",
      "19\t385\t0.0\t1\t0 381 4\n",
      "20\t40624\t0.0\t2\t39934 677 13\n",
      "21\t89\t0.0\t2\t0 87 2\n",
      "22\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40960 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t198\t0.0\t1\t0 198\n",
      "19\t40683\t0.0\t1\t38696 1987\n",
      "20\t79\t0.0\t1\t0 79\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S986_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S986_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S986_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S986_267_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S986_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S986_267_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S986_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S986_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        40,724 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 2.93 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,724\n",
      "  Read 1 with adapter:                  40,704 (100.0%)\n",
      "  Read 2 with adapter:                  40,434 (99.3%)\n",
      "Pairs written (passing filters):        40,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,025,633 bp\n",
      "  Read 1:    12,013,565 bp\n",
      "  Read 2:    12,012,068 bp\n",
      "Total written (filtered):     22,443,682 bp (93.4%)\n",
      "  Read 1:    11,199,749 bp\n",
      "  Read 2:    11,243,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40704 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t340\t0.0\t1\t0 338 2\n",
      "20\t40265\t0.0\t2\t39592 660 13\n",
      "21\t70\t0.0\t2\t0 68 2\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40434 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t210\t0.0\t1\t0 210\n",
      "19\t40125\t0.0\t1\t38082 2043\n",
      "20\t99\t0.0\t1\t0 99\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S987_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S987_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S987_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S987_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S987_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S987_269_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S987_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S987_269_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            79 reads  @    489.3 µs/read;   0.12 M reads/minute\n",
      "Finished in 0.04 s (529 us/read; 0.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 79\n",
      "  Read 1 with adapter:                      78 (98.7%)\n",
      "  Read 2 with adapter:                      12 (15.2%)\n",
      "Pairs written (passing filters):            79 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,339 bp\n",
      "  Read 1:        23,305 bp\n",
      "  Read 2:         6,034 bp\n",
      "Total written (filtered):         27,540 bp (93.9%)\n",
      "  Read 1:        21,746 bp\n",
      "  Read 2:         5,794 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 78 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t77\t0.0\t2\t75 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 12 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t12\t0.0\t1\t0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S988_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S988_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S988_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S988_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S988_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S988_271_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S988_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S988_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,142 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 2.42 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,142\n",
      "  Read 1 with adapter:                  34,076 (99.8%)\n",
      "  Read 2 with adapter:                  33,859 (99.2%)\n",
      "Pairs written (passing filters):        34,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,140,186 bp\n",
      "  Read 1:    10,071,780 bp\n",
      "  Read 2:    10,068,406 bp\n",
      "Total written (filtered):     18,815,552 bp (93.4%)\n",
      "  Read 1:     9,390,375 bp\n",
      "  Read 2:     9,425,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 34076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t193\t0.0\t1\t0 191 2\n",
      "20\t33788\t0.0\t2\t33191 589 8\n",
      "21\t68\t0.0\t2\t0 67 1\n",
      "22\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 33859 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t153\t0.0\t1\t0 153\n",
      "19\t33645\t0.0\t1\t31845 1800\n",
      "20\t61\t0.0\t1\t0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S989_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S989_273_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S989_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S989_273_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S989_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S989_273_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S989_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S989_273_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,965 reads  @     74.6 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.97 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,965\n",
      "  Read 1 with adapter:                  52,902 (99.9%)\n",
      "  Read 2 with adapter:                  52,612 (99.3%)\n",
      "Pairs written (passing filters):        52,965 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,247,747 bp\n",
      "  Read 1:    15,623,457 bp\n",
      "  Read 2:    15,624,290 bp\n",
      "Total written (filtered):     29,190,470 bp (93.4%)\n",
      "  Read 1:    14,565,668 bp\n",
      "  Read 2:    14,624,802 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 52902 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t30\t0.0\t1\t0 0 30\n",
      "19\t347\t0.0\t1\t0 336 11\n",
      "20\t52395\t0.0\t2\t51470 915 10\n",
      "21\t104\t0.0\t2\t0 104\n",
      "22\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 52612 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t221\t0.0\t1\t0 221\n",
      "19\t52310\t0.0\t1\t49481 2829\n",
      "20\t81\t0.0\t1\t0 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S990_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S990_275_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S990_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S990_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S990_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S990_275_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S990_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S990_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,229 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.13 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,229\n",
      "  Read 1 with adapter:                  40,170 (99.9%)\n",
      "  Read 2 with adapter:                  39,991 (99.4%)\n",
      "Pairs written (passing filters):        40,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,733,390 bp\n",
      "  Read 1:    11,867,438 bp\n",
      "  Read 2:    11,865,952 bp\n",
      "Total written (filtered):     22,170,460 bp (93.4%)\n",
      "  Read 1:    11,064,242 bp\n",
      "  Read 2:    11,106,218 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 40170 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t272\t0.0\t1\t0 268 4\n",
      "20\t39806\t0.0\t2\t39032 765 9\n",
      "21\t56\t0.0\t2\t0 56\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 39991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t209\t0.0\t1\t0 209\n",
      "19\t39668\t0.0\t1\t37696 1972\n",
      "20\t114\t0.0\t1\t0 114\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S991_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S991_277_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S991_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S991_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S991_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S991_277_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S991_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S991_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           334 reads  @    157.1 µs/read;   0.38 M reads/minute\n",
      "Finished in 0.06 s (166 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                334\n",
      "  Read 1 with adapter:                     334 (100.0%)\n",
      "  Read 2 with adapter:                      28 (8.4%)\n",
      "Pairs written (passing filters):           333 (99.7%)\n",
      "\n",
      "Total basepairs processed:       115,836 bp\n",
      "  Read 1:        98,530 bp\n",
      "  Read 2:        17,306 bp\n",
      "Total written (filtered):        108,322 bp (93.5%)\n",
      "  Read 1:        91,576 bp\n",
      "  Read 2:        16,746 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 334 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t333\t0.0\t2\t327 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 28 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t28\t0.0\t1\t0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S992_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S992_279_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S992_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S992_279_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S992_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S992_279_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S992_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S992_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,286 reads  @     74.5 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.09 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,286\n",
      "  Read 1 with adapter:                  54,259 (100.0%)\n",
      "  Read 2 with adapter:                  53,920 (99.3%)\n",
      "Pairs written (passing filters):        54,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,019,376 bp\n",
      "  Read 1:    16,014,208 bp\n",
      "  Read 2:    16,005,168 bp\n",
      "Total written (filtered):     29,910,078 bp (93.4%)\n",
      "  Read 1:    14,929,238 bp\n",
      "  Read 2:    14,980,840 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 54259 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t353\t0.0\t1\t0 347 6\n",
      "20\t53750\t0.0\t2\t52755 988 7\n",
      "21\t109\t0.0\t2\t0 108 1\n",
      "22\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 53920 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t252\t0.0\t1\t0 252\n",
      "19\t53568\t0.0\t1\t50771 2797\n",
      "20\t100\t0.0\t1\t0 100\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S993_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S993_281_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S993_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S993_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S993_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S993_281_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S993_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S993_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,093 reads  @     75.6 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.49 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,093\n",
      "  Read 1 with adapter:                  46,073 (100.0%)\n",
      "  Read 2 with adapter:                  45,840 (99.5%)\n",
      "Pairs written (passing filters):        46,093 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,194,420 bp\n",
      "  Read 1:    13,597,055 bp\n",
      "  Read 2:    13,597,365 bp\n",
      "Total written (filtered):     25,402,243 bp (93.4%)\n",
      "  Read 1:    12,675,714 bp\n",
      "  Read 2:    12,726,529 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 46073 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t13\t0.0\t1\t0 0 13\n",
      "19\t250\t0.0\t1\t0 246 4\n",
      "20\t45687\t0.0\t2\t44917 753 17\n",
      "21\t89\t0.0\t2\t0 89\n",
      "22\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 45840 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t204\t0.0\t1\t0 204\n",
      "19\t45556\t0.0\t1\t43150 2406\n",
      "20\t80\t0.0\t1\t0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S994_282_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S994_283_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S994_282_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S994_283_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S994_282_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S994_283_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S994_282_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S994_283_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,880 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.54 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,880\n",
      "  Read 1 with adapter:                  43,850 (99.9%)\n",
      "  Read 2 with adapter:                  43,590 (99.3%)\n",
      "Pairs written (passing filters):        43,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,885,306 bp\n",
      "  Read 1:    12,944,106 bp\n",
      "  Read 2:    12,941,200 bp\n",
      "Total written (filtered):     24,180,388 bp (93.4%)\n",
      "  Read 1:    12,067,285 bp\n",
      "  Read 2:    12,113,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 43850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t16\t0.0\t1\t0 0 16\n",
      "19\t264\t0.0\t1\t0 260 4\n",
      "20\t43481\t0.0\t2\t42718 755 8\n",
      "21\t61\t0.0\t2\t0 61\n",
      "22\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 43590 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t220\t0.0\t1\t0 220\n",
      "19\t43263\t0.0\t1\t40989 2274\n",
      "20\t107\t0.0\t1\t0 107\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S995_284_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S995_285_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S995_284_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S995_285_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S995_284_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S995_285_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S995_284_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S995_285_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           112 reads  @    335.7 µs/read;   0.18 M reads/minute\n",
      "Finished in 0.04 s (371 us/read; 0.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                112\n",
      "  Read 1 with adapter:                     112 (100.0%)\n",
      "  Read 2 with adapter:                      15 (13.4%)\n",
      "Pairs written (passing filters):           112 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,661 bp\n",
      "  Read 1:        33,040 bp\n",
      "  Read 2:         8,621 bp\n",
      "Total written (filtered):         39,123 bp (93.9%)\n",
      "  Read 1:        30,802 bp\n",
      "  Read 2:         8,321 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 112 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t110\t0.0\t2\t108 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 15 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t15\t0.0\t1\t0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S996_286_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S996_287_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S996_286_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S996_287_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S996_286_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S996_287_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S996_286_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S996_287_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,220 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 2.71 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,220\n",
      "  Read 1 with adapter:                  32,186 (99.9%)\n",
      "  Read 2 with adapter:                  31,929 (99.1%)\n",
      "Pairs written (passing filters):        32,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,994,378 bp\n",
      "  Read 1:     9,501,314 bp\n",
      "  Read 2:     9,493,064 bp\n",
      "Total written (filtered):     17,743,925 bp (93.4%)\n",
      "  Read 1:     8,857,425 bp\n",
      "  Read 2:     8,886,500 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 32186 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t11\t0.0\t1\t0 0 11\n",
      "19\t190\t0.0\t1\t0 185 5\n",
      "20\t31901\t0.0\t2\t31348 541 12\n",
      "21\t62\t0.0\t2\t0 61 1\n",
      "22\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 31929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t145\t0.0\t1\t0 145\n",
      "19\t31726\t0.0\t1\t29687 2039\n",
      "20\t58\t0.0\t1\t0 58\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S997_288_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S997_289_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S997_288_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S997_289_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S997_288_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S997_289_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S997_288_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S997_289_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        86,033 reads  @     74.8 µs/read;   0.80 M reads/minute\n",
      "Finished in 6.47 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             86,033\n",
      "  Read 1 with adapter:                  85,948 (99.9%)\n",
      "  Read 2 with adapter:                  85,427 (99.3%)\n",
      "Pairs written (passing filters):        86,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,748,133 bp\n",
      "  Read 1:    25,368,700 bp\n",
      "  Read 2:    25,379,433 bp\n",
      "Total written (filtered):     47,406,595 bp (93.4%)\n",
      "  Read 1:    23,650,091 bp\n",
      "  Read 2:    23,756,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 85948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t33\t0.0\t1\t0 0 33\n",
      "19\t556\t0.0\t1\t0 545 11\n",
      "20\t85135\t0.0\t2\t83668 1447 20\n",
      "21\t177\t0.0\t2\t0 174 3\n",
      "22\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 85427 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t356\t0.0\t1\t0 356\n",
      "19\t84899\t0.0\t1\t80452 4447\n",
      "20\t172\t0.0\t1\t0 172\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S998_290_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S998_291_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S998_290_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S998_291_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S998_290_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S998_291_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S998_290_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S998_291_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,416 reads  @     76.2 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.65 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,416\n",
      "  Read 1 with adapter:                  47,358 (99.9%)\n",
      "  Read 2 with adapter:                  47,077 (99.3%)\n",
      "Pairs written (passing filters):        47,416 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,965,229 bp\n",
      "  Read 1:    13,979,278 bp\n",
      "  Read 2:    13,985,951 bp\n",
      "Total written (filtered):     26,123,903 bp (93.4%)\n",
      "  Read 1:    13,032,285 bp\n",
      "  Read 2:    13,091,618 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 47358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t15\t0.0\t1\t0 0 15\n",
      "19\t264\t0.0\t1\t0 261 3\n",
      "20\t46973\t0.0\t2\t46185 773 15\n",
      "21\t85\t0.0\t2\t0 84 1\n",
      "22\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 47077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t227\t0.0\t1\t0 227\n",
      "19\t46753\t0.0\t1\t44288 2465\n",
      "20\t97\t0.0\t1\t0 97\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S999_292_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S999_293_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S999_292_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S999_293_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S999_292_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-4jbofskk/S999_293_L001_R2_001.fastq.gz --front ^AGGGCAAKYCTGGTGCCAGC -G ^GRCGGTATCTRATCGYCTT /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S999_292_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-yy16d4fo/90f3a12e-c76f-4d75-84c5-32ff4a99957a/data/S999_293_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           389 reads  @    148.6 µs/read;   0.40 M reads/minute\n",
      "Finished in 0.06 s (156 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                389\n",
      "  Read 1 with adapter:                     389 (100.0%)\n",
      "  Read 2 with adapter:                      40 (10.3%)\n",
      "Pairs written (passing filters):           389 (100.0%)\n",
      "\n",
      "Total basepairs processed:       134,804 bp\n",
      "  Read 1:       114,755 bp\n",
      "  Read 2:        20,049 bp\n",
      "Total written (filtered):        126,224 bp (93.6%)\n",
      "  Read 1:       106,975 bp\n",
      "  Read 2:        19,249 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AGGGCAAKYCTGGTGCCAGC; Type: anchored 5'; Length: 20; Trimmed: 389 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "19\t2\t0.0\t1\t0 2\n",
      "20\t386\t0.0\t2\t378 8\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GRCGGTATCTRATCGYCTT; Type: anchored 5'; Length: 19; Trimmed: 40 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t40\t0.0\t1\t0 40\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: 18S-demux-pairedend-trimmed1.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences 18S-paired-end-demux_r1.qza \\\n",
    "    --p-front-f '^AGGGCAAKYCTGGTGCCAGC' \\\n",
    "    --p-front-r '^GRCGGTATCTRATCGYCTT' \\\n",
    "    --o-trimmed-sequences 18S-demux-pairedend-trimmed1.qza \\\n",
    "    --verbose\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-demux-pairedend-trimmed1.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data 18S-demux-pairedend-trimmed1.qza\\\n",
    "  --o-visualization 18S-demux-pairedend-trimmed1.qzv\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-l0rf5pa4')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 7c41d560-f333-41b5-87c3-80ff93c6da0f>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('18S-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: 18S-paired-end-demux_table-dada2_r1.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[Sequence] to: 18S-paired-end-demux_rep-seqs-dada2_r1.qza\u001b[0m\r\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: 18S-paired-end-demux_stats-dada2_r1.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs 18S-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 256 \\\n",
    "  --p-trunc-len-r 184 \\\n",
    "  --p-pooling-method independent \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences 18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --o-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --o-denoising-stats 18S-paired-end-demux_stats-dada2_r1.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-demux-pairedend-trimmed1.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-ap3l0b3i')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 4a947870-6924-4d82-8115-2697ca32a7a9>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#summarize demux stats\n",
    "!qiime demux summarize \\\n",
    "  --i-data 18S-demux-pairedend-trimmed1.qza \\\n",
    "  --o-visualization 18S-demux-pairedend-trimmed1.qzv\n",
    "\n",
    "q2.Visualization.load('18S-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-table_r1.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1h9bm2c8')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 708de170-44df-469f-9421-e152ab3bd1bd>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --o-visualization 18S-table_r1.qzv \\\n",
    "  #--m-sample-metadata-file metadata_18S_v3.txt\n",
    "\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18S-table_r1.qzv')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: 18S-paired-end-demux_rep-seqs-dada2_r1.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-8dkj62h9')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e422077d-f8b3-4fb8-9ae2-88dd5d0758f0>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --o-visualization 18S-paired-end-demux_rep-seqs-dada2_r1.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-qidfp0wh')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e422077d-f8b3-4fb8-9ae2-88dd5d0758f0>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#View the .qzv\n",
    "q2.Visualization.load('18S-paired-end-demux_rep-seqs-dada2_r1.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported 18S-paired-end-demux_table-dada2_r1.qza as BIOMV210DirFmt to directory exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#export table to biom\n",
    "!qiime tools export \\\n",
    "  --input-path 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --output-path exported-feature-table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o 18S_ASVtable.tsv --to-tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "## classify with sklearn SILVA classifier\n",
    "\n",
    "#!qiime feature-classifier classify-sklearn \\\n",
    "#  --i-classifier SILVA_132_99_nb_classifier.qza \\\n",
    "#  --i-reads 18S-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "#  --o-classification 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza\n",
    "#\n",
    "#!qiime metadata tabulate \\\n",
    "#  --m-input-file 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qza \\\n",
    "#  --o-visualization 18SASV-rep-seqs-dada2_sklearnSILVA132_taxonomy.qzv\n",
    "\n",
    "\n",
    "## this doesn't work b/c the clasifier was trained using scikit-learn v 0.19.1 on the jcvi grid, and the local version of sklearn\n",
    "#is 0.23.1. Transferring this fasta to the grid, importing to qiime, and classifying \n",
    "#the seqs using the version of sklearn that's installed there"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "## classify with sklearn MetaZooGene classifier\n",
    "\n",
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /Users/stephaniematthews/Documents/databases/metazoogene_steph_edits/MZG-zhan-classifier_r_trimmed.qza \\\n",
    "  --i-reads 18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --o-classification 18SASV-rep-seqs-dada2_sklearnMZG_taxonomy_trimmed.qza\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file 18SASV-rep-seqs-dada2_sklearnMZG_taxonomy_trimmed.qza \\\n",
    "  --o-visualization 18SASV-rep-seqs-dada2_sklearnMZG_taxonomy_trimmed.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('18SASV-rep-seqs-dada2_sklearnMZG_taxonomy_trimmed.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/chimeras.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/nonchimeras.qza\u001b[0m\n",
      "\u001b[32mSaved UchimeStats to: uchime-dn-out/stats.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: uchime-dn-out/stats.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "##detect chimeras\n",
    "\n",
    "!qiime vsearch uchime-denovo \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --i-sequences 18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --output-dir uchime-dn-out\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file uchime-dn-out/stats.qza \\\n",
    "  --o-visualization uchime-dn-out/stats.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-74m0uug5')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 8ff9a369-bbce-4d65-9023-a9bc77e0688e>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('uchime-dn-out/stats.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: uchime-dn-out/table-nonchimeric-wo-borderline.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: uchime-dn-out/table-nonchimeric-wo-borderline.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "##filter out detected chimeras\n",
    "\n",
    "!qiime feature-table filter-features \\\n",
    "  --i-table 18S-paired-end-demux_table-dada2_r1.qza \\\n",
    "  --m-metadata-file uchime-dn-out/nonchimeras.qza \\\n",
    "  --o-filtered-table uchime-dn-out/table-nonchimeric-wo-borderline.qza\n",
    "!qiime feature-table filter-seqs \\\n",
    "  --i-data 18S-paired-end-demux_rep-seqs-dada2_r1.qza \\\n",
    "  --m-metadata-file uchime-dn-out/nonchimeras.qza \\\n",
    "  --o-filtered-data uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza\n",
    "!qiime feature-table summarize \\\n",
    "  --i-table uchime-dn-out/table-nonchimeric-wo-borderline.qza \\\n",
    "  --o-visualization uchime-dn-out/table-nonchimeric-wo-borderline.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-hkcsqzy3')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: bc98aa39-dc3f-4049-8488-9920cb71dbef>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "##view the seqs to download as fasta\n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza \\\n",
    "  --o-visualization uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported uchime-dn-out/table-nonchimeric-wo-borderline.qza as BIOMV210DirFmt to directory exported-feature-table-nonchimerica-uchime\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#export table to biom\n",
    "!qiime tools export \\\n",
    "  --input-path uchime-dn-out/table-nonchimeric-wo-borderline.qza \\\n",
    "  --output-path exported-feature-table-nonchimerica-uchime\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/18S_ASVtable.tsv --to-tsv\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\"\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/feature-table_json.biom --to-json --table-type=\"OTU table\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
