{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import the qiime2 python module to allow for inline visualizations\n",
    "\n",
    "import qiime2 as q2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#which version of qiime2 and python are we using?\n",
    "\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First, import the paired forward and reverse sequences with a manifest file. Manifest prepared by listing all files in bash and formatting in Excel. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported leray_manifest_runbb_v2_grid.csv as PairedEndFastqManifestPhred33 to COI_1-paired-end-demux.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path leray_manifest_runbb_v2_grid.csv \\\n",
    "  --output-path COI_1-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Trim reverse primers\n",
    "See: https://docs.qiime2.org/2020.6/plugins/available/cutadapt/trim-paired/\n",
    "default is to match IUPAC wildcards ( --p-match-adapter-wildcards T )\n",
    "- these should be the F/R marker primers\n",
    "- cannot handle inosine bases; using N instead\n",
    "- f primer/ r primer\n",
    "- or could just trim off length of primer+barcode? this provides a test of primer accuracy/error \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S701_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S701_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S701_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S701_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S701_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S701_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S701_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S701_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        35,223 reads  @    109.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 3.85 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,223\n",
      "  Read 1 with adapter:                  35,164 (99.8%)\n",
      "  Read 2 with adapter:                  34,841 (98.9%)\n",
      "Pairs written (passing filters):        35,223 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,765,985 bp\n",
      "  Read 1:    10,381,253 bp\n",
      "  Read 2:    10,384,732 bp\n",
      "Total written (filtered):     18,946,355 bp (91.2%)\n",
      "  Read 1:     9,467,242 bp\n",
      "  Read 2:     9,479,113 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t450\t0.0\t2\t0 436 14\n",
      "26\t34492\t0.0\t2\t33504 964 24\n",
      "27\t159\t0.0\t2\t0 156 3\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t378\t0.0\t2\t0 358 20\n",
      "26\t34302\t0.0\t2\t31767 2488 47\n",
      "27\t147\t0.0\t2\t0 134 13\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S702_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S702_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S702_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S702_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S702_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S702_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S702_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S702_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:07        73,432 reads  @    101.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 7.47 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,432\n",
      "  Read 1 with adapter:                  73,363 (99.9%)\n",
      "  Read 2 with adapter:                  73,210 (99.7%)\n",
      "Pairs written (passing filters):        73,432 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,306,684 bp\n",
      "  Read 1:    21,654,700 bp\n",
      "  Read 2:    21,651,984 bp\n",
      "Total written (filtered):     39,496,442 bp (91.2%)\n",
      "  Read 1:    19,747,618 bp\n",
      "  Read 2:    19,748,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t831\t0.0\t2\t0 816 15\n",
      "26\t72039\t0.0\t2\t70608 1405 26\n",
      "27\t379\t0.0\t2\t0 370 9\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 73210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t580\t0.0\t2\t0 536 44\n",
      "26\t72293\t0.0\t2\t66622 5468 203\n",
      "27\t298\t0.0\t2\t0 272 26\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S703_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S703_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S703_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S703_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S703_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S703_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S703_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S703_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,041 reads  @     87.6 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.57 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,041\n",
      "  Read 1 with adapter:                  51,981 (99.9%)\n",
      "  Read 2 with adapter:                  51,952 (99.8%)\n",
      "Pairs written (passing filters):        52,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,694,719 bp\n",
      "  Read 1:    15,345,537 bp\n",
      "  Read 2:    15,349,182 bp\n",
      "Total written (filtered):     27,992,824 bp (91.2%)\n",
      "  Read 1:    13,994,280 bp\n",
      "  Read 2:    13,998,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t604\t0.0\t2\t0 584 20\n",
      "26\t51010\t0.0\t2\t49929 1063 18\n",
      "27\t283\t0.0\t2\t0 275 8\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t337\t0.0\t2\t0 313 24\n",
      "26\t51351\t0.0\t2\t47205 4002 144\n",
      "27\t237\t0.0\t2\t0 225 12\n",
      "28\t10\t0.0\t2\t0 0 10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S704_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S704_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S704_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S704_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S704_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S704_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S704_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S704_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,048 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 2.67 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,048\n",
      "  Read 1 with adapter:                  30,041 (100.0%)\n",
      "  Read 2 with adapter:                  29,768 (99.1%)\n",
      "Pairs written (passing filters):        30,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,693,590 bp\n",
      "  Read 1:     8,864,145 bp\n",
      "  Read 2:     8,829,445 bp\n",
      "Total written (filtered):     16,138,682 bp (91.2%)\n",
      "  Read 1:     8,083,152 bp\n",
      "  Read 2:     8,055,530 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30041 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t309\t0.0\t2\t0 295 14\n",
      "26\t29502\t0.0\t2\t28892 599 11\n",
      "27\t136\t0.0\t2\t0 131 5\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t196\t0.0\t2\t0 176 20\n",
      "26\t29400\t0.0\t2\t26643 2557 200\n",
      "27\t153\t0.0\t2\t0 135 18\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S706_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S706_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S706_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S706_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S706_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S706_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S706_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S706_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,560 reads  @     86.9 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.79 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,560\n",
      "  Read 1 with adapter:                  43,536 (99.9%)\n",
      "  Read 2 with adapter:                  43,435 (99.7%)\n",
      "Pairs written (passing filters):        43,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,695,254 bp\n",
      "  Read 1:    12,849,807 bp\n",
      "  Read 2:    12,845,447 bp\n",
      "Total written (filtered):     23,434,295 bp (91.2%)\n",
      "  Read 1:    11,717,997 bp\n",
      "  Read 2:    11,716,298 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43536 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t472\t0.0\t2\t0 461 11\n",
      "26\t42759\t0.0\t2\t41896 849 14\n",
      "27\t228\t0.0\t2\t0 223 5\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t338\t0.0\t2\t0 318 20\n",
      "26\t42878\t0.0\t2\t39418 3329 131\n",
      "27\t189\t0.0\t2\t0 175 14\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S707_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S707_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S707_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S707_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S707_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S707_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S707_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S707_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,725 reads  @     89.6 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.66 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,725\n",
      "  Read 1 with adapter:                  40,709 (100.0%)\n",
      "  Read 2 with adapter:                  40,606 (99.7%)\n",
      "Pairs written (passing filters):        40,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,020,789 bp\n",
      "  Read 1:    12,013,608 bp\n",
      "  Read 2:    12,007,181 bp\n",
      "Total written (filtered):     21,906,880 bp (91.2%)\n",
      "  Read 1:    10,955,343 bp\n",
      "  Read 2:    10,951,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40709 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t443\t0.0\t2\t0 436 7\n",
      "26\t39990\t0.0\t2\t39184 794 12\n",
      "27\t198\t0.0\t2\t0 187 11\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t307\t0.0\t2\t0 280 27\n",
      "26\t40107\t0.0\t2\t36894 3056 157\n",
      "27\t157\t0.0\t2\t0 152 5\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S708_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S708_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S708_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S708_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S708_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S708_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S708_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S708_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,153 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 3.47 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,153\n",
      "  Read 1 with adapter:                  35,133 (99.9%)\n",
      "  Read 2 with adapter:                  35,089 (99.8%)\n",
      "Pairs written (passing filters):        35,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,738,524 bp\n",
      "  Read 1:    10,368,991 bp\n",
      "  Read 2:    10,369,533 bp\n",
      "Total written (filtered):     18,912,997 bp (91.2%)\n",
      "  Read 1:     9,455,710 bp\n",
      "  Read 2:     9,457,287 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t416\t0.0\t2\t0 403 13\n",
      "26\t34469\t0.0\t2\t33606 833 30\n",
      "27\t185\t0.0\t2\t0 175 10\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35089 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t257\t0.0\t2\t0 229 28\n",
      "26\t34647\t0.0\t2\t31988 2577 82\n",
      "27\t145\t0.0\t2\t0 133 12\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S709_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S709_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S709_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S709_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S709_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S709_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S709_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S709_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        26,844 reads  @    106.4 µs/read;   0.56 M reads/minute\n",
      "Finished in 2.86 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,844\n",
      "  Read 1 with adapter:                  26,820 (99.9%)\n",
      "  Read 2 with adapter:                  26,690 (99.4%)\n",
      "Pairs written (passing filters):        26,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,835,316 bp\n",
      "  Read 1:     7,918,557 bp\n",
      "  Read 2:     7,916,759 bp\n",
      "Total written (filtered):     14,444,456 bp (91.2%)\n",
      "  Read 1:     7,221,479 bp\n",
      "  Read 2:     7,222,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t381\t0.0\t2\t0 368 13\n",
      "26\t26255\t0.0\t2\t25573 670 12\n",
      "27\t133\t0.0\t2\t0 130 3\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t270\t0.0\t2\t0 250 20\n",
      "26\t26268\t0.0\t2\t24341 1888 39\n",
      "27\t124\t0.0\t2\t0 110 14\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S710_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S710_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S710_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S710_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S710_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S710_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S710_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S710_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        33,794 reads  @    104.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 3.52 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,794\n",
      "  Read 1 with adapter:                  33,767 (99.9%)\n",
      "  Read 2 with adapter:                  33,703 (99.7%)\n",
      "Pairs written (passing filters):        33,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,934,051 bp\n",
      "  Read 1:     9,967,947 bp\n",
      "  Read 2:     9,966,104 bp\n",
      "Total written (filtered):     18,180,016 bp (91.2%)\n",
      "  Read 1:     9,090,039 bp\n",
      "  Read 2:     9,089,977 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33767 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t498\t0.0\t2\t0 488 10\n",
      "26\t33042\t0.0\t2\t32270 758 14\n",
      "27\t167\t0.0\t2\t0 161 6\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t292\t0.0\t2\t0 272 20\n",
      "26\t33240\t0.0\t2\t30579 2552 109\n",
      "27\t149\t0.0\t2\t0 137 12\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S711_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S711_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S711_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S711_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S711_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S711_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S711_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S711_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        57,455 reads  @     97.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 5.62 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,455\n",
      "  Read 1 with adapter:                  57,405 (99.9%)\n",
      "  Read 2 with adapter:                  57,351 (99.8%)\n",
      "Pairs written (passing filters):        57,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,896,387 bp\n",
      "  Read 1:    16,948,480 bp\n",
      "  Read 2:    16,947,907 bp\n",
      "Total written (filtered):     30,913,144 bp (91.2%)\n",
      "  Read 1:    15,456,227 bp\n",
      "  Read 2:    15,456,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57405 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t777\t0.0\t2\t0 754 23\n",
      "26\t56120\t0.0\t2\t54832 1260 28\n",
      "27\t388\t0.0\t2\t0 380 8\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57351 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t404\t0.0\t2\t0 374 30\n",
      "26\t56622\t0.0\t2\t52182 4273 167\n",
      "27\t298\t0.0\t2\t0 280 18\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S712_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S712_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S712_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S712_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S712_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S712_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S712_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S712_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,779 reads  @     88.0 µs/read;   0.68 M reads/minute\n",
      "Finished in 2.54 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,779\n",
      "  Read 1 with adapter:                  28,755 (99.9%)\n",
      "  Read 2 with adapter:                  27,924 (97.0%)\n",
      "Pairs written (passing filters):        28,777 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,791,818 bp\n",
      "  Read 1:     8,489,539 bp\n",
      "  Read 2:     8,302,279 bp\n",
      "Total written (filtered):     15,317,653 bp (91.2%)\n",
      "  Read 1:     7,741,389 bp\n",
      "  Read 2:     7,576,264 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t319\t0.0\t2\t0 306 13\n",
      "26\t28155\t0.0\t2\t27549 594 12\n",
      "27\t181\t0.0\t2\t0 173 8\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27924 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t155\t0.0\t2\t0 143 12\n",
      "26\t27607\t0.0\t2\t25013 2401 193\n",
      "27\t150\t0.0\t2\t0 127 23\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S714_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S714_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S714_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S714_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S714_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S714_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S714_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S714_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,472 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.22 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,472\n",
      "  Read 1 with adapter:                  39,448 (99.9%)\n",
      "  Read 2 with adapter:                  39,327 (99.6%)\n",
      "Pairs written (passing filters):        39,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,279,737 bp\n",
      "  Read 1:    11,644,211 bp\n",
      "  Read 2:    11,635,526 bp\n",
      "Total written (filtered):     21,231,712 bp (91.2%)\n",
      "  Read 1:    10,618,533 bp\n",
      "  Read 2:    10,613,179 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39448 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t500\t0.0\t2\t0 488 12\n",
      "26\t38651\t0.0\t2\t37852 790 9\n",
      "27\t229\t0.0\t2\t0 222 7\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t345\t0.0\t2\t0 315 30\n",
      "26\t38785\t0.0\t2\t35739 2909 137\n",
      "27\t144\t0.0\t2\t0 133 11\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S715_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S715_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S715_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S715_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S715_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S715_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S715_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S715_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,150 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 2.95 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,150\n",
      "  Read 1 with adapter:                  34,123 (99.9%)\n",
      "  Read 2 with adapter:                  34,009 (99.6%)\n",
      "Pairs written (passing filters):        34,150 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,143,301 bp\n",
      "  Read 1:    10,074,236 bp\n",
      "  Read 2:    10,069,065 bp\n",
      "Total written (filtered):     18,372,164 bp (91.2%)\n",
      "  Read 1:     9,187,218 bp\n",
      "  Read 2:     9,184,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t470\t0.0\t2\t0 462 8\n",
      "26\t33350\t0.0\t2\t32688 652 10\n",
      "27\t216\t0.0\t2\t0 206 10\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34009 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t264\t0.0\t2\t0 241 23\n",
      "26\t33606\t0.0\t2\t30836 2620 150\n",
      "27\t117\t0.0\t2\t0 108 9\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S716_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S716_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S716_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S716_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S716_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S716_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S716_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S716_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,935 reads  @     90.9 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.46 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,935\n",
      "  Read 1 with adapter:                  37,910 (99.9%)\n",
      "  Read 2 with adapter:                  37,878 (99.8%)\n",
      "Pairs written (passing filters):        37,934 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,380,787 bp\n",
      "  Read 1:    11,190,442 bp\n",
      "  Read 2:    11,190,345 bp\n",
      "Total written (filtered):     20,410,410 bp (91.2%)\n",
      "  Read 1:    10,205,006 bp\n",
      "  Read 2:    10,205,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t477\t0.0\t2\t0 468 9\n",
      "26\t37104\t0.0\t2\t36347 739 18\n",
      "27\t249\t0.0\t2\t0 245 4\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37878 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t312\t0.0\t2\t0 289 23\n",
      "26\t37392\t0.0\t2\t34506 2788 98\n",
      "27\t140\t0.0\t2\t0 130 10\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S717_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S717_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S717_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S717_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S717_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S717_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S717_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S717_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        37,850 reads  @    101.6 µs/read;   0.59 M reads/minute\n",
      "Finished in 3.86 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,850\n",
      "  Read 1 with adapter:                  37,766 (99.8%)\n",
      "  Read 2 with adapter:                  37,497 (99.1%)\n",
      "Pairs written (passing filters):        37,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,318,619 bp\n",
      "  Read 1:    11,157,887 bp\n",
      "  Read 2:    11,160,732 bp\n",
      "Total written (filtered):     20,362,213 bp (91.2%)\n",
      "  Read 1:    10,176,188 bp\n",
      "  Read 2:    10,186,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37766 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t395\t0.0\t2\t0 389 6\n",
      "26\t37166\t0.0\t2\t36313 832 21\n",
      "27\t156\t0.0\t2\t0 153 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37497 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t390\t0.0\t2\t0 353 37\n",
      "26\t36917\t0.0\t2\t34081 2737 99\n",
      "27\t169\t0.0\t2\t0 154 15\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S718_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S718_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S718_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S718_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S718_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S718_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S718_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S718_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,488 reads  @     96.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.80 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,488\n",
      "  Read 1 with adapter:                  39,352 (99.7%)\n",
      "  Read 2 with adapter:                  39,352 (99.7%)\n",
      "Pairs written (passing filters):        39,488 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,281,661 bp\n",
      "  Read 1:    11,635,196 bp\n",
      "  Read 2:    11,646,465 bp\n",
      "Total written (filtered):     21,235,609 bp (91.2%)\n",
      "  Read 1:    10,612,153 bp\n",
      "  Read 2:    10,623,456 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t369\t0.0\t2\t0 366 3\n",
      "26\t38717\t0.0\t2\t37911 791 15\n",
      "27\t192\t0.0\t2\t0 189 3\n",
      "28\t54\t0.0\t2\t0 0 54\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39352 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t300\t0.0\t2\t0 279 21\n",
      "26\t38867\t0.0\t2\t35528 3135 204\n",
      "27\t161\t0.0\t2\t0 145 16\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S719_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S719_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S719_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S719_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S719_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S719_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S719_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S719_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,627 reads  @     86.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.51 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,627\n",
      "  Read 1 with adapter:                  40,262 (99.1%)\n",
      "  Read 2 with adapter:                  40,514 (99.7%)\n",
      "Pairs written (passing filters):        40,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,929,872 bp\n",
      "  Read 1:    11,945,855 bp\n",
      "  Read 2:    11,984,017 bp\n",
      "Total written (filtered):     21,829,868 bp (91.2%)\n",
      "  Read 1:    10,899,128 bp\n",
      "  Read 2:    10,930,740 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40262 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t391\t0.0\t2\t0 383 8\n",
      "26\t39616\t0.0\t2\t38842 763 11\n",
      "27\t160\t0.0\t2\t0 155 5\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40514 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t259\t0.0\t2\t0 238 21\n",
      "26\t40052\t0.0\t2\t36744 3133 175\n",
      "27\t186\t0.0\t2\t0 163 23\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S720_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S720_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S720_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S720_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S720_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S720_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S720_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S720_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        26,720 reads  @    103.8 µs/read;   0.58 M reads/minute\n",
      "Finished in 2.78 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,720\n",
      "  Read 1 with adapter:                  26,608 (99.6%)\n",
      "  Read 2 with adapter:                  26,362 (98.7%)\n",
      "Pairs written (passing filters):        26,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,730,809 bp\n",
      "  Read 1:     7,870,966 bp\n",
      "  Read 2:     7,859,843 bp\n",
      "Total written (filtered):     14,353,465 bp (91.2%)\n",
      "  Read 1:     7,178,999 bp\n",
      "  Read 2:     7,174,466 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26608 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t266\t0.0\t2\t0 260 6\n",
      "26\t26179\t0.0\t2\t25622 548 9\n",
      "27\t118\t0.0\t2\t0 116 2\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t180\t0.0\t2\t0 165 15\n",
      "26\t26022\t0.0\t2\t23342 2430 250\n",
      "27\t147\t0.0\t2\t0 121 26\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S722_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S722_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S722_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S722_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S722_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S722_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S722_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S722_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,800 reads  @     87.9 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.33 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,800\n",
      "  Read 1 with adapter:                  37,745 (99.9%)\n",
      "  Read 2 with adapter:                  37,600 (99.5%)\n",
      "Pairs written (passing filters):        37,800 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,291,493 bp\n",
      "  Read 1:    11,147,617 bp\n",
      "  Read 2:    11,143,876 bp\n",
      "Total written (filtered):     20,332,913 bp (91.2%)\n",
      "  Read 1:    10,166,450 bp\n",
      "  Read 2:    10,166,463 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37745 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t418\t0.0\t2\t0 403 15\n",
      "26\t37092\t0.0\t2\t36253 830 9\n",
      "27\t183\t0.0\t2\t0 178 5\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37600 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t360\t0.0\t2\t0 329 31\n",
      "26\t37036\t0.0\t2\t33802 3010 224\n",
      "27\t175\t0.0\t2\t0 152 23\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S723_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S723_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S723_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S723_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S723_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S723_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S723_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S723_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,386 reads  @     78.2 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.25 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,386\n",
      "  Read 1 with adapter:                  41,361 (99.9%)\n",
      "  Read 2 with adapter:                  41,209 (99.6%)\n",
      "Pairs written (passing filters):        41,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,414,352 bp\n",
      "  Read 1:    12,208,206 bp\n",
      "  Read 2:    12,206,146 bp\n",
      "Total written (filtered):     22,267,941 bp (91.2%)\n",
      "  Read 1:    11,133,150 bp\n",
      "  Read 2:    11,134,791 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41361 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t459\t0.0\t2\t0 448 11\n",
      "26\t40719\t0.0\t2\t39762 935 22\n",
      "27\t149\t0.0\t2\t0 149\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t334\t0.0\t2\t0 302 32\n",
      "26\t40633\t0.0\t2\t37084 3361 188\n",
      "27\t185\t0.0\t2\t0 168 17\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S724_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S724_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S724_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S724_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S724_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S724_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S724_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S724_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,290 reads  @     95.6 µs/read;   0.63 M reads/minute\n",
      "Finished in 3.57 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,290\n",
      "  Read 1 with adapter:                  37,245 (99.9%)\n",
      "  Read 2 with adapter:                  37,182 (99.7%)\n",
      "Pairs written (passing filters):        37,290 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,995,047 bp\n",
      "  Read 1:    10,997,111 bp\n",
      "  Read 2:    10,997,936 bp\n",
      "Total written (filtered):     20,060,166 bp (91.2%)\n",
      "  Read 1:    10,028,869 bp\n",
      "  Read 2:    10,031,297 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t342\t0.0\t2\t0 334 8\n",
      "26\t36677\t0.0\t2\t35858 806 13\n",
      "27\t158\t0.0\t2\t0 153 5\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t270\t0.0\t2\t0 246 24\n",
      "26\t36721\t0.0\t2\t33526 2995 200\n",
      "27\t165\t0.0\t2\t0 151 14\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S725_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S725_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S725_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S725_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S725_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S725_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S725_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S725_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        20,397 reads  @     95.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 1.95 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,397\n",
      "  Read 1 with adapter:                  20,385 (99.9%)\n",
      "  Read 2 with adapter:                  20,251 (99.3%)\n",
      "Pairs written (passing filters):        20,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,031,681 bp\n",
      "  Read 1:     6,017,107 bp\n",
      "  Read 2:     6,014,574 bp\n",
      "Total written (filtered):     10,975,568 bp (91.2%)\n",
      "  Read 1:     5,487,335 bp\n",
      "  Read 2:     5,488,233 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t301\t0.0\t2\t0 292 9\n",
      "26\t19983\t0.0\t2\t19160 794 29\n",
      "27\t79\t0.0\t2\t0 76 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20251 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t259\t0.0\t2\t0 243 16\n",
      "26\t19881\t0.0\t2\t18392 1460 29\n",
      "27\t88\t0.0\t2\t0 87 1\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S726_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S726_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S726_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S726_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S726_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S726_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S726_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S726_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,413 reads  @     89.2 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.34 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,413\n",
      "  Read 1 with adapter:                  37,383 (99.9%)\n",
      "  Read 2 with adapter:                  37,246 (99.6%)\n",
      "Pairs written (passing filters):        37,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,065,028 bp\n",
      "  Read 1:    11,035,606 bp\n",
      "  Read 2:    11,029,422 bp\n",
      "Total written (filtered):     20,124,897 bp (91.2%)\n",
      "  Read 1:    10,063,764 bp\n",
      "  Read 2:    10,061,133 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t410\t0.0\t2\t0 397 13\n",
      "26\t36691\t0.0\t2\t35907 769 15\n",
      "27\t194\t0.0\t2\t0 189 5\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37246 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t298\t0.0\t2\t0 282 16\n",
      "26\t36748\t0.0\t2\t33871 2774 103\n",
      "27\t169\t0.0\t2\t0 152 17\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S727_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S727_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S727_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S727_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S727_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S727_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S727_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S727_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,496 reads  @     85.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.12 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,496\n",
      "  Read 1 with adapter:                  36,470 (99.9%)\n",
      "  Read 2 with adapter:                  36,428 (99.8%)\n",
      "Pairs written (passing filters):        36,496 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,528,833 bp\n",
      "  Read 1:    10,764,632 bp\n",
      "  Read 2:    10,764,201 bp\n",
      "Total written (filtered):     19,633,688 bp (91.2%)\n",
      "  Read 1:     9,816,553 bp\n",
      "  Read 2:     9,817,135 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36470 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t435\t0.0\t2\t0 423 12\n",
      "26\t35774\t0.0\t2\t35012 747 15\n",
      "27\t164\t0.0\t2\t0 160 4\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36428 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t216\t0.0\t2\t0 196 20\n",
      "26\t36027\t0.0\t2\t33315 2641 71\n",
      "27\t172\t0.0\t2\t0 157 15\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S728_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S728_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S728_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S728_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S728_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S728_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S728_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S728_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,461 reads  @     95.9 µs/read;   0.63 M reads/minute\n",
      "Finished in 2.74 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,461\n",
      "  Read 1 with adapter:                  28,433 (99.9%)\n",
      "  Read 2 with adapter:                  28,346 (99.6%)\n",
      "Pairs written (passing filters):        28,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,781,965 bp\n",
      "  Read 1:     8,395,050 bp\n",
      "  Read 2:     8,386,915 bp\n",
      "Total written (filtered):     15,305,899 bp (91.2%)\n",
      "  Read 1:     7,655,908 bp\n",
      "  Read 2:     7,649,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t330\t0.0\t2\t0 320 10\n",
      "26\t27891\t0.0\t2\t27292 588 11\n",
      "27\t150\t0.0\t2\t0 145 5\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t206\t0.0\t2\t0 189 17\n",
      "26\t28000\t0.0\t2\t25650 2237 113\n",
      "27\t126\t0.0\t2\t0 119 7\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S729_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S729_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S729_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S729_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S729_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S729_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S729_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S729_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S730_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S730_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S730_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S730_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S730_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S730_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S730_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S730_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,104 reads  @     79.2 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.81 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,104\n",
      "  Read 1 with adapter:                  48,070 (99.9%)\n",
      "  Read 2 with adapter:                  47,833 (99.4%)\n",
      "Pairs written (passing filters):        48,103 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,344,525 bp\n",
      "  Read 1:    14,190,450 bp\n",
      "  Read 2:    14,154,075 bp\n",
      "Total written (filtered):     25,851,341 bp (91.2%)\n",
      "  Read 1:    12,940,786 bp\n",
      "  Read 2:    12,910,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48070 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t597\t0.0\t2\t0 585 12\n",
      "26\t47244\t0.0\t2\t46190 1035 19\n",
      "27\t190\t0.0\t2\t0 188 2\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47833 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t399\t0.0\t2\t0 369 30\n",
      "26\t47180\t0.0\t2\t43547 3508 125\n",
      "27\t191\t0.0\t2\t0 180 11\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S731_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S731_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S731_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S731_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S731_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S731_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S731_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S731_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        18,642 reads  @    106.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 2.00 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,642\n",
      "  Read 1 with adapter:                  18,634 (100.0%)\n",
      "  Read 2 with adapter:                  18,539 (99.4%)\n",
      "Pairs written (passing filters):        18,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,996,091 bp\n",
      "  Read 1:     5,499,382 bp\n",
      "  Read 2:     5,496,709 bp\n",
      "Total written (filtered):     10,029,793 bp (91.2%)\n",
      "  Read 1:     5,015,039 bp\n",
      "  Read 2:     5,014,754 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 18634 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t247\t0.0\t2\t0 238 9\n",
      "26\t18262\t0.0\t2\t17711 532 19\n",
      "27\t100\t0.0\t2\t0 97 3\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 18539 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t156\t0.0\t2\t0 147 9\n",
      "26\t18282\t0.0\t2\t15814 2219 249\n",
      "27\t89\t0.0\t2\t0 72 17\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S732_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S732_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S732_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S732_57_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S732_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S732_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S732_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S732_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,235 reads  @     90.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.63 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,235\n",
      "  Read 1 with adapter:                  40,204 (99.9%)\n",
      "  Read 2 with adapter:                  40,193 (99.9%)\n",
      "Pairs written (passing filters):        40,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,737,381 bp\n",
      "  Read 1:    11,868,664 bp\n",
      "  Read 2:    11,868,717 bp\n",
      "Total written (filtered):     21,647,527 bp (91.2%)\n",
      "  Read 1:    10,823,677 bp\n",
      "  Read 2:    10,823,850 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t493\t0.0\t2\t0 486 7\n",
      "26\t39478\t0.0\t2\t38575 885 18\n",
      "27\t178\t0.0\t2\t0 170 8\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t297\t0.0\t2\t0 276 21\n",
      "26\t39720\t0.0\t2\t36821 2822 77\n",
      "27\t154\t0.0\t2\t0 145 9\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S733_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S733_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S733_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S733_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S733_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S733_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S733_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S733_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        30,949 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.01 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,949\n",
      "  Read 1 with adapter:                  30,933 (99.9%)\n",
      "  Read 2 with adapter:                  30,746 (99.3%)\n",
      "Pairs written (passing filters):        30,949 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,255,877 bp\n",
      "  Read 1:     9,129,398 bp\n",
      "  Read 2:     9,126,479 bp\n",
      "Total written (filtered):     16,652,604 bp (91.2%)\n",
      "  Read 1:     8,325,298 bp\n",
      "  Read 2:     8,327,306 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 30933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t408\t0.0\t2\t0 395 13\n",
      "26\t30289\t0.0\t2\t29455 810 24\n",
      "27\t146\t0.0\t2\t0 141 5\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30746 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t335\t0.0\t2\t0 308 27\n",
      "26\t30247\t0.0\t2\t28052 2154 41\n",
      "27\t140\t0.0\t2\t0 126 14\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S734_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S734_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S734_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S734_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S734_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S734_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S734_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S734_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,945 reads  @     80.1 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.04 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,945\n",
      "  Read 1 with adapter:                  37,929 (100.0%)\n",
      "  Read 2 with adapter:                  37,860 (99.8%)\n",
      "Pairs written (passing filters):        37,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,384,254 bp\n",
      "  Read 1:    11,193,601 bp\n",
      "  Read 2:    11,190,653 bp\n",
      "Total written (filtered):     20,414,081 bp (91.2%)\n",
      "  Read 1:    10,207,634 bp\n",
      "  Read 2:    10,206,447 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37929 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t473\t0.0\t2\t0 465 8\n",
      "26\t37110\t0.0\t2\t36335 759 16\n",
      "27\t286\t0.0\t2\t0 283 3\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37860 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t321\t0.0\t2\t0 302 19\n",
      "26\t37353\t0.0\t2\t34585 2695 73\n",
      "27\t177\t0.0\t2\t0 167 10\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S735_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S735_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S735_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S735_63_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S735_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S735_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S735_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S735_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,370 reads  @     84.6 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.10 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,370\n",
      "  Read 1 with adapter:                  48,346 (100.0%)\n",
      "  Read 2 with adapter:                  48,288 (99.8%)\n",
      "Pairs written (passing filters):        48,370 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,536,521 bp\n",
      "  Read 1:    14,268,861 bp\n",
      "  Read 2:    14,267,660 bp\n",
      "Total written (filtered):     26,024,164 bp (91.2%)\n",
      "  Read 1:    13,011,978 bp\n",
      "  Read 2:    13,012,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t546\t0.0\t2\t0 534 12\n",
      "26\t47379\t0.0\t2\t46407 954 18\n",
      "27\t341\t0.0\t2\t0 335 6\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48288 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t333\t0.0\t2\t0 298 35\n",
      "26\t47698\t0.0\t2\t44038 3541 119\n",
      "27\t155\t0.0\t2\t0 143 12\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S736_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S736_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S736_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S736_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S736_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S736_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S736_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S736_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        26,437 reads  @    103.1 µs/read;   0.58 M reads/minute\n",
      "Finished in 2.73 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,437\n",
      "  Read 1 with adapter:                  26,431 (100.0%)\n",
      "  Read 2 with adapter:                  26,154 (98.9%)\n",
      "Pairs written (passing filters):        26,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,559,397 bp\n",
      "  Read 1:     7,798,700 bp\n",
      "  Read 2:     7,760,697 bp\n",
      "Total written (filtered):     14,191,559 bp (91.2%)\n",
      "  Read 1:     7,110,834 bp\n",
      "  Read 2:     7,080,725 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26431 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t318\t0.0\t2\t0 309 9\n",
      "26\t25918\t0.0\t2\t25348 559 11\n",
      "27\t155\t0.0\t2\t0 151 4\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t176\t0.0\t2\t0 165 11\n",
      "26\t25823\t0.0\t2\t23585 2145 93\n",
      "27\t126\t0.0\t2\t0 114 12\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S738_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S738_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S738_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S738_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S738_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S738_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S738_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S738_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,449 reads  @    105.1 µs/read;   0.57 M reads/minute\n",
      "Finished in 4.68 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,449\n",
      "  Read 1 with adapter:                  44,420 (99.9%)\n",
      "  Read 2 with adapter:                  44,320 (99.7%)\n",
      "Pairs written (passing filters):        44,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,216,211 bp\n",
      "  Read 1:    13,110,278 bp\n",
      "  Read 2:    13,105,933 bp\n",
      "Total written (filtered):     23,909,409 bp (91.2%)\n",
      "  Read 1:    11,955,633 bp\n",
      "  Read 2:    11,953,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44420 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t556\t0.0\t2\t0 539 17\n",
      "26\t43572\t0.0\t2\t42632 928 12\n",
      "27\t215\t0.0\t2\t0 211 4\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44320 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t353\t0.0\t2\t0 324 29\n",
      "26\t43763\t0.0\t2\t40464 3205 94\n",
      "27\t170\t0.0\t2\t0 156 14\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S739_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S739_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S739_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S739_69_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S739_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S739_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S739_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S739_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,459 reads  @    107.2 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.77 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,459\n",
      "  Read 1 with adapter:                  44,441 (100.0%)\n",
      "  Read 2 with adapter:                  44,300 (99.6%)\n",
      "Pairs written (passing filters):        44,459 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,227,665 bp\n",
      "  Read 1:    13,115,167 bp\n",
      "  Read 2:    13,112,498 bp\n",
      "Total written (filtered):     23,920,738 bp (91.2%)\n",
      "  Read 1:    11,959,941 bp\n",
      "  Read 2:    11,960,797 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44441 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t547\t0.0\t2\t0 536 11\n",
      "26\t43579\t0.0\t2\t42662 905 12\n",
      "27\t223\t0.0\t2\t0 216 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44300 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t325\t0.0\t2\t0 295 30\n",
      "26\t43754\t0.0\t2\t40086 3499 169\n",
      "27\t184\t0.0\t2\t0 170 14\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S740_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S740_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S740_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S740_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S740_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S740_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S740_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S740_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        54,222 reads  @     97.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 5.28 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,222\n",
      "  Read 1 with adapter:                  54,200 (100.0%)\n",
      "  Read 2 with adapter:                  54,127 (99.8%)\n",
      "Pairs written (passing filters):        54,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,988,958 bp\n",
      "  Read 1:    15,995,183 bp\n",
      "  Read 2:    15,993,775 bp\n",
      "Total written (filtered):     29,172,722 bp (91.2%)\n",
      "  Read 1:    14,586,155 bp\n",
      "  Read 2:    14,586,567 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54200 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t636\t0.0\t2\t0 621 15\n",
      "26\t53130\t0.0\t2\t51930 1166 34\n",
      "27\t292\t0.0\t2\t0 289 3\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54127 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t395\t0.0\t2\t0 367 28\n",
      "26\t53447\t0.0\t2\t49386 3939 122\n",
      "27\t173\t0.0\t2\t0 157 16\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S741_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S741_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S741_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S741_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S741_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S741_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S741_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S741_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,028 reads  @     97.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 2.85 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,028\n",
      "  Read 1 with adapter:                  29,011 (99.9%)\n",
      "  Read 2 with adapter:                  28,820 (99.3%)\n",
      "Pairs written (passing filters):        29,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,122,926 bp\n",
      "  Read 1:     8,562,961 bp\n",
      "  Read 2:     8,559,965 bp\n",
      "Total written (filtered):     15,619,525 bp (91.2%)\n",
      "  Read 1:     7,808,714 bp\n",
      "  Read 2:     7,810,811 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t287\t0.0\t2\t0 277 10\n",
      "26\t28498\t0.0\t2\t27593 882 23\n",
      "27\t152\t0.0\t2\t0 147 5\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28820 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t276\t0.0\t2\t0 248 28\n",
      "26\t28393\t0.0\t2\t26275 2069 49\n",
      "27\t132\t0.0\t2\t0 122 10\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S742_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S742_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S742_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S742_75_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S742_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S742_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S742_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S742_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        40,045 reads  @    101.3 µs/read;   0.59 M reads/minute\n",
      "Finished in 4.06 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,045\n",
      "  Read 1 with adapter:                  40,021 (99.9%)\n",
      "  Read 2 with adapter:                  39,949 (99.8%)\n",
      "Pairs written (passing filters):        40,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,621,966 bp\n",
      "  Read 1:    11,812,785 bp\n",
      "  Read 2:    11,809,181 bp\n",
      "Total written (filtered):     21,542,917 bp (91.2%)\n",
      "  Read 1:    10,772,318 bp\n",
      "  Read 2:    10,770,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40021 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t413\t0.0\t2\t0 404 9\n",
      "26\t39309\t0.0\t2\t38364 931 14\n",
      "27\t208\t0.0\t2\t0 203 5\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39949 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t293\t0.0\t2\t0 269 24\n",
      "26\t39427\t0.0\t2\t36368 2957 102\n",
      "27\t209\t0.0\t2\t0 180 29\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S743_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S743_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S743_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S743_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S743_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S743_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S743_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S743_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,110 reads  @     86.8 µs/read;   0.69 M reads/minute\n",
      "Finished in 3.75 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,110\n",
      "  Read 1 with adapter:                  43,091 (100.0%)\n",
      "  Read 2 with adapter:                  42,999 (99.7%)\n",
      "Pairs written (passing filters):        43,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,431,241 bp\n",
      "  Read 1:    12,716,935 bp\n",
      "  Read 2:    12,714,306 bp\n",
      "Total written (filtered):     23,192,997 bp (91.2%)\n",
      "  Read 1:    11,596,619 bp\n",
      "  Read 2:    11,596,378 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43091 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t350\t0.0\t2\t0 342 8\n",
      "26\t42430\t0.0\t2\t41616 805 9\n",
      "27\t234\t0.0\t2\t0 230 4\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42999 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t284\t0.0\t2\t0 254 30\n",
      "26\t42494\t0.0\t2\t39227 3152 115\n",
      "27\t156\t0.0\t2\t0 150 6\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S744_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S744_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S744_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S744_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S744_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S744_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S744_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S744_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,792 reads  @     97.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 2.90 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,792\n",
      "  Read 1 with adapter:                  29,769 (99.9%)\n",
      "  Read 2 with adapter:                  29,366 (98.6%)\n",
      "Pairs written (passing filters):        29,789 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,519,625 bp\n",
      "  Read 1:     8,788,627 bp\n",
      "  Read 2:     8,730,998 bp\n",
      "Total written (filtered):     15,981,459 bp (91.2%)\n",
      "  Read 1:     8,013,922 bp\n",
      "  Read 2:     7,967,537 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29769 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t276\t0.0\t2\t0 266 10\n",
      "26\t29275\t0.0\t2\t28652 612 11\n",
      "27\t172\t0.0\t2\t0 167 5\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t191\t0.0\t2\t0 166 25\n",
      "26\t29017\t0.0\t2\t26479 2388 150\n",
      "27\t136\t0.0\t2\t0 121 15\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S746_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S746_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S746_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S746_81_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S746_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S746_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S746_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S746_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,920 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.41 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,920\n",
      "  Read 1 with adapter:                  40,899 (99.9%)\n",
      "  Read 2 with adapter:                  40,791 (99.7%)\n",
      "Pairs written (passing filters):        40,920 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,133,300 bp\n",
      "  Read 1:    12,071,302 bp\n",
      "  Read 2:    12,061,998 bp\n",
      "Total written (filtered):     22,009,600 bp (91.2%)\n",
      "  Read 1:    11,008,002 bp\n",
      "  Read 2:    11,001,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t350\t0.0\t2\t0 337 13\n",
      "26\t40271\t0.0\t2\t39440 818 13\n",
      "27\t224\t0.0\t2\t0 220 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t345\t0.0\t2\t0 319 26\n",
      "26\t40245\t0.0\t2\t37042 3099 104\n",
      "27\t155\t0.0\t2\t0 143 12\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S747_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S747_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S747_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S747_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S747_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S747_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S747_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S747_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,213 reads  @    103.3 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.06 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,213\n",
      "  Read 1 with adapter:                  39,187 (99.9%)\n",
      "  Read 2 with adapter:                  39,110 (99.7%)\n",
      "Pairs written (passing filters):        39,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,131,791 bp\n",
      "  Read 1:    11,567,681 bp\n",
      "  Read 2:    11,564,110 bp\n",
      "Total written (filtered):     21,096,251 bp (91.2%)\n",
      "  Read 1:    10,548,921 bp\n",
      "  Read 2:    10,547,330 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39187 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t381\t0.0\t2\t0 373 8\n",
      "26\t38544\t0.0\t2\t37710 820 14\n",
      "27\t197\t0.0\t2\t0 192 5\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t287\t0.0\t2\t0 264 23\n",
      "26\t38619\t0.0\t2\t35596 2896 127\n",
      "27\t153\t0.0\t2\t0 141 12\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S748_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S748_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S748_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S748_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S748_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S748_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S748_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S748_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        35,687 reads  @    102.0 µs/read;   0.59 M reads/minute\n",
      "Finished in 3.64 s (102 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,687\n",
      "  Read 1 with adapter:                  35,664 (99.9%)\n",
      "  Read 2 with adapter:                  35,640 (99.9%)\n",
      "Pairs written (passing filters):        35,687 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,054,744 bp\n",
      "  Read 1:    10,527,176 bp\n",
      "  Read 2:    10,527,568 bp\n",
      "Total written (filtered):     19,200,982 bp (91.2%)\n",
      "  Read 1:     9,599,940 bp\n",
      "  Read 2:     9,601,042 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35664 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t322\t0.0\t2\t0 313 9\n",
      "26\t35092\t0.0\t2\t34299 774 19\n",
      "27\t178\t0.0\t2\t0 174 4\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35640 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t284\t0.0\t2\t0 260 24\n",
      "26\t35161\t0.0\t2\t32519 2581 61\n",
      "27\t160\t0.0\t2\t0 149 11\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S749_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S749_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S749_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S749_87_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S749_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S749_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S749_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S749_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,295 reads  @     99.4 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.61 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,295\n",
      "  Read 1 with adapter:                  46,279 (100.0%)\n",
      "  Read 2 with adapter:                  46,118 (99.6%)\n",
      "Pairs written (passing filters):        46,295 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,304,578 bp\n",
      "  Read 1:    13,655,678 bp\n",
      "  Read 2:    13,648,900 bp\n",
      "Total written (filtered):     24,902,636 bp (91.2%)\n",
      "  Read 1:    12,452,617 bp\n",
      "  Read 2:    12,450,019 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46279 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t522\t0.0\t2\t0 510 12\n",
      "26\t45432\t0.0\t2\t44281 1129 22\n",
      "27\t229\t0.0\t2\t0 219 10\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46118 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t495\t0.0\t2\t0 446 49\n",
      "26\t45296\t0.0\t2\t41113 3928 255\n",
      "27\t258\t0.0\t2\t0 231 27\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S750_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S750_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S750_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S750_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S750_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S750_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S750_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S750_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,821 reads  @     85.8 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.94 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,821\n",
      "  Read 1 with adapter:                  45,803 (100.0%)\n",
      "  Read 2 with adapter:                  45,734 (99.8%)\n",
      "Pairs written (passing filters):        45,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,032,043 bp\n",
      "  Read 1:    13,516,163 bp\n",
      "  Read 2:    13,515,880 bp\n",
      "Total written (filtered):     24,652,331 bp (91.2%)\n",
      "  Read 1:    12,325,459 bp\n",
      "  Read 2:    12,326,872 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t505\t0.0\t2\t0 491 14\n",
      "26\t44960\t0.0\t2\t43836 1102 22\n",
      "27\t277\t0.0\t2\t0 267 10\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45734 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t366\t0.0\t2\t0 337 29\n",
      "26\t45072\t0.0\t2\t41499 3439 134\n",
      "27\t238\t0.0\t2\t0 217 21\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S751_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S751_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S751_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S751_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S751_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S751_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S751_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S751_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           875 reads  @    153.3 µs/read;   0.39 M reads/minute\n",
      "Finished in 0.14 s (156 us/read; 0.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                875\n",
      "  Read 1 with adapter:                     380 (43.4%)\n",
      "  Read 2 with adapter:                     821 (93.8%)\n",
      "Pairs written (passing filters):           875 (100.0%)\n",
      "\n",
      "Total basepairs processed:       515,038 bp\n",
      "  Read 1:       258,124 bp\n",
      "  Read 2:       256,914 bp\n",
      "Total written (filtered):        483,814 bp (93.9%)\n",
      "  Read 1:       248,237 bp\n",
      "  Read 2:       235,577 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 380 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t8\t0.0\t2\t0 7 1\n",
      "26\t357\t0.0\t2\t349 7 1\n",
      "27\t11\t0.0\t2\t0 4 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 821 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t14\t0.0\t2\t0 13 1\n",
      "26\t801\t0.0\t2\t712 86 3\n",
      "27\t3\t0.0\t2\t0 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S752_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S752_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S752_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S752_93_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S752_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S752_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S752_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S752_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,144 reads  @    108.4 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.69 s (109 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,144\n",
      "  Read 1 with adapter:                  42,927 (99.5%)\n",
      "  Read 2 with adapter:                  42,868 (99.4%)\n",
      "Pairs written (passing filters):        43,144 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,400,988 bp\n",
      "  Read 1:    12,677,782 bp\n",
      "  Read 2:    12,723,206 bp\n",
      "Total written (filtered):     23,170,853 bp (91.2%)\n",
      "  Read 1:    11,561,905 bp\n",
      "  Read 2:    11,608,948 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42927 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t501\t0.0\t2\t0 486 15\n",
      "26\t42121\t0.0\t2\t40792 1282 47\n",
      "27\t230\t0.0\t2\t0 227 3\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t469\t0.0\t2\t0 434 35\n",
      "26\t42178\t0.0\t2\t39102 3014 62\n",
      "27\t187\t0.0\t2\t0 177 10\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S753_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S753_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S753_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S753_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S753_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S753_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S753_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S753_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,560 reads  @     98.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.08 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,560\n",
      "  Read 1 with adapter:                  41,376 (99.6%)\n",
      "  Read 2 with adapter:                  41,471 (99.8%)\n",
      "Pairs written (passing filters):        41,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,474,020 bp\n",
      "  Read 1:    12,216,205 bp\n",
      "  Read 2:    12,257,815 bp\n",
      "Total written (filtered):     22,320,310 bp (91.2%)\n",
      "  Read 1:    11,140,611 bp\n",
      "  Read 2:    11,179,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41376 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t461\t0.0\t2\t0 449 12\n",
      "26\t40641\t0.0\t2\t39666 958 17\n",
      "27\t217\t0.0\t2\t0 210 7\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41471 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t309\t0.0\t2\t0 284 25\n",
      "26\t40943\t0.0\t2\t37748 3093 102\n",
      "27\t187\t0.0\t2\t0 167 20\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S754_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S754_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S754_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S754_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S754_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S754_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S754_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S754_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,227 reads  @     85.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.36 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,227\n",
      "  Read 1 with adapter:                  38,969 (99.3%)\n",
      "  Read 2 with adapter:                  39,167 (99.8%)\n",
      "Pairs written (passing filters):        39,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,079,214 bp\n",
      "  Read 1:    11,508,458 bp\n",
      "  Read 2:    11,570,756 bp\n",
      "Total written (filtered):     21,047,784 bp (91.2%)\n",
      "  Read 1:    10,495,317 bp\n",
      "  Read 2:    10,552,467 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38969 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t367\t0.0\t2\t0 355 12\n",
      "26\t38297\t0.0\t2\t37476 802 19\n",
      "27\t224\t0.0\t2\t0 220 4\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39167 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t234\t0.0\t2\t0 214 20\n",
      "26\t38725\t0.0\t2\t35769 2865 91\n",
      "27\t187\t0.0\t2\t0 171 16\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S755_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S755_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S755_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S755_99_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S755_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S755_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S755_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S755_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        30,010 reads  @     85.0 µs/read;   0.71 M reads/minute\n",
      "Finished in 2.56 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,010\n",
      "  Read 1 with adapter:                  29,956 (99.8%)\n",
      "  Read 2 with adapter:                  29,747 (99.1%)\n",
      "Pairs written (passing filters):        30,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,650,270 bp\n",
      "  Read 1:     8,842,186 bp\n",
      "  Read 2:     8,808,084 bp\n",
      "Total written (filtered):     16,098,089 bp (91.2%)\n",
      "  Read 1:     8,063,381 bp\n",
      "  Read 2:     8,034,708 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t306\t0.0\t2\t0 287 19\n",
      "26\t29427\t0.0\t2\t28793 620 14\n",
      "27\t139\t0.0\t2\t0 136 3\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t173\t0.0\t2\t0 156 17\n",
      "26\t29424\t0.0\t2\t26784 2496 144\n",
      "27\t137\t0.0\t2\t0 122 15\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S757_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S757_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S757_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S757_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S757_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S757_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S757_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S757_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,370 reads  @     83.1 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.36 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,370\n",
      "  Read 1 with adapter:                  52,192 (99.7%)\n",
      "  Read 2 with adapter:                  52,271 (99.8%)\n",
      "Pairs written (passing filters):        52,370 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,856,920 bp\n",
      "  Read 1:    15,411,377 bp\n",
      "  Read 2:    15,445,543 bp\n",
      "Total written (filtered):     28,141,158 bp (91.2%)\n",
      "  Read 1:    14,054,475 bp\n",
      "  Read 2:    14,086,683 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52192 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t548\t0.0\t2\t0 535 13\n",
      "26\t51232\t0.0\t2\t50181 1031 20\n",
      "27\t294\t0.0\t2\t0 290 4\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t443\t0.0\t2\t0 410 33\n",
      "26\t51561\t0.0\t2\t47608 3828 125\n",
      "27\t241\t0.0\t2\t0 227 14\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S758_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S758_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S758_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S758_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S758_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S758_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S758_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S758_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,711 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.44 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,711\n",
      "  Read 1 with adapter:                  38,619 (99.8%)\n",
      "  Read 2 with adapter:                  38,637 (99.8%)\n",
      "Pairs written (passing filters):        38,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,818,761 bp\n",
      "  Read 1:    11,402,673 bp\n",
      "  Read 2:    11,416,088 bp\n",
      "Total written (filtered):     20,810,323 bp (91.2%)\n",
      "  Read 1:    10,398,656 bp\n",
      "  Read 2:    10,411,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38619 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t410\t0.0\t2\t0 404 6\n",
      "26\t37899\t0.0\t2\t37073 807 19\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t309\t0.0\t2\t0 288 21\n",
      "26\t38140\t0.0\t2\t35317 2740 83\n",
      "27\t156\t0.0\t2\t0 145 11\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S759_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S759_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S759_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S759_105_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S759_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S759_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S759_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S759_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        44,101 reads  @     92.4 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.08 s (93 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,101\n",
      "  Read 1 with adapter:                  43,874 (99.5%)\n",
      "  Read 2 with adapter:                  44,050 (99.9%)\n",
      "Pairs written (passing filters):        44,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,971,138 bp\n",
      "  Read 1:    12,961,968 bp\n",
      "  Read 2:    13,009,170 bp\n",
      "Total written (filtered):     23,685,368 bp (91.2%)\n",
      "  Read 1:    11,821,395 bp\n",
      "  Read 2:    11,863,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t475\t0.0\t2\t0 458 17\n",
      "26\t43079\t0.0\t2\t42058 996 25\n",
      "27\t236\t0.0\t2\t0 226 10\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t343\t0.0\t2\t0 308 35\n",
      "26\t43467\t0.0\t2\t40118 3250 99\n",
      "27\t188\t0.0\t2\t0 183 5\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S760_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S760_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S760_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S760_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S760_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S760_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S760_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S760_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        36,849 reads  @    104.5 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.86 s (105 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,849\n",
      "  Read 1 with adapter:                  36,772 (99.8%)\n",
      "  Read 2 with adapter:                  36,556 (99.2%)\n",
      "Pairs written (passing filters):        36,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,727,071 bp\n",
      "  Read 1:    10,860,972 bp\n",
      "  Read 2:    10,866,099 bp\n",
      "Total written (filtered):     19,820,950 bp (91.2%)\n",
      "  Read 1:     9,905,082 bp\n",
      "  Read 2:     9,915,868 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t441\t0.0\t2\t0 431 10\n",
      "26\t36068\t0.0\t2\t35182 872 14\n",
      "27\t223\t0.0\t2\t0 219 4\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36556 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t407\t0.0\t2\t0 374 33\n",
      "26\t35922\t0.0\t2\t33213 2654 55\n",
      "27\t200\t0.0\t2\t0 184 16\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S761_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S761_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S761_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S761_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S761_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S761_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S761_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S761_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        35,414 reads  @    110.3 µs/read;   0.54 M reads/minute\n",
      "Finished in 3.92 s (111 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,414\n",
      "  Read 1 with adapter:                  35,315 (99.7%)\n",
      "  Read 2 with adapter:                  35,356 (99.8%)\n",
      "Pairs written (passing filters):        35,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,879,187 bp\n",
      "  Read 1:    10,433,887 bp\n",
      "  Read 2:    10,445,300 bp\n",
      "Total written (filtered):     19,042,021 bp (91.2%)\n",
      "  Read 1:     9,515,849 bp\n",
      "  Read 2:     9,526,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t409\t0.0\t2\t0 396 13\n",
      "26\t34625\t0.0\t2\t33739 863 23\n",
      "27\t225\t0.0\t2\t0 217 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35356 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t279\t0.0\t2\t0 263 16\n",
      "26\t34904\t0.0\t2\t32093 2707 104\n",
      "27\t147\t0.0\t2\t0 134 13\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S762_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S762_111_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S762_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S762_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S762_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S762_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,767 reads  @     96.5 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.75 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,767\n",
      "  Read 1 with adapter:                  38,715 (99.9%)\n",
      "  Read 2 with adapter:                  38,692 (99.8%)\n",
      "Pairs written (passing filters):        38,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,865,192 bp\n",
      "  Read 1:    11,432,045 bp\n",
      "  Read 2:    11,433,147 bp\n",
      "Total written (filtered):     20,852,769 bp (91.2%)\n",
      "  Read 1:    10,425,552 bp\n",
      "  Read 2:    10,427,217 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38715 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t423\t0.0\t2\t0 415 8\n",
      "26\t37962\t0.0\t2\t37002 943 17\n",
      "27\t250\t0.0\t2\t0 242 8\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38692 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t257\t0.0\t2\t0 237 20\n",
      "26\t38213\t0.0\t2\t35142 2971 100\n",
      "27\t205\t0.0\t2\t0 183 22\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S763_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S763_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S763_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S763_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S763_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S763_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S763_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S763_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,363 reads  @     86.5 µs/read;   0.69 M reads/minute\n",
      "Finished in 2.72 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,363\n",
      "  Read 1 with adapter:                  31,343 (99.9%)\n",
      "  Read 2 with adapter:                  30,609 (97.6%)\n",
      "Pairs written (passing filters):        31,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,334,127 bp\n",
      "  Read 1:     9,251,516 bp\n",
      "  Read 2:     9,082,611 bp\n",
      "Total written (filtered):     16,722,639 bp (91.2%)\n",
      "  Read 1:     8,435,818 bp\n",
      "  Read 2:     8,286,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31343 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t319\t0.0\t2\t0 312 7\n",
      "26\t30758\t0.0\t2\t30178 568 12\n",
      "27\t172\t0.0\t2\t0 169 3\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 30609 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t189\t0.0\t2\t0 177 12\n",
      "26\t30250\t0.0\t2\t27523 2544 183\n",
      "27\t155\t0.0\t2\t0 138 17\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S765_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S765_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S765_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S765_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S765_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S765_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S765_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S765_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,327 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.21 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,327\n",
      "  Read 1 with adapter:                  39,306 (99.9%)\n",
      "  Read 2 with adapter:                  39,139 (99.5%)\n",
      "Pairs written (passing filters):        39,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,193,528 bp\n",
      "  Read 1:    11,600,903 bp\n",
      "  Read 2:    11,592,625 bp\n",
      "Total written (filtered):     21,153,948 bp (91.2%)\n",
      "  Read 1:    10,578,770 bp\n",
      "  Read 2:    10,575,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t402\t0.0\t2\t0 397 5\n",
      "26\t38594\t0.0\t2\t37811 771 12\n",
      "27\t250\t0.0\t2\t0 244 6\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t323\t0.0\t2\t0 292 31\n",
      "26\t38629\t0.0\t2\t35536 2971 122\n",
      "27\t138\t0.0\t2\t0 129 9\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S766_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S766_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S766_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S766_117_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S766_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S766_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S766_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S766_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,203 reads  @     83.6 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.45 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,203\n",
      "  Read 1 with adapter:                  41,179 (99.9%)\n",
      "  Read 2 with adapter:                  41,079 (99.7%)\n",
      "Pairs written (passing filters):        41,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,305,063 bp\n",
      "  Read 1:    12,154,862 bp\n",
      "  Read 2:    12,150,201 bp\n",
      "Total written (filtered):     22,166,512 bp (91.2%)\n",
      "  Read 1:    11,084,244 bp\n",
      "  Read 2:    11,082,268 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t443\t0.0\t2\t0 431 12\n",
      "26\t40353\t0.0\t2\t39544 799 10\n",
      "27\t291\t0.0\t2\t0 287 4\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41079 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t302\t0.0\t2\t0 280 22\n",
      "26\t40572\t0.0\t2\t37389 3057 126\n",
      "27\t173\t0.0\t2\t0 154 19\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S767_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S767_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S767_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S767_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S767_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S767_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S767_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S767_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,391 reads  @     90.1 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.01 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,391\n",
      "  Read 1 with adapter:                  44,362 (99.9%)\n",
      "  Read 2 with adapter:                  44,339 (99.9%)\n",
      "Pairs written (passing filters):        44,391 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,188,393 bp\n",
      "  Read 1:    13,093,759 bp\n",
      "  Read 2:    13,094,634 bp\n",
      "Total written (filtered):     23,882,445 bp (91.2%)\n",
      "  Read 1:    11,940,492 bp\n",
      "  Read 2:    11,941,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t488\t0.0\t2\t0 475 13\n",
      "26\t43509\t0.0\t2\t42557 941 11\n",
      "27\t291\t0.0\t2\t0 288 3\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44339 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t337\t0.0\t2\t0 300 37\n",
      "26\t43780\t0.0\t2\t40456 3208 116\n",
      "27\t168\t0.0\t2\t0 160 8\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S768_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S768_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S768_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S768_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S768_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S768_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S768_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S768_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        35,297 reads  @     96.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.40 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,297\n",
      "  Read 1 with adapter:                  35,232 (99.8%)\n",
      "  Read 2 with adapter:                  35,044 (99.3%)\n",
      "Pairs written (passing filters):        35,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,812,249 bp\n",
      "  Read 1:    10,403,791 bp\n",
      "  Read 2:    10,408,458 bp\n",
      "Total written (filtered):     18,985,408 bp (91.2%)\n",
      "  Read 1:     9,487,890 bp\n",
      "  Read 2:     9,497,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t306\t0.0\t2\t0 298 8\n",
      "26\t34714\t0.0\t2\t33965 741 8\n",
      "27\t161\t0.0\t2\t0 158 3\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35044 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t345\t0.0\t2\t0 322 23\n",
      "26\t34523\t0.0\t2\t32056 2414 53\n",
      "27\t151\t0.0\t2\t0 147 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S769_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S769_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S769_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S769_123_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S769_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S769_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S769_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S769_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,490 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 3.73 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,490\n",
      "  Read 1 with adapter:                  40,430 (99.9%)\n",
      "  Read 2 with adapter:                  40,432 (99.9%)\n",
      "Pairs written (passing filters):        40,490 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,880,545 bp\n",
      "  Read 1:    11,937,519 bp\n",
      "  Read 2:    11,943,026 bp\n",
      "Total written (filtered):     21,778,295 bp (91.2%)\n",
      "  Read 1:    10,886,403 bp\n",
      "  Read 2:    10,891,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t327\t0.0\t2\t0 315 12\n",
      "26\t39829\t0.0\t2\t39014 805 10\n",
      "27\t217\t0.0\t2\t0 210 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t312\t0.0\t2\t0 285 27\n",
      "26\t39890\t0.0\t2\t36794 3002 94\n",
      "27\t214\t0.0\t2\t0 195 19\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S770_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S770_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S770_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S770_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S770_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S770_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S770_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S770_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,003 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.34 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,003\n",
      "  Read 1 with adapter:                  40,956 (99.9%)\n",
      "  Read 2 with adapter:                  40,942 (99.9%)\n",
      "Pairs written (passing filters):        41,003 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,186,857 bp\n",
      "  Read 1:    12,091,471 bp\n",
      "  Read 2:    12,095,386 bp\n",
      "Total written (filtered):     22,057,504 bp (91.2%)\n",
      "  Read 1:    11,026,522 bp\n",
      "  Read 2:    11,030,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40956 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t300\t0.0\t2\t0 291 9\n",
      "26\t40316\t0.0\t2\t39466 840 10\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t246\t0.0\t2\t0 225 21\n",
      "26\t40526\t0.0\t2\t37602 2820 104\n",
      "27\t158\t0.0\t2\t0 140 18\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S771_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S771_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S771_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S771_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S771_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S771_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S771_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S771_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        27,386 reads  @     98.2 µs/read;   0.61 M reads/minute\n",
      "Finished in 2.70 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,386\n",
      "  Read 1 with adapter:                  27,324 (99.8%)\n",
      "  Read 2 with adapter:                  27,165 (99.2%)\n",
      "Pairs written (passing filters):        27,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,132,991 bp\n",
      "  Read 1:     8,072,347 bp\n",
      "  Read 2:     8,060,644 bp\n",
      "Total written (filtered):     14,716,341 bp (91.2%)\n",
      "  Read 1:     7,361,935 bp\n",
      "  Read 2:     7,354,406 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t214\t0.0\t2\t0 206 8\n",
      "26\t26905\t0.0\t2\t26350 545 10\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27165 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t165\t0.0\t2\t0 151 14\n",
      "26\t26876\t0.0\t2\t24625 2126 125\n",
      "27\t111\t0.0\t2\t0 103 8\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S773_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S773_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S773_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S773_129_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S773_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S773_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S773_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S773_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,371 reads  @     85.9 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.39 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,371\n",
      "  Read 1 with adapter:                  39,123 (99.4%)\n",
      "  Read 2 with adapter:                  39,306 (99.8%)\n",
      "Pairs written (passing filters):        39,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,183,696 bp\n",
      "  Read 1:    11,570,824 bp\n",
      "  Read 2:    11,612,872 bp\n",
      "Total written (filtered):     21,144,844 bp (91.2%)\n",
      "  Read 1:    10,553,761 bp\n",
      "  Read 2:    10,591,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39123 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t381\t0.0\t2\t0 372 9\n",
      "26\t38472\t0.0\t2\t37621 832 19\n",
      "27\t218\t0.0\t2\t0 207 11\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t337\t0.0\t2\t0 307 30\n",
      "26\t38765\t0.0\t2\t35827 2850 88\n",
      "27\t178\t0.0\t2\t0 166 12\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S774_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S774_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S774_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S774_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S774_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S774_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S774_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S774_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,603 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 2.71 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,603\n",
      "  Read 1 with adapter:                  35,403 (99.4%)\n",
      "  Read 2 with adapter:                  35,535 (99.8%)\n",
      "Pairs written (passing filters):        35,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,968,741 bp\n",
      "  Read 1:    10,467,651 bp\n",
      "  Read 2:    10,501,090 bp\n",
      "Total written (filtered):     19,124,628 bp (91.2%)\n",
      "  Read 1:     9,547,384 bp\n",
      "  Read 2:     9,577,244 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35403 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t336\t0.0\t2\t0 326 10\n",
      "26\t34898\t0.0\t2\t33964 920 14\n",
      "27\t145\t0.0\t2\t0 142 3\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35535 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t267\t0.0\t2\t0 247 20\n",
      "26\t35065\t0.0\t2\t32324 2642 99\n",
      "27\t163\t0.0\t2\t0 152 11\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S775_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S775_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S775_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S775_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S775_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S775_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S775_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S775_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        59,879 reads  @     93.1 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.58 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,879\n",
      "  Read 1 with adapter:                  59,815 (99.9%)\n",
      "  Read 2 with adapter:                  59,811 (99.9%)\n",
      "Pairs written (passing filters):        59,879 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,319,793 bp\n",
      "  Read 1:    17,657,770 bp\n",
      "  Read 2:    17,662,023 bp\n",
      "Total written (filtered):     32,209,765 bp (91.2%)\n",
      "  Read 1:    16,102,589 bp\n",
      "  Read 2:    16,107,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t474\t0.0\t2\t0 463 11\n",
      "26\t58876\t0.0\t2\t57645 1203 28\n",
      "27\t357\t0.0\t2\t0 341 16\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t481\t0.0\t2\t0 442 39\n",
      "26\t59057\t0.0\t2\t54675 4290 92\n",
      "27\t224\t0.0\t2\t0 207 17\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S776_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S776_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S776_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S776_135_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S776_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S776_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S776_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S776_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        32,407 reads  @    100.5 µs/read;   0.60 M reads/minute\n",
      "Finished in 3.26 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,407\n",
      "  Read 1 with adapter:                  32,358 (99.8%)\n",
      "  Read 2 with adapter:                  32,179 (99.3%)\n",
      "Pairs written (passing filters):        32,407 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,114,269 bp\n",
      "  Read 1:     9,557,784 bp\n",
      "  Read 2:     9,556,485 bp\n",
      "Total written (filtered):     17,436,744 bp (91.2%)\n",
      "  Read 1:     8,716,680 bp\n",
      "  Read 2:     8,720,064 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32358 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t385\t0.0\t2\t0 372 13\n",
      "26\t31770\t0.0\t2\t30793 948 29\n",
      "27\t165\t0.0\t2\t0 154 11\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32179 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t391\t0.0\t2\t0 366 25\n",
      "26\t31575\t0.0\t2\t29154 2368 53\n",
      "27\t176\t0.0\t2\t0 160 16\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S777_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S777_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S777_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S777_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S777_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S777_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S777_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S777_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,683 reads  @     94.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.46 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,683\n",
      "  Read 1 with adapter:                  36,646 (99.9%)\n",
      "  Read 2 with adapter:                  36,527 (99.6%)\n",
      "Pairs written (passing filters):        36,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,630,872 bp\n",
      "  Read 1:    10,820,292 bp\n",
      "  Read 2:    10,810,580 bp\n",
      "Total written (filtered):     19,728,540 bp (91.2%)\n",
      "  Read 1:     9,867,515 bp\n",
      "  Read 2:     9,861,025 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t307\t0.0\t2\t0 297 10\n",
      "26\t36061\t0.0\t2\t35310 738 13\n",
      "27\t204\t0.0\t2\t0 197 7\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36527 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t299\t0.0\t2\t0 274 25\n",
      "26\t36057\t0.0\t2\t33214 2737 106\n",
      "27\t146\t0.0\t2\t0 136 10\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S778_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S778_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S778_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S778_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S778_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S778_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S778_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S778_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,833 reads  @     88.2 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.43 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,833\n",
      "  Read 1 with adapter:                  38,811 (99.9%)\n",
      "  Read 2 with adapter:                  38,777 (99.9%)\n",
      "Pairs written (passing filters):        38,833 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,909,112 bp\n",
      "  Read 1:    11,455,300 bp\n",
      "  Read 2:    11,453,812 bp\n",
      "Total written (filtered):     20,891,905 bp (91.2%)\n",
      "  Read 1:    10,446,190 bp\n",
      "  Read 2:    10,445,715 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t344\t0.0\t2\t0 335 9\n",
      "26\t38155\t0.0\t2\t37312 824 19\n",
      "27\t212\t0.0\t2\t0 201 11\n",
      "28\t89\t0.0\t2\t0 0 89\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t268\t0.0\t2\t0 255 13\n",
      "26\t38313\t0.0\t2\t35338 2882 93\n",
      "27\t181\t0.0\t2\t0 166 15\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S779_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S779_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S779_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S779_141_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S779_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S779_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S779_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S779_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,665 reads  @     97.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 2.89 s (98 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,665\n",
      "  Read 1 with adapter:                  29,639 (99.9%)\n",
      "  Read 2 with adapter:                  29,481 (99.4%)\n",
      "Pairs written (passing filters):        29,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,488,829 bp\n",
      "  Read 1:     8,750,342 bp\n",
      "  Read 2:     8,738,487 bp\n",
      "Total written (filtered):     15,951,801 bp (91.2%)\n",
      "  Read 1:     7,979,787 bp\n",
      "  Read 2:     7,972,014 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t251\t0.0\t2\t0 245 6\n",
      "26\t29198\t0.0\t2\t28631 551 16\n",
      "27\t140\t0.0\t2\t0 137 3\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 29481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t188\t0.0\t2\t0 167 21\n",
      "26\t29136\t0.0\t2\t26555 2415 166\n",
      "27\t139\t0.0\t2\t0 121 18\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S781_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S781_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S781_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S781_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S781_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S781_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S781_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S781_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        36,897 reads  @     81.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.02 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,897\n",
      "  Read 1 with adapter:                  36,868 (99.9%)\n",
      "  Read 2 with adapter:                  36,686 (99.4%)\n",
      "Pairs written (passing filters):        36,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,741,770 bp\n",
      "  Read 1:    10,883,819 bp\n",
      "  Read 2:    10,857,951 bp\n",
      "Total written (filtered):     19,829,696 bp (91.2%)\n",
      "  Read 1:     9,925,444 bp\n",
      "  Read 2:     9,904,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t369\t0.0\t2\t0 363 6\n",
      "26\t36323\t0.0\t2\t35553 759 11\n",
      "27\t160\t0.0\t2\t0 152 8\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36686 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t323\t0.0\t2\t0 303 20\n",
      "26\t36195\t0.0\t2\t33514 2590 91\n",
      "27\t130\t0.0\t2\t0 121 9\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S782_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S782_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S782_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S782_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S782_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S782_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S782_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S782_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        68,313 reads  @     94.0 µs/read;   0.64 M reads/minute\n",
      "Finished in 6.43 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,313\n",
      "  Read 1 with adapter:                  68,261 (99.9%)\n",
      "  Read 2 with adapter:                  68,152 (99.8%)\n",
      "Pairs written (passing filters):        68,313 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,295,829 bp\n",
      "  Read 1:    20,148,308 bp\n",
      "  Read 2:    20,147,521 bp\n",
      "Total written (filtered):     36,749,586 bp (91.2%)\n",
      "  Read 1:    18,373,830 bp\n",
      "  Read 2:    18,375,756 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t636\t0.0\t2\t0 619 17\n",
      "26\t67233\t0.0\t2\t65778 1429 26\n",
      "27\t324\t0.0\t2\t0 320 4\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68152 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t497\t0.0\t2\t0 455 42\n",
      "26\t67332\t0.0\t2\t62256 4908 168\n",
      "27\t284\t0.0\t2\t0 265 19\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S783_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S783_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S783_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S783_147_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S783_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S783_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S783_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S783_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        35,217 reads  @    106.3 µs/read;   0.56 M reads/minute\n",
      "Finished in 3.75 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,217\n",
      "  Read 1 with adapter:                  35,180 (99.9%)\n",
      "  Read 2 with adapter:                  35,161 (99.8%)\n",
      "Pairs written (passing filters):        35,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,774,708 bp\n",
      "  Read 1:    10,386,724 bp\n",
      "  Read 2:    10,387,984 bp\n",
      "Total written (filtered):     18,946,146 bp (91.2%)\n",
      "  Read 1:     9,472,211 bp\n",
      "  Read 2:     9,473,935 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t364\t0.0\t2\t0 353 11\n",
      "26\t34610\t0.0\t2\t33827 774 9\n",
      "27\t163\t0.0\t2\t0 156 7\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 35161 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t299\t0.0\t2\t0 278 21\n",
      "26\t34667\t0.0\t2\t31974 2598 95\n",
      "27\t168\t0.0\t2\t0 149 19\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S784_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S784_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S784_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S784_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S784_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S784_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S784_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S784_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,765 reads  @    109.4 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.47 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,765\n",
      "  Read 1 with adapter:                  40,743 (99.9%)\n",
      "  Read 2 with adapter:                  40,521 (99.4%)\n",
      "Pairs written (passing filters):        40,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,047,607 bp\n",
      "  Read 1:    12,025,655 bp\n",
      "  Read 2:    12,021,952 bp\n",
      "Total written (filtered):     21,934,972 bp (91.2%)\n",
      "  Read 1:    10,966,311 bp\n",
      "  Read 2:    10,968,661 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40743 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t338\t0.0\t2\t0 330 8\n",
      "26\t40094\t0.0\t2\t39037 1021 36\n",
      "27\t198\t0.0\t2\t0 194 4\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40521 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t431\t0.0\t2\t0 403 28\n",
      "26\t39846\t0.0\t2\t36874 2906 66\n",
      "27\t216\t0.0\t2\t0 198 18\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S785_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S785_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S785_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S785_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S785_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S785_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S785_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S785_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        41,441 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.12 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,441\n",
      "  Read 1 with adapter:                  41,421 (100.0%)\n",
      "  Read 2 with adapter:                  41,378 (99.8%)\n",
      "Pairs written (passing filters):        41,441 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,448,347 bp\n",
      "  Read 1:    12,224,999 bp\n",
      "  Read 2:    12,223,348 bp\n",
      "Total written (filtered):     22,295,797 bp (91.2%)\n",
      "  Read 1:    11,148,120 bp\n",
      "  Read 2:    11,147,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41421 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t341\t0.0\t2\t0 331 10\n",
      "26\t40779\t0.0\t2\t39962 800 17\n",
      "27\t244\t0.0\t2\t0 239 5\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41378 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t321\t0.0\t2\t0 299 22\n",
      "26\t40855\t0.0\t2\t37784 2994 77\n",
      "27\t184\t0.0\t2\t0 171 13\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S786_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S786_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S786_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S786_153_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S786_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S786_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S786_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S786_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,931 reads  @    100.0 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.00 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,931\n",
      "  Read 1 with adapter:                  39,912 (100.0%)\n",
      "  Read 2 with adapter:                  39,807 (99.7%)\n",
      "Pairs written (passing filters):        39,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,557,533 bp\n",
      "  Read 1:    11,779,627 bp\n",
      "  Read 2:    11,777,906 bp\n",
      "Total written (filtered):     21,484,866 bp (91.2%)\n",
      "  Read 1:    10,741,907 bp\n",
      "  Read 2:    10,742,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39912 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t305\t0.0\t2\t0 293 12\n",
      "26\t39292\t0.0\t2\t38509 771 12\n",
      "27\t249\t0.0\t2\t0 241 8\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t279\t0.0\t2\t0 253 26\n",
      "26\t39329\t0.0\t2\t35911 3256 162\n",
      "27\t126\t0.0\t2\t0 114 12\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S787_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S787_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S787_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S787_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S787_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S787_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S787_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S787_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        26,527 reads  @    123.2 µs/read;   0.49 M reads/minute\n",
      "Finished in 3.28 s (124 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,527\n",
      "  Read 1 with adapter:                  26,499 (99.9%)\n",
      "  Read 2 with adapter:                  26,133 (98.5%)\n",
      "Pairs written (passing filters):        26,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,595,566 bp\n",
      "  Read 1:     7,825,250 bp\n",
      "  Read 2:     7,770,316 bp\n",
      "Total written (filtered):     14,226,132 bp (91.2%)\n",
      "  Read 1:     7,135,238 bp\n",
      "  Read 2:     7,090,894 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26499 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t211\t0.0\t2\t0 206 5\n",
      "26\t26110\t0.0\t2\t25549 554 7\n",
      "27\t143\t0.0\t2\t0 140 3\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26133 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t167\t0.0\t2\t0 149 18\n",
      "26\t25818\t0.0\t2\t23472 2190 156\n",
      "27\t121\t0.0\t2\t0 107 14\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S789_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S789_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S789_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S789_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S789_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S789_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S789_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S789_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,472 reads  @    126.1 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.50 s (126 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,472\n",
      "  Read 1 with adapter:                  43,435 (99.9%)\n",
      "  Read 2 with adapter:                  43,337 (99.7%)\n",
      "Pairs written (passing filters):        43,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,638,802 bp\n",
      "  Read 1:    12,823,331 bp\n",
      "  Read 2:    12,815,471 bp\n",
      "Total written (filtered):     23,383,003 bp (91.2%)\n",
      "  Read 1:    11,694,151 bp\n",
      "  Read 2:    11,688,852 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43435 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t379\t0.0\t2\t0 368 11\n",
      "26\t42803\t0.0\t2\t41866 924 13\n",
      "27\t189\t0.0\t2\t0 184 5\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t359\t0.0\t2\t0 321 38\n",
      "26\t42745\t0.0\t2\t39395 3242 108\n",
      "27\t186\t0.0\t2\t0 173 13\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S790_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S790_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S790_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S790_159_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S790_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S790_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S790_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S790_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,047 reads  @    119.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 5.17 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,047\n",
      "  Read 1 with adapter:                  43,022 (99.9%)\n",
      "  Read 2 with adapter:                  42,948 (99.8%)\n",
      "Pairs written (passing filters):        43,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,393,862 bp\n",
      "  Read 1:    12,698,849 bp\n",
      "  Read 2:    12,695,013 bp\n",
      "Total written (filtered):     23,158,808 bp (91.2%)\n",
      "  Read 1:    11,580,317 bp\n",
      "  Read 2:    11,578,491 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43022 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t365\t0.0\t2\t0 358 7\n",
      "26\t42348\t0.0\t2\t41438 891 19\n",
      "27\t225\t0.0\t2\t0 218 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42948 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t318\t0.0\t2\t0 289 29\n",
      "26\t42427\t0.0\t2\t39042 3254 131\n",
      "27\t170\t0.0\t2\t0 155 15\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S791_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S791_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S791_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S791_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S791_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S791_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S791_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S791_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        40,836 reads  @    115.5 µs/read;   0.52 M reads/minute\n",
      "Finished in 4.72 s (116 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,836\n",
      "  Read 1 with adapter:                  40,815 (99.9%)\n",
      "  Read 2 with adapter:                  40,771 (99.8%)\n",
      "Pairs written (passing filters):        40,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,092,358 bp\n",
      "  Read 1:    12,046,579 bp\n",
      "  Read 2:    12,045,779 bp\n",
      "Total written (filtered):     21,971,234 bp (91.2%)\n",
      "  Read 1:    10,985,373 bp\n",
      "  Read 2:    10,985,861 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t327\t0.0\t2\t0 322 5\n",
      "26\t40170\t0.0\t2\t39277 883 10\n",
      "27\t205\t0.0\t2\t0 196 9\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40771 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t334\t0.0\t2\t0 298 36\n",
      "26\t40241\t0.0\t2\t36961 3142 138\n",
      "27\t126\t0.0\t2\t0 113 13\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S792_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S792_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S792_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S792_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S792_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S792_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S792_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S792_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:02        24,818 reads  @    116.9 µs/read;   0.51 M reads/minute\n",
      "Finished in 2.91 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,818\n",
      "  Read 1 with adapter:                  24,793 (99.9%)\n",
      "  Read 2 with adapter:                  24,663 (99.4%)\n",
      "Pairs written (passing filters):        24,818 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,638,173 bp\n",
      "  Read 1:     7,319,503 bp\n",
      "  Read 2:     7,318,670 bp\n",
      "Total written (filtered):     13,352,575 bp (91.2%)\n",
      "  Read 1:     6,674,976 bp\n",
      "  Read 2:     6,677,599 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 24793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t259\t0.0\t2\t0 251 8\n",
      "26\t24377\t0.0\t2\t23610 743 24\n",
      "27\t102\t0.0\t2\t0 98 4\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 24663 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t273\t0.0\t2\t0 261 12\n",
      "26\t24261\t0.0\t2\t22549 1672 40\n",
      "27\t120\t0.0\t2\t0 113 7\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S793_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S793_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S793_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S793_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S793_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S793_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S793_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S793_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        41,494 reads  @    107.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.48 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,494\n",
      "  Read 1 with adapter:                  41,459 (99.9%)\n",
      "  Read 2 with adapter:                  41,385 (99.7%)\n",
      "Pairs written (passing filters):        41,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,473,065 bp\n",
      "  Read 1:    12,238,389 bp\n",
      "  Read 2:    12,234,676 bp\n",
      "Total written (filtered):     22,319,333 bp (91.2%)\n",
      "  Read 1:    11,160,594 bp\n",
      "  Read 2:    11,158,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t388\t0.0\t2\t0 377 11\n",
      "26\t40808\t0.0\t2\t39818 974 16\n",
      "27\t185\t0.0\t2\t0 179 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t278\t0.0\t2\t0 258 20\n",
      "26\t40875\t0.0\t2\t37551 3204 120\n",
      "27\t211\t0.0\t2\t0 182 29\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S794_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S794_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S794_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S794_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S794_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S794_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S794_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S794_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,159 reads  @     97.8 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.03 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,159\n",
      "  Read 1 with adapter:                  41,136 (99.9%)\n",
      "  Read 2 with adapter:                  41,023 (99.7%)\n",
      "Pairs written (passing filters):        41,159 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,275,640 bp\n",
      "  Read 1:    12,141,027 bp\n",
      "  Read 2:    12,134,613 bp\n",
      "Total written (filtered):     22,139,673 bp (91.2%)\n",
      "  Read 1:    11,071,575 bp\n",
      "  Read 2:    11,068,098 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41136 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t367\t0.0\t2\t0 356 11\n",
      "26\t40488\t0.0\t2\t39545 921 22\n",
      "27\t203\t0.0\t2\t0 196 7\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41023 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t271\t0.0\t2\t0 244 27\n",
      "26\t40552\t0.0\t2\t37008 3404 140\n",
      "27\t144\t0.0\t2\t0 127 17\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S795_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S795_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S795_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S795_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S795_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S795_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S795_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S795_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:03        32,684 reads  @    105.8 µs/read;   0.57 M reads/minute\n",
      "Finished in 3.46 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,684\n",
      "  Read 1 with adapter:                  32,655 (99.9%)\n",
      "  Read 2 with adapter:                  32,337 (98.9%)\n",
      "Pairs written (passing filters):        32,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,246,274 bp\n",
      "  Read 1:     9,639,352 bp\n",
      "  Read 2:     9,606,922 bp\n",
      "Total written (filtered):     17,555,790 bp (91.2%)\n",
      "  Read 1:     8,789,603 bp\n",
      "  Read 2:     8,766,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t255\t0.0\t2\t0 252 3\n",
      "26\t32207\t0.0\t2\t31535 658 14\n",
      "27\t163\t0.0\t2\t0 157 6\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "25\t203\t0.0\t2\t0 180 23\n",
      "26\t31953\t0.0\t2\t28974 2778 201\n",
      "27\t166\t0.0\t2\t0 144 22\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S797_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S797_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S797_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S797_171_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S797_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S797_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S797_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S797_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        46,831 reads  @    109.7 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.14 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,831\n",
      "  Read 1 with adapter:                  46,790 (99.9%)\n",
      "  Read 2 with adapter:                  46,655 (99.6%)\n",
      "Pairs written (passing filters):        46,831 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,615,891 bp\n",
      "  Read 1:    13,812,117 bp\n",
      "  Read 2:    13,803,774 bp\n",
      "Total written (filtered):     25,186,585 bp (91.2%)\n",
      "  Read 1:    12,595,647 bp\n",
      "  Read 2:    12,590,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46790 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t399\t0.0\t2\t0 393 6\n",
      "26\t46085\t0.0\t2\t45202 858 25\n",
      "27\t227\t0.0\t2\t0 223 4\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46655 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t407\t0.0\t2\t0 373 34\n",
      "26\t46008\t0.0\t2\t42260 3603 145\n",
      "27\t187\t0.0\t2\t0 166 21\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S798_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S798_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S798_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S798_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S798_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S798_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S798_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S798_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        46,205 reads  @    104.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.82 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,205\n",
      "  Read 1 with adapter:                  46,163 (99.9%)\n",
      "  Read 2 with adapter:                  46,071 (99.7%)\n",
      "Pairs written (passing filters):        46,205 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,254,308 bp\n",
      "  Read 1:    13,627,464 bp\n",
      "  Read 2:    13,626,844 bp\n",
      "Total written (filtered):     24,856,417 bp (91.2%)\n",
      "  Read 1:    12,427,281 bp\n",
      "  Read 2:    12,429,136 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46163 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t387\t0.0\t2\t0 381 6\n",
      "26\t45431\t0.0\t2\t44493 925 13\n",
      "27\t254\t0.0\t2\t0 247 7\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46071 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t358\t0.0\t2\t0 323 35\n",
      "26\t45474\t0.0\t2\t41841 3473 160\n",
      "27\t186\t0.0\t2\t0 172 14\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S799_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S799_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S799_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S799_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S799_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S799_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S799_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S799_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,795 reads  @    103.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 5.05 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,795\n",
      "  Read 1 with adapter:                  48,760 (99.9%)\n",
      "  Read 2 with adapter:                  48,747 (99.9%)\n",
      "Pairs written (passing filters):        48,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,786,482 bp\n",
      "  Read 1:    14,392,684 bp\n",
      "  Read 2:    14,393,798 bp\n",
      "Total written (filtered):     26,251,628 bp (91.2%)\n",
      "  Read 1:    13,125,104 bp\n",
      "  Read 2:    13,126,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48760 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t456\t0.0\t2\t0 444 12\n",
      "26\t48035\t0.0\t2\t46967 1041 27\n",
      "27\t202\t0.0\t2\t0 197 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48747 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t344\t0.0\t2\t0 311 33\n",
      "26\t48173\t0.0\t2\t44395 3643 135\n",
      "27\t192\t0.0\t2\t0 180 12\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S800_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S800_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S800_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S800_177_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S800_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S800_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S800_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S800_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        42,511 reads  @     99.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.21 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,511\n",
      "  Read 1 with adapter:                  42,484 (99.9%)\n",
      "  Read 2 with adapter:                  42,409 (99.8%)\n",
      "Pairs written (passing filters):        42,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,078,240 bp\n",
      "  Read 1:    12,538,954 bp\n",
      "  Read 2:    12,539,286 bp\n",
      "Total written (filtered):     22,871,232 bp (91.2%)\n",
      "  Read 1:    11,434,555 bp\n",
      "  Read 2:    11,436,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42484 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t471\t0.0\t2\t0 462 9\n",
      "26\t41711\t0.0\t2\t40633 1052 26\n",
      "27\t222\t0.0\t2\t0 214 8\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t305\t0.0\t2\t0 274 31\n",
      "26\t41818\t0.0\t2\t38427 3249 142\n",
      "27\t228\t0.0\t2\t0 209 19\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S801_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S801_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S801_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S801_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S801_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S801_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S801_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S801_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,092 reads  @     87.3 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.21 s (88 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,092\n",
      "  Read 1 with adapter:                  48,061 (99.9%)\n",
      "  Read 2 with adapter:                  47,996 (99.8%)\n",
      "Pairs written (passing filters):        48,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,372,926 bp\n",
      "  Read 1:    14,186,981 bp\n",
      "  Read 2:    14,185,945 bp\n",
      "Total written (filtered):     25,875,918 bp (91.2%)\n",
      "  Read 1:    12,937,613 bp\n",
      "  Read 2:    12,938,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t603\t0.0\t2\t0 584 19\n",
      "26\t47028\t0.0\t2\t45741 1252 35\n",
      "27\t339\t0.0\t2\t0 325 14\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t505\t0.0\t2\t0 461 44\n",
      "26\t47187\t0.0\t2\t43413 3648 126\n",
      "27\t239\t0.0\t2\t0 219 20\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S802_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S802_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S802_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S802_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S802_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S802_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S802_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S802_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,682 reads  @    102.7 µs/read;   0.58 M reads/minute\n",
      "Finished in 0.18 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,682\n",
      "  Read 1 with adapter:                   1,681 (99.9%)\n",
      "  Read 2 with adapter:                   1,675 (99.6%)\n",
      "Pairs written (passing filters):         1,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:       991,914 bp\n",
      "  Read 1:       496,188 bp\n",
      "  Read 2:       495,726 bp\n",
      "Total written (filtered):        904,669 bp (91.2%)\n",
      "  Read 1:       452,492 bp\n",
      "  Read 2:       452,177 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t23\t0.0\t2\t0 23\n",
      "26\t1643\t0.0\t2\t1588 55\n",
      "27\t9\t0.0\t2\t0 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1675 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t14\t0.0\t2\t0 10 4\n",
      "26\t1646\t0.0\t2\t1485 149 12\n",
      "27\t13\t0.0\t2\t0 13\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S803_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S803_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S803_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S803_183_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S803_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S803_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S803_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S803_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        43,787 reads  @    107.8 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.73 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,787\n",
      "  Read 1 with adapter:                  43,700 (99.8%)\n",
      "  Read 2 with adapter:                  43,646 (99.7%)\n",
      "Pairs written (passing filters):        43,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,808,908 bp\n",
      "  Read 1:    12,905,166 bp\n",
      "  Read 2:    12,903,742 bp\n",
      "Total written (filtered):     23,538,387 bp (91.2%)\n",
      "  Read 1:    11,769,200 bp\n",
      "  Read 2:    11,769,187 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43700 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t534\t0.0\t2\t0 521 13\n",
      "26\t42880\t0.0\t2\t42000 861 19\n",
      "27\t208\t0.0\t2\t0 201 7\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43646 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t436\t0.0\t2\t0 409 27\n",
      "26\t42962\t0.0\t2\t39690 3179 93\n",
      "27\t217\t0.0\t2\t0 199 18\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S804_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S804_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S804_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S804_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S804_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S804_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S804_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S804_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:08        77,761 reads  @    104.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 8.11 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,761\n",
      "  Read 1 with adapter:                  77,676 (99.9%)\n",
      "  Read 2 with adapter:                  77,598 (99.8%)\n",
      "Pairs written (passing filters):        77,761 (100.0%)\n",
      "\n",
      "Total basepairs processed:    45,861,569 bp\n",
      "  Read 1:    22,927,215 bp\n",
      "  Read 2:    22,934,354 bp\n",
      "Total written (filtered):     41,825,093 bp (91.2%)\n",
      "  Read 1:    20,908,064 bp\n",
      "  Read 2:    20,917,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 77676 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t882\t0.0\t2\t0 855 27\n",
      "26\t76311\t0.0\t2\t74677 1610 24\n",
      "27\t381\t0.0\t2\t0 374 7\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 77598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t584\t0.0\t2\t0 536 48\n",
      "26\t76630\t0.0\t2\t70454 5924 252\n",
      "27\t335\t0.0\t2\t0 300 35\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S805_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S805_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S805_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S805_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S805_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S805_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S805_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S805_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,840 reads  @     95.3 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.19 s (96 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,840\n",
      "  Read 1 with adapter:                  43,808 (99.9%)\n",
      "  Read 2 with adapter:                  43,667 (99.6%)\n",
      "Pairs written (passing filters):        43,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,853,428 bp\n",
      "  Read 1:    12,929,609 bp\n",
      "  Read 2:    12,923,819 bp\n",
      "Total written (filtered):     23,579,207 bp (91.2%)\n",
      "  Read 1:    11,790,697 bp\n",
      "  Read 2:    11,788,510 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43808 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t459\t0.0\t2\t0 450 9\n",
      "26\t43024\t0.0\t2\t42084 925 15\n",
      "27\t227\t0.0\t2\t0 222 5\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43667 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t266\t0.0\t2\t0 240 26\n",
      "26\t43145\t0.0\t2\t39269 3630 246\n",
      "27\t235\t0.0\t2\t0 208 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S806_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S806_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S806_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S806_189_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S806_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S806_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S806_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S806_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,235 reads  @     86.9 µs/read;   0.69 M reads/minute\n",
      "Finished in 5.85 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,235\n",
      "  Read 1 with adapter:                  67,205 (100.0%)\n",
      "  Read 2 with adapter:                  66,696 (99.2%)\n",
      "Pairs written (passing filters):        67,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,569,293 bp\n",
      "  Read 1:    19,833,889 bp\n",
      "  Read 2:    19,735,404 bp\n",
      "Total written (filtered):     36,088,157 bp (91.2%)\n",
      "  Read 1:    18,086,656 bp\n",
      "  Read 2:    18,001,501 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67205 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t669\t0.0\t2\t0 647 22\n",
      "26\t66016\t0.0\t2\t64659 1335 22\n",
      "27\t356\t0.0\t2\t0 348 8\n",
      "28\t136\t0.0\t2\t0 0 136\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66696 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t486\t0.0\t2\t0 452 34\n",
      "26\t65857\t0.0\t2\t60829 4830 198\n",
      "27\t313\t0.0\t2\t0 285 28\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S807_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S807_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S807_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S807_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S807_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S807_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S807_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S807_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,146 reads  @     89.8 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.79 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,146\n",
      "  Read 1 with adapter:                  42,122 (99.9%)\n",
      "  Read 2 with adapter:                  42,064 (99.8%)\n",
      "Pairs written (passing filters):        42,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,863,616 bp\n",
      "  Read 1:    12,432,652 bp\n",
      "  Read 2:    12,430,964 bp\n",
      "Total written (filtered):     22,675,170 bp (91.2%)\n",
      "  Read 1:    11,337,695 bp\n",
      "  Read 2:    11,337,475 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t496\t0.0\t2\t0 486 10\n",
      "26\t41292\t0.0\t2\t40463 809 20\n",
      "27\t279\t0.0\t2\t0 271 8\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42064 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t370\t0.0\t2\t0 345 25\n",
      "26\t41468\t0.0\t2\t38311 3059 98\n",
      "27\t201\t0.0\t2\t0 189 12\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S808_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S808_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S808_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S808_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S808_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S808_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S808_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S808_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,364 reads  @     85.5 µs/read;   0.70 M reads/minute\n",
      "Finished in 5.42 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,364\n",
      "  Read 1 with adapter:                  63,324 (99.9%)\n",
      "  Read 2 with adapter:                  63,248 (99.8%)\n",
      "Pairs written (passing filters):        63,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,379,841 bp\n",
      "  Read 1:    18,691,495 bp\n",
      "  Read 2:    18,688,346 bp\n",
      "Total written (filtered):     34,089,378 bp (91.2%)\n",
      "  Read 1:    17,045,321 bp\n",
      "  Read 2:    17,044,057 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t798\t0.0\t2\t0 783 15\n",
      "26\t61995\t0.0\t2\t60753 1214 28\n",
      "27\t366\t0.0\t2\t0 354 12\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63248 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t450\t0.0\t2\t0 418 32\n",
      "26\t62484\t0.0\t2\t57576 4727 181\n",
      "27\t273\t0.0\t2\t0 247 26\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S809_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S809_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S809_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S809_195_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S809_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S809_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S809_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S809_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,854 reads  @     91.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.56 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,854\n",
      "  Read 1 with adapter:                  49,807 (99.9%)\n",
      "  Read 2 with adapter:                  49,723 (99.7%)\n",
      "Pairs written (passing filters):        49,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,405,014 bp\n",
      "  Read 1:    14,706,285 bp\n",
      "  Read 2:    14,698,729 bp\n",
      "Total written (filtered):     26,817,573 bp (91.2%)\n",
      "  Read 1:    13,411,653 bp\n",
      "  Read 2:    13,405,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49807 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t697\t0.0\t2\t0 669 28\n",
      "26\t48752\t0.0\t2\t47603 1121 28\n",
      "27\t237\t0.0\t2\t0 229 8\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49723 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t331\t0.0\t2\t0 296 35\n",
      "26\t49037\t0.0\t2\t45065 3816 156\n",
      "27\t280\t0.0\t2\t0 260 20\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S810_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S810_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S810_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S810_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S810_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S810_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S810_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S810_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,177 reads  @     97.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.63 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,177\n",
      "  Read 1 with adapter:                  47,142 (99.9%)\n",
      "  Read 2 with adapter:                  47,017 (99.7%)\n",
      "Pairs written (passing filters):        47,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,823,873 bp\n",
      "  Read 1:    13,916,404 bp\n",
      "  Read 2:    13,907,469 bp\n",
      "Total written (filtered):     25,376,117 bp (91.2%)\n",
      "  Read 1:    12,690,975 bp\n",
      "  Read 2:    12,685,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47142 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t652\t0.0\t2\t0 625 27\n",
      "26\t46093\t0.0\t2\t44967 1105 21\n",
      "27\t257\t0.0\t2\t0 249 8\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t380\t0.0\t2\t0 340 40\n",
      "26\t46353\t0.0\t2\t42295 3879 179\n",
      "27\t215\t0.0\t2\t0 196 19\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S811_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S811_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S811_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S811_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S811_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S811_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S811_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S811_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        38,536 reads  @    107.2 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.14 s (107 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,536\n",
      "  Read 1 with adapter:                  38,503 (99.9%)\n",
      "  Read 2 with adapter:                  38,430 (99.7%)\n",
      "Pairs written (passing filters):        38,536 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,725,217 bp\n",
      "  Read 1:    11,366,940 bp\n",
      "  Read 2:    11,358,277 bp\n",
      "Total written (filtered):     20,725,355 bp (91.2%)\n",
      "  Read 1:    10,366,061 bp\n",
      "  Read 2:    10,359,294 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38503 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t388\t0.0\t2\t0 382 6\n",
      "26\t37876\t0.0\t2\t37076 788 12\n",
      "27\t177\t0.0\t2\t0 171 6\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t393\t0.0\t2\t0 353 40\n",
      "26\t37828\t0.0\t2\t34925 2798 105\n",
      "27\t186\t0.0\t2\t0 171 15\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S812_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S812_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S812_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S812_201_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S812_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S812_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S812_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S812_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,535 reads  @    109.4 µs/read;   0.55 M reads/minute\n",
      "Finished in 4.34 s (110 us/read; 0.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,535\n",
      "  Read 1 with adapter:                  39,506 (99.9%)\n",
      "  Read 2 with adapter:                  39,466 (99.8%)\n",
      "Pairs written (passing filters):        39,535 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,323,548 bp\n",
      "  Read 1:    11,662,081 bp\n",
      "  Read 2:    11,661,467 bp\n",
      "Total written (filtered):     21,270,558 bp (91.2%)\n",
      "  Read 1:    10,635,097 bp\n",
      "  Read 2:    10,635,461 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t424\t0.0\t2\t0 417 7\n",
      "26\t38836\t0.0\t2\t37985 831 20\n",
      "27\t204\t0.0\t2\t0 198 6\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39466 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t318\t0.0\t2\t0 299 19\n",
      "26\t38928\t0.0\t2\t35920 2911 97\n",
      "27\t192\t0.0\t2\t0 176 16\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S813_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S813_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S813_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S813_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S813_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S813_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S813_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S813_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        39,785 reads  @    103.9 µs/read;   0.58 M reads/minute\n",
      "Finished in 4.14 s (104 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,785\n",
      "  Read 1 with adapter:                  39,654 (99.7%)\n",
      "  Read 2 with adapter:                  39,707 (99.8%)\n",
      "Pairs written (passing filters):        39,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,454,133 bp\n",
      "  Read 1:    11,721,554 bp\n",
      "  Read 2:    11,732,579 bp\n",
      "Total written (filtered):     21,390,913 bp (91.2%)\n",
      "  Read 1:    10,690,698 bp\n",
      "  Read 2:    10,700,215 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39654 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t411\t0.0\t2\t0 401 10\n",
      "26\t38963\t0.0\t2\t38048 897 18\n",
      "27\t201\t0.0\t2\t0 195 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t8\t0.0\t2\t0 0 8\n",
      "25\t270\t0.0\t2\t0 248 22\n",
      "26\t39171\t0.0\t2\t36155 2905 111\n",
      "27\t248\t0.0\t2\t0 221 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S814_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S814_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S814_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S814_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S814_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S814_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S814_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S814_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,411 reads  @     88.2 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.75 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,411\n",
      "  Read 1 with adapter:                  42,055 (99.2%)\n",
      "  Read 2 with adapter:                  42,137 (99.4%)\n",
      "Pairs written (passing filters):        42,411 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,933,462 bp\n",
      "  Read 1:    12,470,235 bp\n",
      "  Read 2:    12,463,227 bp\n",
      "Total written (filtered):     22,744,735 bp (91.2%)\n",
      "  Read 1:    11,376,923 bp\n",
      "  Read 2:    11,367,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42055 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t430\t0.0\t2\t0 420 10\n",
      "26\t41339\t0.0\t2\t40457 862 20\n",
      "27\t188\t0.0\t2\t0 178 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t352\t0.0\t2\t0 319 33\n",
      "26\t41535\t0.0\t2\t38376 3064 95\n",
      "27\t227\t0.0\t2\t0 212 15\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S815_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S815_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S815_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S815_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S815_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S815_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S815_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S815_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,009 reads  @     88.4 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.72 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,009\n",
      "  Read 1 with adapter:                  41,991 (100.0%)\n",
      "  Read 2 with adapter:                  41,910 (99.8%)\n",
      "Pairs written (passing filters):        42,008 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,782,234 bp\n",
      "  Read 1:    12,392,633 bp\n",
      "  Read 2:    12,389,601 bp\n",
      "Total written (filtered):     22,600,928 bp (91.2%)\n",
      "  Read 1:    11,300,752 bp\n",
      "  Read 2:    11,300,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t483\t0.0\t2\t0 475 8\n",
      "26\t41162\t0.0\t2\t40370 779 13\n",
      "27\t251\t0.0\t2\t0 245 6\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41910 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t446\t0.0\t2\t0 409 37\n",
      "26\t41213\t0.0\t2\t37935 3147 131\n",
      "27\t223\t0.0\t2\t0 208 15\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S816_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S816_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S816_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S816_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S816_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S816_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S816_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S816_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,751 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.96 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,751\n",
      "  Read 1 with adapter:                  48,716 (99.9%)\n",
      "  Read 2 with adapter:                  48,660 (99.8%)\n",
      "Pairs written (passing filters):        48,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,760,809 bp\n",
      "  Read 1:    14,381,331 bp\n",
      "  Read 2:    14,379,478 bp\n",
      "Total written (filtered):     26,229,030 bp (91.2%)\n",
      "  Read 1:    13,114,674 bp\n",
      "  Read 2:    13,114,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t549\t0.0\t2\t0 531 18\n",
      "26\t47825\t0.0\t2\t46870 926 29\n",
      "27\t267\t0.0\t2\t0 262 5\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48660 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t321\t0.0\t2\t0 300 21\n",
      "26\t48067\t0.0\t2\t44305 3614 148\n",
      "27\t205\t0.0\t2\t0 183 22\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S817_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S817_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S817_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S817_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S817_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S817_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S817_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S817_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,849 reads  @     88.2 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.43 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,849\n",
      "  Read 1 with adapter:                  38,834 (100.0%)\n",
      "  Read 2 with adapter:                  38,753 (99.8%)\n",
      "Pairs written (passing filters):        38,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,917,596 bp\n",
      "  Read 1:    11,460,439 bp\n",
      "  Read 2:    11,457,157 bp\n",
      "Total written (filtered):     20,900,513 bp (91.2%)\n",
      "  Read 1:    10,450,922 bp\n",
      "  Read 2:    10,449,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38834 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t445\t0.0\t2\t0 439 6\n",
      "26\t38098\t0.0\t2\t37383 705 10\n",
      "27\t212\t0.0\t2\t0 207 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t250\t0.0\t2\t0 216 34\n",
      "26\t38269\t0.0\t2\t35114 3025 130\n",
      "27\t202\t0.0\t2\t0 186 16\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S818_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S818_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S818_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S818_213_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S818_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S818_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S818_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S818_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,612 reads  @     92.2 µs/read;   0.65 M reads/minute\n",
      "Finished in 4.03 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,612\n",
      "  Read 1 with adapter:                  43,588 (99.9%)\n",
      "  Read 2 with adapter:                  43,513 (99.8%)\n",
      "Pairs written (passing filters):        43,612 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,728,438 bp\n",
      "  Read 1:    12,865,120 bp\n",
      "  Read 2:    12,863,318 bp\n",
      "Total written (filtered):     23,464,045 bp (91.2%)\n",
      "  Read 1:    11,731,986 bp\n",
      "  Read 2:    11,732,059 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43588 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t446\t0.0\t2\t0 441 5\n",
      "26\t42835\t0.0\t2\t41968 855 12\n",
      "27\t250\t0.0\t2\t0 246 4\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43513 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t326\t0.0\t2\t0 296 30\n",
      "26\t42954\t0.0\t2\t39475 3325 154\n",
      "27\t191\t0.0\t2\t0 175 16\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S819_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S819_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S819_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S819_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S819_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S819_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S819_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S819_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        43,668 reads  @     98.4 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.31 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,668\n",
      "  Read 1 with adapter:                  43,639 (99.9%)\n",
      "  Read 2 with adapter:                  43,481 (99.6%)\n",
      "Pairs written (passing filters):        43,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,753,657 bp\n",
      "  Read 1:    12,881,651 bp\n",
      "  Read 2:    12,872,006 bp\n",
      "Total written (filtered):     23,489,150 bp (91.2%)\n",
      "  Read 1:    11,747,398 bp\n",
      "  Read 2:    11,741,752 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t606\t0.0\t2\t0 590 16\n",
      "26\t42739\t0.0\t2\t41744 983 12\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t474\t0.0\t2\t0 440 34\n",
      "26\t42758\t0.0\t2\t39076 3474 208\n",
      "27\t216\t0.0\t2\t0 194 22\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S820_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S820_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S820_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S820_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S820_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S820_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S820_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S820_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,042 reads  @     98.6 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.45 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,042\n",
      "  Read 1 with adapter:                  45,011 (99.9%)\n",
      "  Read 2 with adapter:                  44,836 (99.5%)\n",
      "Pairs written (passing filters):        45,042 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,570,183 bp\n",
      "  Read 1:    13,286,069 bp\n",
      "  Read 2:    13,284,114 bp\n",
      "Total written (filtered):     24,234,545 bp (91.2%)\n",
      "  Read 1:    12,116,039 bp\n",
      "  Read 2:    12,118,506 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45011 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t589\t0.0\t2\t0 581 8\n",
      "26\t44068\t0.0\t2\t43179 874 15\n",
      "27\t255\t0.0\t2\t0 250 5\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t331\t0.0\t2\t0 301 30\n",
      "26\t44262\t0.0\t2\t40238 3759 265\n",
      "27\t223\t0.0\t2\t0 195 28\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S821_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S821_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S821_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S821_219_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S821_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S821_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S821_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S821_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,966 reads  @     89.0 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.36 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,966\n",
      "  Read 1 with adapter:                  48,942 (100.0%)\n",
      "  Read 2 with adapter:                  48,722 (99.5%)\n",
      "Pairs written (passing filters):        48,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,882,651 bp\n",
      "  Read 1:    14,444,374 bp\n",
      "  Read 2:    14,438,277 bp\n",
      "Total written (filtered):     26,343,543 bp (91.2%)\n",
      "  Read 1:    13,172,007 bp\n",
      "  Read 2:    13,171,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t597\t0.0\t2\t0 576 21\n",
      "26\t47929\t0.0\t2\t46955 964 10\n",
      "27\t276\t0.0\t2\t0 269 7\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48722 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t341\t0.0\t2\t0 309 32\n",
      "26\t48060\t0.0\t2\t43653 4081 326\n",
      "27\t304\t0.0\t2\t0 277 27\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S822_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S822_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S822_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S822_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S822_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S822_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S822_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S822_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        47,054 reads  @     99.0 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.66 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,054\n",
      "  Read 1 with adapter:                  47,024 (99.9%)\n",
      "  Read 2 with adapter:                  46,706 (99.3%)\n",
      "Pairs written (passing filters):        47,054 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,722,824 bp\n",
      "  Read 1:    13,880,365 bp\n",
      "  Read 2:    13,842,459 bp\n",
      "Total written (filtered):     25,286,190 bp (91.2%)\n",
      "  Read 1:    12,658,009 bp\n",
      "  Read 2:    12,628,181 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47024 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t573\t0.0\t2\t0 556 17\n",
      "26\t46126\t0.0\t2\t45118 989 19\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t322\t0.0\t2\t0 288 34\n",
      "26\t46102\t0.0\t2\t42064 3820 218\n",
      "27\t260\t0.0\t2\t0 225 35\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S823_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S823_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S823_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S823_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S823_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S823_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S823_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S823_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,679 reads  @     95.2 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.36 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,679\n",
      "  Read 1 with adapter:                  45,651 (99.9%)\n",
      "  Read 2 with adapter:                  45,256 (99.1%)\n",
      "Pairs written (passing filters):        45,679 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,899,954 bp\n",
      "  Read 1:    13,475,149 bp\n",
      "  Read 2:    13,424,805 bp\n",
      "Total written (filtered):     24,536,893 bp (91.2%)\n",
      "  Read 1:    12,288,517 bp\n",
      "  Read 2:    12,248,376 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45651 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t525\t0.0\t2\t0 515 10\n",
      "26\t44863\t0.0\t2\t43947 895 21\n",
      "27\t203\t0.0\t2\t0 197 6\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45256 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t439\t0.0\t2\t0 390 49\n",
      "26\t44536\t0.0\t2\t40586 3700 250\n",
      "27\t254\t0.0\t2\t0 227 27\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S824_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S824_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S824_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S824_225_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S824_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S824_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S824_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S824_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,709 reads  @     87.4 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.18 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,709\n",
      "  Read 1 with adapter:                  47,681 (99.9%)\n",
      "  Read 2 with adapter:                  47,587 (99.7%)\n",
      "Pairs written (passing filters):        47,709 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,145,419 bp\n",
      "  Read 1:    14,073,847 bp\n",
      "  Read 2:    14,071,572 bp\n",
      "Total written (filtered):     25,668,871 bp (91.2%)\n",
      "  Read 1:    12,834,482 bp\n",
      "  Read 2:    12,834,389 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t555\t0.0\t2\t0 538 17\n",
      "26\t46862\t0.0\t2\t45877 969 16\n",
      "27\t210\t0.0\t2\t0 200 10\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47587 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t344\t0.0\t2\t0 324 20\n",
      "26\t46948\t0.0\t2\t42838 3880 230\n",
      "27\t261\t0.0\t2\t0 227 34\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S825_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S825_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S825_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S825_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S825_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S825_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S825_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S825_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,017 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.54 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,017\n",
      "  Read 1 with adapter:                  58,990 (100.0%)\n",
      "  Read 2 with adapter:                  58,630 (99.3%)\n",
      "Pairs written (passing filters):        59,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,780,486 bp\n",
      "  Read 1:    17,409,999 bp\n",
      "  Read 2:    17,370,487 bp\n",
      "Total written (filtered):     31,723,024 bp (91.2%)\n",
      "  Read 1:    15,876,844 bp\n",
      "  Read 2:    15,846,180 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t742\t0.0\t2\t0 725 17\n",
      "26\t57997\t0.0\t2\t56476 1487 34\n",
      "27\t213\t0.0\t2\t0 207 6\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58630 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t427\t0.0\t2\t0 388 39\n",
      "26\t57820\t0.0\t2\t52726 4796 298\n",
      "27\t312\t0.0\t2\t0 283 29\n",
      "28\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S826_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S826_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S826_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S826_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S826_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S826_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S826_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S826_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        42,947 reads  @    100.0 µs/read;   0.60 M reads/minute\n",
      "Finished in 4.31 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,947\n",
      "  Read 1 with adapter:                  42,928 (100.0%)\n",
      "  Read 2 with adapter:                  42,793 (99.6%)\n",
      "Pairs written (passing filters):        42,947 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,334,152 bp\n",
      "  Read 1:    12,669,102 bp\n",
      "  Read 2:    12,665,050 bp\n",
      "Total written (filtered):     23,105,713 bp (91.2%)\n",
      "  Read 1:    11,553,188 bp\n",
      "  Read 2:    11,552,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42928 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t504\t0.0\t2\t0 493 11\n",
      "26\t42137\t0.0\t2\t41185 937 15\n",
      "27\t204\t0.0\t2\t0 199 5\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t316\t0.0\t2\t0 281 35\n",
      "26\t42234\t0.0\t2\t38545 3504 185\n",
      "27\t199\t0.0\t2\t0 182 17\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S827_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S827_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S827_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S827_231_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S827_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S827_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S827_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S827_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,584 reads  @     96.3 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.91 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,584\n",
      "  Read 1 with adapter:                  40,315 (99.3%)\n",
      "  Read 2 with adapter:                  40,518 (99.8%)\n",
      "Pairs written (passing filters):        40,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,875,743 bp\n",
      "  Read 1:    11,906,919 bp\n",
      "  Read 2:    11,968,824 bp\n",
      "Total written (filtered):     21,774,591 bp (91.2%)\n",
      "  Read 1:    10,858,998 bp\n",
      "  Read 2:    10,915,593 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40315 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t537\t0.0\t2\t0 526 11\n",
      "26\t39493\t0.0\t2\t38457 1018 18\n",
      "27\t222\t0.0\t2\t0 212 10\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t473\t0.0\t2\t0 436 37\n",
      "26\t39768\t0.0\t2\t36593 3092 83\n",
      "27\t238\t0.0\t2\t0 218 20\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S828_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S828_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S828_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S828_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S828_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S828_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S828_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S828_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,011 reads  @     95.0 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.29 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,011\n",
      "  Read 1 with adapter:                  44,806 (99.5%)\n",
      "  Read 2 with adapter:                  44,930 (99.8%)\n",
      "Pairs written (passing filters):        45,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,508,002 bp\n",
      "  Read 1:    13,231,237 bp\n",
      "  Read 2:    13,276,765 bp\n",
      "Total written (filtered):     24,175,038 bp (91.2%)\n",
      "  Read 1:    12,066,375 bp\n",
      "  Read 2:    12,108,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44806 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t467\t0.0\t2\t0 447 20\n",
      "26\t43999\t0.0\t2\t43067 912 20\n",
      "27\t219\t0.0\t2\t0 214 5\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44930 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t314\t0.0\t2\t0 291 23\n",
      "26\t44343\t0.0\t2\t40907 3300 136\n",
      "27\t230\t0.0\t2\t0 216 14\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S829_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S829_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S829_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S829_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S829_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S829_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S829_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S829_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,074 reads  @     91.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.13 s (92 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,074\n",
      "  Read 1 with adapter:                  44,968 (99.8%)\n",
      "  Read 2 with adapter:                  44,990 (99.8%)\n",
      "Pairs written (passing filters):        45,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,568,763 bp\n",
      "  Read 1:    13,276,614 bp\n",
      "  Read 2:    13,292,149 bp\n",
      "Total written (filtered):     24,229,948 bp (91.2%)\n",
      "  Read 1:    12,107,444 bp\n",
      "  Read 2:    12,122,504 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44968 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t445\t0.0\t2\t0 437 8\n",
      "26\t44140\t0.0\t2\t43155 967 18\n",
      "27\t235\t0.0\t2\t0 227 8\n",
      "28\t127\t0.0\t2\t0 0 127\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t319\t0.0\t2\t0 300 19\n",
      "26\t44400\t0.0\t2\t41058 3227 115\n",
      "27\t238\t0.0\t2\t0 215 23\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S830_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S830_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S830_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S830_237_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S830_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S830_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S830_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S830_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,164 reads  @     94.7 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.67 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,164\n",
      "  Read 1 with adapter:                  48,876 (99.4%)\n",
      "  Read 2 with adapter:                  48,952 (99.6%)\n",
      "Pairs written (passing filters):        49,164 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,908,056 bp\n",
      "  Read 1:    14,436,220 bp\n",
      "  Read 2:    14,471,836 bp\n",
      "Total written (filtered):     26,364,800 bp (91.2%)\n",
      "  Read 1:    13,165,611 bp\n",
      "  Read 2:    13,199,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t552\t0.0\t2\t0 535 17\n",
      "26\t47955\t0.0\t2\t46912 1024 19\n",
      "27\t281\t0.0\t2\t0 273 8\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48952 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t356\t0.0\t2\t0 323 33\n",
      "26\t48314\t0.0\t2\t44663 3559 92\n",
      "27\t249\t0.0\t2\t0 226 23\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S831_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S831_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S831_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S831_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S831_238_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S831_239_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S831_238_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S831_239_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        46,559 reads  @     96.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.48 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,559\n",
      "  Read 1 with adapter:                  46,532 (99.9%)\n",
      "  Read 2 with adapter:                  46,517 (99.9%)\n",
      "Pairs written (passing filters):        46,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,466,808 bp\n",
      "  Read 1:    13,733,960 bp\n",
      "  Read 2:    13,732,848 bp\n",
      "Total written (filtered):     25,048,052 bp (91.2%)\n",
      "  Read 1:    12,524,429 bp\n",
      "  Read 2:    12,523,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t560\t0.0\t2\t0 541 19\n",
      "26\t45668\t0.0\t2\t44635 1009 24\n",
      "27\t245\t0.0\t2\t0 234 11\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t475\t0.0\t2\t0 441 34\n",
      "26\t45763\t0.0\t2\t42485 3204 74\n",
      "27\t212\t0.0\t2\t0 202 10\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S832_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S832_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S832_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S832_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S832_240_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S832_241_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S832_240_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S832_241_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,893 reads  @     83.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.61 s (84 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,893\n",
      "  Read 1 with adapter:                  54,858 (99.9%)\n",
      "  Read 2 with adapter:                  54,823 (99.9%)\n",
      "Pairs written (passing filters):        54,893 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,385,641 bp\n",
      "  Read 1:    16,192,985 bp\n",
      "  Read 2:    16,192,656 bp\n",
      "Total written (filtered):     29,534,393 bp (91.2%)\n",
      "  Read 1:    14,767,058 bp\n",
      "  Read 2:    14,767,335 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t666\t0.0\t2\t0 653 13\n",
      "26\t53850\t0.0\t2\t52725 1107 18\n",
      "27\t279\t0.0\t2\t0 271 8\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54823 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t412\t0.0\t2\t0 382 30\n",
      "26\t54100\t0.0\t2\t50107 3876 117\n",
      "27\t235\t0.0\t2\t0 218 17\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S833_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S833_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S833_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S833_243_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S833_242_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S833_243_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S833_242_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S833_243_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        53,908 reads  @    112.5 µs/read;   0.53 M reads/minute\n",
      "Finished in 6.06 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,908\n",
      "  Read 1 with adapter:                  53,881 (99.9%)\n",
      "  Read 2 with adapter:                  53,811 (99.8%)\n",
      "Pairs written (passing filters):        53,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,799,091 bp\n",
      "  Read 1:    15,902,552 bp\n",
      "  Read 2:    15,896,539 bp\n",
      "Total written (filtered):     28,999,472 bp (91.2%)\n",
      "  Read 1:    14,501,929 bp\n",
      "  Read 2:    14,497,543 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53881 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t628\t0.0\t2\t0 615 13\n",
      "26\t52863\t0.0\t2\t51730 1120 13\n",
      "27\t335\t0.0\t2\t0 323 12\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53811 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t406\t0.0\t2\t0 387 19\n",
      "26\t53074\t0.0\t2\t49128 3829 117\n",
      "27\t290\t0.0\t2\t0 266 24\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S834_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S834_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S834_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S834_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S834_244_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S834_245_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S834_244_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S834_245_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        41,632 reads  @    149.2 µs/read;   0.40 M reads/minute\n",
      "Finished in 6.22 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,632\n",
      "  Read 1 with adapter:                  41,607 (99.9%)\n",
      "  Read 2 with adapter:                  41,573 (99.9%)\n",
      "Pairs written (passing filters):        41,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,560,015 bp\n",
      "  Read 1:    12,281,250 bp\n",
      "  Read 2:    12,278,765 bp\n",
      "Total written (filtered):     22,397,729 bp (91.2%)\n",
      "  Read 1:    11,199,773 bp\n",
      "  Read 2:    11,197,956 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t536\t0.0\t2\t0 526 10\n",
      "26\t40809\t0.0\t2\t39845 953 11\n",
      "27\t217\t0.0\t2\t0 212 5\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t9\t0.0\t2\t0 0 9\n",
      "25\t327\t0.0\t2\t0 311 16\n",
      "26\t40992\t0.0\t2\t37892 3032 68\n",
      "27\t234\t0.0\t2\t0 212 22\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S835_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S835_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S835_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S835_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S835_246_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S835_247_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S835_246_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S835_247_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        54,666 reads  @    134.7 µs/read;   0.45 M reads/minute\n",
      "Finished in 7.37 s (135 us/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,666\n",
      "  Read 1 with adapter:                  54,626 (99.9%)\n",
      "  Read 2 with adapter:                  54,606 (99.9%)\n",
      "Pairs written (passing filters):        54,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,249,083 bp\n",
      "  Read 1:    16,124,408 bp\n",
      "  Read 2:    16,124,675 bp\n",
      "Total written (filtered):     29,409,373 bp (91.2%)\n",
      "  Read 1:    14,704,164 bp\n",
      "  Read 2:    14,705,209 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t477\t0.0\t2\t0 462 15\n",
      "26\t53738\t0.0\t2\t52431 1289 18\n",
      "27\t257\t0.0\t2\t0 248 9\n",
      "28\t124\t0.0\t2\t0 0 124\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54606 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t535\t0.0\t2\t0 504 31\n",
      "26\t53726\t0.0\t2\t49530 4082 114\n",
      "27\t297\t0.0\t2\t0 279 18\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S836_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S836_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S836_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S836_249_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S836_248_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S836_249_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S836_248_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S836_249_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        47,212 reads  @    122.9 µs/read;   0.49 M reads/minute\n",
      "Finished in 5.81 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,212\n",
      "  Read 1 with adapter:                  47,183 (99.9%)\n",
      "  Read 2 with adapter:                  47,137 (99.8%)\n",
      "Pairs written (passing filters):        47,212 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,852,851 bp\n",
      "  Read 1:    13,926,426 bp\n",
      "  Read 2:    13,926,425 bp\n",
      "Total written (filtered):     25,400,646 bp (91.2%)\n",
      "  Read 1:    12,699,673 bp\n",
      "  Read 2:    12,700,973 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47183 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t354\t0.0\t2\t0 347 7\n",
      "26\t46457\t0.0\t2\t45554 883 20\n",
      "27\t315\t0.0\t2\t0 307 8\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t321\t0.0\t2\t0 298 23\n",
      "26\t46577\t0.0\t2\t42859 3614 104\n",
      "27\t223\t0.0\t2\t0 203 20\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S837_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S837_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S837_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S837_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S837_250_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S837_251_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S837_250_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S837_251_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        89,775 reads  @    122.5 µs/read;   0.49 M reads/minute\n",
      "Finished in 11.01 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,775\n",
      "  Read 1 with adapter:                  89,690 (99.9%)\n",
      "  Read 2 with adapter:                  89,555 (99.8%)\n",
      "Pairs written (passing filters):        89,775 (100.0%)\n",
      "\n",
      "Total basepairs processed:    52,948,241 bp\n",
      "  Read 1:    26,476,127 bp\n",
      "  Read 2:    26,472,114 bp\n",
      "Total written (filtered):     48,287,854 bp (91.2%)\n",
      "  Read 1:    24,144,232 bp\n",
      "  Read 2:    24,143,622 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 89690 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t683\t0.0\t2\t0 669 14\n",
      "26\t88341\t0.0\t2\t86425 1888 28\n",
      "27\t530\t0.0\t2\t0 521 9\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 89555 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t620\t0.0\t2\t0 549 71\n",
      "26\t88328\t0.0\t2\t81173 6847 308\n",
      "27\t392\t0.0\t2\t0 363 29\n",
      "28\t180\t0.0\t2\t0 0 180\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S838_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S838_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S838_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S838_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S838_252_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S838_253_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S838_252_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S838_253_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        50,477 reads  @    144.0 µs/read;   0.42 M reads/minute\n",
      "Finished in 7.28 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,477\n",
      "  Read 1 with adapter:                  50,346 (99.7%)\n",
      "  Read 2 with adapter:                  50,366 (99.8%)\n",
      "Pairs written (passing filters):        50,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,762,693 bp\n",
      "  Read 1:    14,872,328 bp\n",
      "  Read 2:    14,890,365 bp\n",
      "Total written (filtered):     27,144,354 bp (91.2%)\n",
      "  Read 1:    13,563,401 bp\n",
      "  Read 2:    13,580,953 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50346 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t385\t0.0\t2\t0 379 6\n",
      "26\t49609\t0.0\t2\t48526 1065 18\n",
      "27\t292\t0.0\t2\t0 281 11\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t377\t0.0\t2\t0 343 34\n",
      "26\t49684\t0.0\t2\t45899 3671 114\n",
      "27\t261\t0.0\t2\t0 242 19\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S839_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S839_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S839_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S839_255_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S839_254_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S839_255_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S839_254_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S839_255_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        43,434 reads  @    124.0 µs/read;   0.48 M reads/minute\n",
      "Finished in 5.40 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,434\n",
      "  Read 1 with adapter:                  43,363 (99.8%)\n",
      "  Read 2 with adapter:                  43,319 (99.7%)\n",
      "Pairs written (passing filters):        43,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,617,613 bp\n",
      "  Read 1:    12,808,062 bp\n",
      "  Read 2:    12,809,551 bp\n",
      "Total written (filtered):     23,364,146 bp (91.2%)\n",
      "  Read 1:    11,680,696 bp\n",
      "  Read 2:    11,683,450 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43363 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t411\t0.0\t2\t0 399 12\n",
      "26\t42630\t0.0\t2\t41733 882 15\n",
      "27\t257\t0.0\t2\t0 254 3\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43319 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t409\t0.0\t2\t0 364 45\n",
      "26\t42656\t0.0\t2\t39151 3347 158\n",
      "27\t212\t0.0\t2\t0 195 17\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S840_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S840_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S840_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S840_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S840_256_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S840_257_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S840_256_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S840_257_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[         8<-] 00:00:24       187,244 reads  @    130.4 µs/read;   0.46 M reads/minute\n",
      "Finished in 24.43 s (130 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,244\n",
      "  Read 1 with adapter:                 187,037 (99.9%)\n",
      "  Read 2 with adapter:                 186,996 (99.9%)\n",
      "Pairs written (passing filters):       187,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:   110,456,621 bp\n",
      "  Read 1:    55,226,520 bp\n",
      "  Read 2:    55,230,101 bp\n",
      "Total written (filtered):    100,732,918 bp (91.2%)\n",
      "  Read 1:    50,364,365 bp\n",
      "  Read 2:    50,368,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 187037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t1880\t0.0\t2\t0 1844 36\n",
      "26\t183983\t0.0\t2\t180005 3917 61\n",
      "27\t923\t0.0\t2\t0 897 26\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 186996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t1300\t0.0\t2\t0 1178 122\n",
      "26\t184703\t0.0\t2\t170655 13608 440\n",
      "27\t862\t0.0\t2\t0 797 65\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S841_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S841_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S841_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S841_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S841_258_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S841_259_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S841_258_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S841_259_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:06        51,182 reads  @    122.8 µs/read;   0.49 M reads/minute\n",
      "Finished in 6.29 s (123 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,182\n",
      "  Read 1 with adapter:                  51,149 (99.9%)\n",
      "  Read 2 with adapter:                  51,094 (99.8%)\n",
      "Pairs written (passing filters):        51,182 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,190,817 bp\n",
      "  Read 1:    15,096,853 bp\n",
      "  Read 2:    15,093,964 bp\n",
      "Total written (filtered):     27,532,601 bp (91.2%)\n",
      "  Read 1:    13,767,031 bp\n",
      "  Read 2:    13,765,570 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t455\t0.0\t2\t0 445 10\n",
      "26\t50300\t0.0\t2\t49192 1083 25\n",
      "27\t277\t0.0\t2\t0 262 15\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51094 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t322\t0.0\t2\t0 294 28\n",
      "26\t50482\t0.0\t2\t46567 3788 127\n",
      "27\t236\t0.0\t2\t0 219 17\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S842_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S842_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S842_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S842_261_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S842_260_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S842_261_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S842_260_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S842_261_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        50,286 reads  @    110.0 µs/read;   0.55 M reads/minute\n",
      "Finished in 5.54 s (110 us/read; 0.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,286\n",
      "  Read 1 with adapter:                  50,267 (100.0%)\n",
      "  Read 2 with adapter:                  50,220 (99.9%)\n",
      "Pairs written (passing filters):        50,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,667,052 bp\n",
      "  Read 1:    14,833,487 bp\n",
      "  Read 2:    14,833,565 bp\n",
      "Total written (filtered):     27,054,439 bp (91.2%)\n",
      "  Read 1:    13,526,514 bp\n",
      "  Read 2:    13,527,925 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50267 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t433\t0.0\t2\t0 424 9\n",
      "26\t49426\t0.0\t2\t48365 1045 16\n",
      "27\t268\t0.0\t2\t0 266 2\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50220 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t424\t0.0\t2\t0 394 30\n",
      "26\t49499\t0.0\t2\t45607 3767 125\n",
      "27\t206\t0.0\t2\t0 184 22\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S843_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S843_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S843_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S843_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S843_262_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S843_263_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S843_262_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S843_263_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        48,503 reads  @    105.7 µs/read;   0.57 M reads/minute\n",
      "Finished in 5.14 s (106 us/read; 0.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,503\n",
      "  Read 1 with adapter:                  48,479 (100.0%)\n",
      "  Read 2 with adapter:                  48,429 (99.8%)\n",
      "Pairs written (passing filters):        48,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,613,600 bp\n",
      "  Read 1:    14,308,072 bp\n",
      "  Read 2:    14,305,528 bp\n",
      "Total written (filtered):     26,094,276 bp (91.2%)\n",
      "  Read 1:    13,047,638 bp\n",
      "  Read 2:    13,046,638 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t393\t0.0\t2\t0 376 17\n",
      "26\t47741\t0.0\t2\t46737 984 20\n",
      "27\t245\t0.0\t2\t0 240 5\n",
      "28\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48429 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t474\t0.0\t2\t0 437 37\n",
      "26\t47700\t0.0\t2\t44102 3500 98\n",
      "27\t236\t0.0\t2\t0 212 24\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S844_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S844_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S844_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S844_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S844_264_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S844_265_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S844_264_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S844_265_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        42,205 reads  @    113.0 µs/read;   0.53 M reads/minute\n",
      "Finished in 4.77 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,205\n",
      "  Read 1 with adapter:                  42,185 (100.0%)\n",
      "  Read 2 with adapter:                  42,132 (99.8%)\n",
      "Pairs written (passing filters):        42,205 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,897,779 bp\n",
      "  Read 1:    12,450,132 bp\n",
      "  Read 2:    12,447,647 bp\n",
      "Total written (filtered):     22,705,717 bp (91.2%)\n",
      "  Read 1:    11,353,337 bp\n",
      "  Read 2:    11,352,380 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t326\t0.0\t2\t0 320 6\n",
      "26\t41548\t0.0\t2\t40540 984 24\n",
      "27\t207\t0.0\t2\t0 198 9\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42132 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t335\t0.0\t2\t0 315 20\n",
      "26\t41588\t0.0\t2\t38196 3266 126\n",
      "27\t180\t0.0\t2\t0 168 12\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S845_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S845_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S845_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S845_267_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S845_266_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S845_267_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S845_266_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S845_267_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        45,301 reads  @     97.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 4.44 s (98 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,301\n",
      "  Read 1 with adapter:                  45,281 (100.0%)\n",
      "  Read 2 with adapter:                  45,093 (99.5%)\n",
      "Pairs written (passing filters):        45,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,716,038 bp\n",
      "  Read 1:    13,363,486 bp\n",
      "  Read 2:    13,352,552 bp\n",
      "Total written (filtered):     24,366,341 bp (91.2%)\n",
      "  Read 1:    12,186,184 bp\n",
      "  Read 2:    12,180,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45281 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t323\t0.0\t2\t0 307 16\n",
      "26\t44647\t0.0\t2\t43730 906 11\n",
      "27\t247\t0.0\t2\t0 238 9\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45093 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t12\t0.0\t2\t0 0 12\n",
      "25\t277\t0.0\t2\t0 249 28\n",
      "26\t44564\t0.0\t2\t40362 3962 240\n",
      "27\t202\t0.0\t2\t0 179 23\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S846_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S846_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S846_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S846_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S846_268_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S846_269_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S846_268_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S846_269_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,644 reads  @     96.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.21 s (96 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,644\n",
      "  Read 1 with adapter:                  43,615 (99.9%)\n",
      "  Read 2 with adapter:                  43,531 (99.7%)\n",
      "Pairs written (passing filters):        43,644 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,748,231 bp\n",
      "  Read 1:    12,874,594 bp\n",
      "  Read 2:    12,873,637 bp\n",
      "Total written (filtered):     23,482,607 bp (91.2%)\n",
      "  Read 1:    11,740,656 bp\n",
      "  Read 2:    11,741,951 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43615 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t334\t0.0\t2\t0 327 7\n",
      "26\t42967\t0.0\t2\t42086 862 19\n",
      "27\t262\t0.0\t2\t0 257 5\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43531 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t319\t0.0\t2\t0 297 22\n",
      "26\t42960\t0.0\t2\t39664 3198 98\n",
      "27\t217\t0.0\t2\t0 198 19\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S847_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S847_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S847_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S847_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S847_270_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S847_271_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S847_270_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S847_271_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        41,362 reads  @    119.9 µs/read;   0.50 M reads/minute\n",
      "Finished in 4.97 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,362\n",
      "  Read 1 with adapter:                  41,321 (99.9%)\n",
      "  Read 2 with adapter:                  41,272 (99.8%)\n",
      "Pairs written (passing filters):        41,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,395,715 bp\n",
      "  Read 1:    12,198,672 bp\n",
      "  Read 2:    12,197,043 bp\n",
      "Total written (filtered):     22,248,631 bp (91.2%)\n",
      "  Read 1:    11,124,442 bp\n",
      "  Read 2:    11,124,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t352\t0.0\t2\t0 342 10\n",
      "26\t40711\t0.0\t2\t39851 838 22\n",
      "27\t204\t0.0\t2\t0 198 6\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41272 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t420\t0.0\t2\t0 388 32\n",
      "26\t40617\t0.0\t2\t37385 3120 112\n",
      "27\t212\t0.0\t2\t0 192 20\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S848_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S848_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S848_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S848_273_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S848_272_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S848_273_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S848_272_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S848_273_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:06        64,922 reads  @    106.3 µs/read;   0.56 M reads/minute\n",
      "Finished in 6.91 s (106 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,922\n",
      "  Read 1 with adapter:                  64,874 (99.9%)\n",
      "  Read 2 with adapter:                  64,815 (99.8%)\n",
      "Pairs written (passing filters):        64,922 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,294,757 bp\n",
      "  Read 1:    19,146,758 bp\n",
      "  Read 2:    19,147,999 bp\n",
      "Total written (filtered):     34,923,084 bp (91.2%)\n",
      "  Read 1:    17,460,175 bp\n",
      "  Read 2:    17,462,909 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64874 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t535\t0.0\t2\t0 519 16\n",
      "26\t63927\t0.0\t2\t62555 1352 20\n",
      "27\t322\t0.0\t2\t0 310 12\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64815 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t465\t0.0\t2\t0 437 28\n",
      "26\t63978\t0.0\t2\t58913 4883 182\n",
      "27\t299\t0.0\t2\t0 280 19\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S849_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S849_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S849_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S849_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S849_274_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S849_275_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S849_274_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S849_275_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        56,558 reads  @    101.3 µs/read;   0.59 M reads/minute\n",
      "Finished in 5.74 s (101 us/read; 0.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,558\n",
      "  Read 1 with adapter:                  56,534 (100.0%)\n",
      "  Read 2 with adapter:                  56,432 (99.8%)\n",
      "Pairs written (passing filters):        56,558 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,359,830 bp\n",
      "  Read 1:    16,682,927 bp\n",
      "  Read 2:    16,676,903 bp\n",
      "Total written (filtered):     30,422,884 bp (91.2%)\n",
      "  Read 1:    15,213,163 bp\n",
      "  Read 2:    15,209,721 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56534 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t486\t0.0\t2\t0 479 7\n",
      "26\t55664\t0.0\t2\t54607 1034 23\n",
      "27\t290\t0.0\t2\t0 286 4\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56432 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t380\t0.0\t2\t0 354 26\n",
      "26\t55694\t0.0\t2\t51340 4184 170\n",
      "27\t278\t0.0\t2\t0 257 21\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S850_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S850_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S850_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S850_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S850_276_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S850_277_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S850_276_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S850_277_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:04        44,232 reads  @    107.6 µs/read;   0.56 M reads/minute\n",
      "Finished in 4.77 s (108 us/read; 0.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,232\n",
      "  Read 1 with adapter:                  44,207 (99.9%)\n",
      "  Read 2 with adapter:                  44,154 (99.8%)\n",
      "Pairs written (passing filters):        44,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,094,735 bp\n",
      "  Read 1:    13,047,413 bp\n",
      "  Read 2:    13,047,322 bp\n",
      "Total written (filtered):     23,797,559 bp (91.2%)\n",
      "  Read 1:    11,898,137 bp\n",
      "  Read 2:    11,899,422 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44207 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t385\t0.0\t2\t0 376 9\n",
      "26\t43555\t0.0\t2\t42652 886 17\n",
      "27\t191\t0.0\t2\t0 191\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t340\t0.0\t2\t0 309 31\n",
      "26\t43569\t0.0\t2\t40211 3239 119\n",
      "27\t214\t0.0\t2\t0 196 18\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S851_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S851_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S851_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S851_279_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S851_278_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S851_279_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S851_278_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S851_279_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        50,263 reads  @    100.5 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.06 s (101 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,263\n",
      "  Read 1 with adapter:                  50,242 (100.0%)\n",
      "  Read 2 with adapter:                  50,074 (99.6%)\n",
      "Pairs written (passing filters):        50,263 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,650,892 bp\n",
      "  Read 1:    14,827,101 bp\n",
      "  Read 2:    14,823,791 bp\n",
      "Total written (filtered):     27,042,857 bp (91.2%)\n",
      "  Read 1:    13,521,009 bp\n",
      "  Read 2:    13,521,848 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50242 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t578\t0.0\t2\t0 562 16\n",
      "26\t49320\t0.0\t2\t48044 1255 21\n",
      "27\t250\t0.0\t2\t0 240 10\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50074 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t378\t0.0\t2\t0 341 37\n",
      "26\t49309\t0.0\t2\t44506 4515 288\n",
      "27\t305\t0.0\t2\t0 282 23\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S852_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S852_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S852_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S852_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S852_280_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-9f0og_lk/S852_281_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S852_280_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-5m2qhsyo/156fea36-5e9a-46ab-abcc-850c505370d4/data/S852_281_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        44,604 reads  @     89.5 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.00 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,604\n",
      "  Read 1 with adapter:                  44,572 (99.9%)\n",
      "  Read 2 with adapter:                  44,520 (99.8%)\n",
      "Pairs written (passing filters):        44,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,313,960 bp\n",
      "  Read 1:    13,158,150 bp\n",
      "  Read 2:    13,155,810 bp\n",
      "Total written (filtered):     23,997,856 bp (91.2%)\n",
      "  Read 1:    11,999,501 bp\n",
      "  Read 2:    11,998,355 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t581\t0.0\t2\t0 567 14\n",
      "26\t43593\t0.0\t2\t42457 1116 20\n",
      "27\t306\t0.0\t2\t0 295 11\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44520 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t345\t0.0\t2\t0 317 28\n",
      "26\t43873\t0.0\t2\t40270 3474 129\n",
      "27\t248\t0.0\t2\t0 224 24\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: COI_1-demux-pairedend-trimmed1.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences COI_1-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences COI_1-demux-pairedend-trimmed1.qza \\\n",
    "    --verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI_1-demux-pairedend-trimmed1.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data COI_1-demux-pairedend-trimmed1.qza \\\n",
    "  --o-visualization COI_1-demux-pairedend-trimmed1.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-mphc4418')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 5efbc65d-b411-4943-ad78-86511b894c26>"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('COI_1-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI_1-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI_1-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: COI_1-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs COI_1-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 258 \\\n",
    "  --p-trunc-len-r 168 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method pooled \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats COI_1-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-table.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization COI-table.qzv \\\n",
    "  \n",
    "#--m-sample-metadata-file metadata_COI_clean.txt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-tcuzzdne')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 618879c4-1885-473a-be9b-176d7910456e>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2_vsearch97.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime vsearch cluster-features-de-novo \\\n",
    "  --i-table COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --i-sequences COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --p-perc-identity 0.97 \\\n",
    "  --o-clustered-table  COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --o-clustered-sequences COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza  \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-sj23qp5n')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 2b42649a-556d-416f-9b14-14661a1f9321>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_table-dada2_vsearch97.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize \\\n",
    "  --i-table COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --o-visualization COI-paired-end-demux_table-dada2_vsearch97.qzv \\\n",
    " # --m-sample-metadata-file metadata_COI_clean.txt\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-cdspvdl9')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 57b20fda-80f9-4fff-b5ab-78c90c7bc492>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_table-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported COI-paired-end-demux_table-dada2_vsearch97.qza as BIOMV210DirFmt to directory exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --output-path exported-feature-table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o COI_97vsearch_OTUtable.tsv --to-tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31m\u001b[1mPlugin error from feature-classifier:\n",
      "\n",
      "  [Errno 28] No space left on device\n",
      "\n",
      "Debug info has been saved to /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-q2cli-err-t9c84duv.log\u001b[0m\n",
      "\u001b[31m\u001b[1mThere was an issue with loading the file COI-midori-sklearn-taxonomy_untrimmed.qza as metadata:\n",
      "\n",
      "  Metadata file path doesn't exist, or the path points to something other than a file. Please check that the path exists, has read permissions, and points to a regular file (not a directory): COI-midori-sklearn-taxonomy_untrimmed.qza\n",
      "\n",
      "  There may be more errors present in the metadata file. To get a full report, sample/feature metadata files can be validated with Keemei: https://keemei.qiime2.org\n",
      "\n",
      "  Find details on QIIME 2 metadata requirements here: https://docs.qiime2.org/2020.6/tutorials/metadata/\u001b[0m\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /Users/stephaniematthews/Documents/Chapter1/qiime2_analyses/midori_database/midori-classifier_untrimmed.qza \\\n",
    "  --i-reads COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza \\\n",
    "  --o-classification COI-midori-sklearn-taxonomy_untrimmed.qza\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file COI-midori-sklearn-taxonomy_untrimmed.qza \\\n",
    "  --o-visualization COI-midori-sklearn-taxonomy_untrimmed.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: COI-mzg_trimmed_taxonomy-oct22.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: COI-mzg_trimmed_taxonomy-oct22.qzv\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-se52jf8u')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 9e215dac-5ede-49bf-99ef-dfc2b121ff81>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /Users/stephaniematthews/Documents/databases/MetaZooGene26Oct2022/MZG-leray-classifier_r_trimmed.qza \\\n",
    "  --i-reads COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza \\\n",
    "  --o-classification COI-mzg_trimmed_taxonomy-oct22.qza\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file COI-mzg_trimmed_taxonomy-oct22.qza \\\n",
    "  --o-visualization COI-mzg_trimmed_taxonomy-oct22.qzv\n",
    "\n",
    "import qiime2 as q2\n",
    "q2.Visualization.load('COI-mzg_trimmed_taxonomy-oct22.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-MZG-sklearn-taxonomy_untrimmed.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-rc7_h4ep')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c1f52f59-7ad5-4e9e-93ba-55add0750ec1>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file COI-midori-sklearn-taxonomy_untrimmed.qza \\\n",
    "  --o-visualization COI-MZG-sklearn-taxonomy_untrimmed.qzv\n",
    "\n",
    "import qiime2 as q2\n",
    "q2.Visualization.load('COI-MZG-sklearn-taxonomy_untrimmed.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/chimeras.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/nonchimeras.qza\u001b[0m\n",
      "\u001b[32mSaved UchimeStats to: uchime-dn-out/stats.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: uchime-dn-out/stats.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "##detect chimeras\n",
    "\n",
    "!qiime vsearch uchime-denovo \\\n",
    "  --i-table COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --i-sequences COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza \\\n",
    "  --p-minh 0.13 \\\n",
    "  --output-dir uchime-dn-out\n",
    "\n",
    "!qiime metadata tabulate \\\n",
    "  --m-input-file uchime-dn-out/stats.qza \\\n",
    "  --o-visualization uchime-dn-out/stats.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-drwk11kn')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 043c6ba6-9d4a-4158-b7ad-9e9ec9604821>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('uchime-dn-out/stats.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: uchime-dn-out/table-nonchimeric-wo-borderline.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: uchime-dn-out/table-nonchimeric-wo-borderline.qzv\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "##filter out detected chimeras\n",
    "\n",
    "!qiime feature-table filter-features \\\n",
    "  --i-table COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --m-metadata-file uchime-dn-out/nonchimeras.qza \\\n",
    "  --o-filtered-table uchime-dn-out/table-nonchimeric-wo-borderline.qza\n",
    "!qiime feature-table filter-seqs \\\n",
    "  --i-data COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza \\\n",
    "  --m-metadata-file uchime-dn-out/nonchimeras.qza \\\n",
    "  --o-filtered-data uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza\n",
    "!qiime feature-table summarize \\\n",
    "  --i-table uchime-dn-out/table-nonchimeric-wo-borderline.qza \\\n",
    "  --o-visualization uchime-dn-out/table-nonchimeric-wo-borderline.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv\u001b[0m\r\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-7ck0063v')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 9e3b5c15-c2ab-4d2c-8b87-3489b7f2ddd4>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "##view the seqs to download as fasta\n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qza \\\n",
    "  --o-visualization uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv\n",
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('uchime-dn-out/rep-seqs-nonchimeric-wo-borderline.qzv')\n",
    "#download as fasta from visualized data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported uchime-dn-out/table-nonchimeric-wo-borderline.qza as BIOMV210DirFmt to directory exported-feature-table-nonchimerica-uchime\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#export table to biom\n",
    "!qiime tools export \\\n",
    "  --input-path uchime-dn-out/table-nonchimeric-wo-borderline.qza \\\n",
    "  --output-path exported-feature-table-nonchimerica-uchime\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/18S_ASVtable.tsv --to-tsv\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\"\n",
    "!biom convert -i exported-feature-table-nonchimerica-uchime/feature-table.biom -o exported-feature-table-nonchimerica-uchime/feature-table_json.biom --to-json --table-type=\"OTU table\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
